<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.7.29">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">

<meta name="author" content="Luna Villalobos Araya">
<meta name="author" content="Amaru Simón Agüero Jiménez">
<meta name="dcterms.date" content="2025-11-25">

<title>Análisis de Redes en Cáncer de Mama</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1em; /* quarto-specific, see https://github.com/quarto-dev/quarto-cli/issues/4556 */ 
  vertical-align: middle;
}
/* CSS for syntax highlighting */
html { -webkit-text-size-adjust: 100%; }
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
  }
pre.numberSource { margin-left: 3em;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
</style>


<script src="index_files/libs/clipboard/clipboard.min.js"></script>
<script src="index_files/libs/quarto-html/quarto.js" type="module"></script>
<script src="index_files/libs/quarto-html/tabsets/tabsets.js" type="module"></script>
<script src="index_files/libs/quarto-html/popper.min.js"></script>
<script src="index_files/libs/quarto-html/tippy.umd.min.js"></script>
<script src="index_files/libs/quarto-html/anchor.min.js"></script>
<link href="index_files/libs/quarto-html/tippy.css" rel="stylesheet">
<link href="index_files/libs/quarto-html/quarto-syntax-highlighting-0815c480559380816a4d1ea211a47e91.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="index_files/libs/bootstrap/bootstrap.min.js"></script>
<link href="index_files/libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="index_files/libs/bootstrap/bootstrap-813c323200a87c37e262811031999de4.min.css" rel="stylesheet" append-hash="true" id="quarto-bootstrap" data-mode="light">
<style>html{ scroll-behavior: smooth; }</style>
<link href="index_files/libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet">

<script src="index_files/libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>

<link href="index_files/libs/datatables-css-0.0.0/datatables-crosstalk.css" rel="stylesheet">

<script src="index_files/libs/datatables-binding-0.33/datatables.js"></script>

<script src="index_files/libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>

<link href="index_files/libs/dt-core-1.13.6/css/jquery.dataTables.min.css" rel="stylesheet">

<link href="index_files/libs/dt-core-1.13.6/css/jquery.dataTables.extra.css" rel="stylesheet">

<script src="index_files/libs/dt-core-1.13.6/js/jquery.dataTables.min.js"></script>

<script src="index_files/libs/jszip-1.13.6/jszip.min.js"></script>

<link href="index_files/libs/dt-ext-buttons-1.13.6/css/buttons.dataTables.min.css" rel="stylesheet">

<script src="index_files/libs/dt-ext-buttons-1.13.6/js/dataTables.buttons.min.js"></script>

<script src="index_files/libs/dt-ext-buttons-1.13.6/js/buttons.html5.min.js"></script>

<script src="index_files/libs/dt-ext-buttons-1.13.6/js/buttons.colVis.min.js"></script>

<script src="index_files/libs/dt-ext-buttons-1.13.6/js/buttons.print.min.js"></script>

<link href="index_files/libs/crosstalk-1.2.1/css/crosstalk.min.css" rel="stylesheet">

<script src="index_files/libs/crosstalk-1.2.1/js/crosstalk.min.js"></script>


  <script src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6"></script>
  <script src="https://cdn.jsdelivr.net/npm/mathjax@3/es5/tex-chtml-full.js" type="text/javascript"></script>

<script type="text/javascript">
const typesetMath = (el) => {
  if (window.MathJax) {
    // MathJax Typeset
    window.MathJax.typeset([el]);
  } else if (window.katex) {
    // KaTeX Render
    var mathElements = el.getElementsByClassName("math");
    var macros = [];
    for (var i = 0; i < mathElements.length; i++) {
      var texText = mathElements[i].firstChild;
      if (mathElements[i].tagName == "SPAN") {
        window.katex.render(texText.data, mathElements[i], {
          displayMode: mathElements[i].classList.contains('display'),
          throwOnError: false,
          macros: macros,
          fleqn: false
        });
      }
    }
  }
}
window.Quarto = {
  typesetMath
};
</script>

</head>

<body class="quarto-light">

<div id="quarto-content" class="page-columns page-rows-contents page-layout-article">
<div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
  <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">Table of contents</h2>
   
  <ul>
  <li><a href="#metodología-análisis-de-redes-en-cáncer-de-mama" id="toc-metodología-análisis-de-redes-en-cáncer-de-mama" class="nav-link active" data-scroll-target="#metodología-análisis-de-redes-en-cáncer-de-mama"><span class="header-section-number">1</span> Metodología: Análisis de Redes en Cáncer de Mama</a>
  <ul>
  <li><a href="#fuente-de-datos-y-características-del-estudio" id="toc-fuente-de-datos-y-características-del-estudio" class="nav-link" data-scroll-target="#fuente-de-datos-y-características-del-estudio"><span class="header-section-number">1.1</span> Fuente de Datos y Características del Estudio</a>
  <ul>
  <li><a href="#dataset-metabric" id="toc-dataset-metabric" class="nav-link" data-scroll-target="#dataset-metabric"><span class="header-section-number">1.1.1</span> Dataset METABRIC</a></li>
  <li><a href="#variables-analizadas" id="toc-variables-analizadas" class="nav-link" data-scroll-target="#variables-analizadas"><span class="header-section-number">1.1.2</span> Variables Analizadas</a></li>
  </ul></li>
  <li><a href="#preparación-y-procesamiento-de-datos" id="toc-preparación-y-procesamiento-de-datos" class="nav-link" data-scroll-target="#preparación-y-procesamiento-de-datos"><span class="header-section-number">1.2</span> Preparación y Procesamiento de Datos</a>
  <ul>
  <li><a href="#limpieza-de-datos" id="toc-limpieza-de-datos" class="nav-link" data-scroll-target="#limpieza-de-datos"><span class="header-section-number">1.2.1</span> Limpieza de Datos</a></li>
  <li><a href="#transformación-de-datos-para-análisis-de-redes" id="toc-transformación-de-datos-para-análisis-de-redes" class="nav-link" data-scroll-target="#transformación-de-datos-para-análisis-de-redes"><span class="header-section-number">1.2.2</span> Transformación de Datos para Análisis de Redes</a></li>
  </ul></li>
  <li><a href="#construcción-de-redes-de-co-ocurrencia" id="toc-construcción-de-redes-de-co-ocurrencia" class="nav-link" data-scroll-target="#construcción-de-redes-de-co-ocurrencia"><span class="header-section-number">1.3</span> Construcción de Redes de Co-ocurrencia</a>
  <ul>
  <li><a href="#principio-metodológico" id="toc-principio-metodológico" class="nav-link" data-scroll-target="#principio-metodológico"><span class="header-section-number">1.3.1</span> Principio Metodológico</a></li>
  <li><a href="#algoritmo-de-construcción" id="toc-algoritmo-de-construcción" class="nav-link" data-scroll-target="#algoritmo-de-construcción"><span class="header-section-number">1.3.2</span> Algoritmo de Construcción</a></li>
  <li><a href="#redes-construidas" id="toc-redes-construidas" class="nav-link" data-scroll-target="#redes-construidas"><span class="header-section-number">1.3.3</span> Redes Construidas</a></li>
  </ul></li>
  <li><a href="#métricas-de-topología-de-redes" id="toc-métricas-de-topología-de-redes" class="nav-link" data-scroll-target="#métricas-de-topología-de-redes"><span class="header-section-number">1.4</span> Métricas de Topología de Redes</a>
  <ul>
  <li><a href="#métricas-globales" id="toc-métricas-globales" class="nav-link" data-scroll-target="#métricas-globales"><span class="header-section-number">1.4.1</span> Métricas Globales</a></li>
  <li><a href="#métricas-de-centralidad-nodal" id="toc-métricas-de-centralidad-nodal" class="nav-link" data-scroll-target="#métricas-de-centralidad-nodal"><span class="header-section-number">1.4.2</span> Métricas de Centralidad Nodal</a></li>
  </ul></li>
  <li><a href="#detección-de-comunidades" id="toc-detección-de-comunidades" class="nav-link" data-scroll-target="#detección-de-comunidades"><span class="header-section-number">1.5</span> Detección de Comunidades</a>
  <ul>
  <li><a href="#algoritmo-de-louvain" id="toc-algoritmo-de-louvain" class="nav-link" data-scroll-target="#algoritmo-de-louvain"><span class="header-section-number">1.5.1</span> Algoritmo de Louvain</a></li>
  <li><a href="#caracterización-de-comunidades" id="toc-caracterización-de-comunidades" class="nav-link" data-scroll-target="#caracterización-de-comunidades"><span class="header-section-number">1.5.2</span> Caracterización de Comunidades</a></li>
  </ul></li>
  <li><a href="#análisis-de-k-core" id="toc-análisis-de-k-core" class="nav-link" data-scroll-target="#análisis-de-k-core"><span class="header-section-number">1.6</span> Análisis de K-Core</a>
  <ul>
  <li><a href="#definición" id="toc-definición" class="nav-link" data-scroll-target="#definición"><span class="header-section-number">1.6.1</span> Definición</a></li>
  <li><a href="#procedimiento" id="toc-procedimiento" class="nav-link" data-scroll-target="#procedimiento"><span class="header-section-number">1.6.2</span> Procedimiento</a></li>
  <li><a href="#interpretación-biológica" id="toc-interpretación-biológica" class="nav-link" data-scroll-target="#interpretación-biológica"><span class="header-section-number">1.6.3</span> Interpretación Biológica</a></li>
  </ul></li>
  <li><a href="#identificación-de-nodos-hub-y-puente" id="toc-identificación-de-nodos-hub-y-puente" class="nav-link" data-scroll-target="#identificación-de-nodos-hub-y-puente"><span class="header-section-number">1.7</span> Identificación de Nodos Hub y Puente</a>
  <ul>
  <li><a href="#nodos-hub" id="toc-nodos-hub" class="nav-link" data-scroll-target="#nodos-hub"><span class="header-section-number">1.7.1</span> Nodos Hub</a></li>
  <li><a href="#nodos-puente-gateway-nodes" id="toc-nodos-puente-gateway-nodes" class="nav-link" data-scroll-target="#nodos-puente-gateway-nodes"><span class="header-section-number">1.7.2</span> Nodos Puente (Gateway Nodes)</a></li>
  </ul></li>
  <li><a href="#análisis-comparativos" id="toc-análisis-comparativos" class="nav-link" data-scroll-target="#análisis-comparativos"><span class="header-section-number">1.8</span> Análisis Comparativos</a>
  <ul>
  <li><a href="#comparación-entre-sistemas-de-clasificación" id="toc-comparación-entre-sistemas-de-clasificación" class="nav-link" data-scroll-target="#comparación-entre-sistemas-de-clasificación"><span class="header-section-number">1.8.1</span> Comparación Entre Sistemas de Clasificación</a></li>
  <li><a href="#análisis-de-subgrupos-específicos" id="toc-análisis-de-subgrupos-específicos" class="nav-link" data-scroll-target="#análisis-de-subgrupos-específicos"><span class="header-section-number">1.8.2</span> Análisis de Subgrupos Específicos</a></li>
  <li><a href="#análisis-de-supervivencia-y-recurrencia" id="toc-análisis-de-supervivencia-y-recurrencia" class="nav-link" data-scroll-target="#análisis-de-supervivencia-y-recurrencia"><span class="header-section-number">1.8.3</span> Análisis de Supervivencia y Recurrencia</a></li>
  </ul></li>
  <li><a href="#software-y-herramientas-computacionales" id="toc-software-y-herramientas-computacionales" class="nav-link" data-scroll-target="#software-y-herramientas-computacionales"><span class="header-section-number">1.9</span> Software y Herramientas Computacionales</a>
  <ul>
  <li><a href="#lenguaje-y-entorno" id="toc-lenguaje-y-entorno" class="nav-link" data-scroll-target="#lenguaje-y-entorno"><span class="header-section-number">1.9.1</span> Lenguaje y Entorno</a></li>
  <li><a href="#paquetes-utilizados" id="toc-paquetes-utilizados" class="nav-link" data-scroll-target="#paquetes-utilizados"><span class="header-section-number">1.9.2</span> Paquetes Utilizados</a></li>
  <li><a href="#parámetros-de-visualización" id="toc-parámetros-de-visualización" class="nav-link" data-scroll-target="#parámetros-de-visualización"><span class="header-section-number">1.9.3</span> Parámetros de Visualización</a></li>
  </ul></li>
  <li><a href="#consideraciones-éticas-y-de-datos" id="toc-consideraciones-éticas-y-de-datos" class="nav-link" data-scroll-target="#consideraciones-éticas-y-de-datos"><span class="header-section-number">1.10</span> Consideraciones Éticas y de Datos</a>
  <ul>
  <li><a href="#datos-públicos" id="toc-datos-públicos" class="nav-link" data-scroll-target="#datos-públicos"><span class="header-section-number">1.10.1</span> Datos Públicos</a></li>
  <li><a href="#reproducibilidad" id="toc-reproducibilidad" class="nav-link" data-scroll-target="#reproducibilidad"><span class="header-section-number">1.10.2</span> Reproducibilidad</a></li>
  <li><a href="#limitaciones-metodológicas" id="toc-limitaciones-metodológicas" class="nav-link" data-scroll-target="#limitaciones-metodológicas"><span class="header-section-number">1.10.3</span> Limitaciones Metodológicas</a></li>
  </ul></li>
  <li><a href="#validación-y-control-de-calidad" id="toc-validación-y-control-de-calidad" class="nav-link" data-scroll-target="#validación-y-control-de-calidad"><span class="header-section-number">1.11</span> Validación y Control de Calidad</a>
  <ul>
  <li><a href="#validación-de-redes" id="toc-validación-de-redes" class="nav-link" data-scroll-target="#validación-de-redes"><span class="header-section-number">1.11.1</span> Validación de Redes</a></li>
  <li><a href="#control-de-calidad-de-datos" id="toc-control-de-calidad-de-datos" class="nav-link" data-scroll-target="#control-de-calidad-de-datos"><span class="header-section-number">1.11.2</span> Control de Calidad de Datos</a></li>
  <li><a href="#sensibilidad-de-parámetros" id="toc-sensibilidad-de-parámetros" class="nav-link" data-scroll-target="#sensibilidad-de-parámetros"><span class="header-section-number">1.11.3</span> Sensibilidad de Parámetros</a></li>
  </ul></li>
  <li><a href="#criterios-de-interpretación" id="toc-criterios-de-interpretación" class="nav-link" data-scroll-target="#criterios-de-interpretación"><span class="header-section-number">1.12</span> Criterios de Interpretación</a>
  <ul>
  <li><a href="#significancia-de-métricas" id="toc-significancia-de-métricas" class="nav-link" data-scroll-target="#significancia-de-métricas"><span class="header-section-number">1.12.1</span> Significancia de Métricas</a></li>
  <li><a href="#umbrales-de-centralidad" id="toc-umbrales-de-centralidad" class="nav-link" data-scroll-target="#umbrales-de-centralidad"><span class="header-section-number">1.12.2</span> Umbrales de Centralidad</a></li>
  <li><a href="#relevancia-clínica" id="toc-relevancia-clínica" class="nav-link" data-scroll-target="#relevancia-clínica"><span class="header-section-number">1.12.3</span> Relevancia Clínica</a></li>
  </ul></li>
  <li><a href="#referencias-metodológicas" id="toc-referencias-metodológicas" class="nav-link" data-scroll-target="#referencias-metodológicas"><span class="header-section-number">1.13</span> Referencias Metodológicas</a></li>
  </ul></li>
  <li><a href="#resultados" id="toc-resultados" class="nav-link" data-scroll-target="#resultados"><span class="header-section-number">2</span> Resultados</a>
  <ul>
  <li><a href="#exploración-de-clasificaciones-génicas" id="toc-exploración-de-clasificaciones-génicas" class="nav-link" data-scroll-target="#exploración-de-clasificaciones-génicas"><span class="header-section-number">2.1</span> Exploración de Clasificaciones Génicas</a>
  <ul>
  <li><a href="#distribución-de-subtipos-moleculares" id="toc-distribución-de-subtipos-moleculares" class="nav-link" data-scroll-target="#distribución-de-subtipos-moleculares"><span class="header-section-number">2.1.1</span> Distribución de Subtipos Moleculares</a>
  <ul>
  <li><a href="#tabla-distribución-pam50-claudin-low" id="toc-tabla-distribución-pam50-claudin-low" class="nav-link" data-scroll-target="#tabla-distribución-pam50-claudin-low"><span class="header-section-number">2.1.1.1</span> Tabla: Distribución PAM50 + Claudin-low</a></li>
  <li><a href="#tabla-distribución-3-gene-classifier" id="toc-tabla-distribución-3-gene-classifier" class="nav-link" data-scroll-target="#tabla-distribución-3-gene-classifier"><span class="header-section-number">2.1.1.2</span> Tabla: Distribución 3-Gene Classifier</a></li>
  <li><a href="#tabla-distribución-integrative-cluster" id="toc-tabla-distribución-integrative-cluster" class="nav-link" data-scroll-target="#tabla-distribución-integrative-cluster"><span class="header-section-number">2.1.1.3</span> Tabla: Distribución Integrative Cluster</a></li>
  </ul></li>
  <li><a href="#visualización-de-distribuciones" id="toc-visualización-de-distribuciones" class="nav-link" data-scroll-target="#visualización-de-distribuciones"><span class="header-section-number">2.1.2</span> Visualización de Distribuciones</a></li>
  </ul></li>
  <li><a href="#preparación-de-datos-para-análisis-de-redes" id="toc-preparación-de-datos-para-análisis-de-redes" class="nav-link" data-scroll-target="#preparación-de-datos-para-análisis-de-redes"><span class="header-section-number">2.2</span> Preparación de Datos para Análisis de Redes</a>
  <ul>
  <li><a href="#selección-y-limpieza-de-variables-clínicas" id="toc-selección-y-limpieza-de-variables-clínicas" class="nav-link" data-scroll-target="#selección-y-limpieza-de-variables-clínicas"><span class="header-section-number">2.2.1</span> Selección y Limpieza de Variables Clínicas</a></li>
  </ul></li>
  <li><a href="#construcción-de-redes-de-co-ocurrencia-1" id="toc-construcción-de-redes-de-co-ocurrencia-1" class="nav-link" data-scroll-target="#construcción-de-redes-de-co-ocurrencia-1"><span class="header-section-number">2.3</span> Construcción de Redes de Co-ocurrencia</a>
  <ul>
  <li><a href="#red-por-subtipo-pam50" id="toc-red-por-subtipo-pam50" class="nav-link" data-scroll-target="#red-por-subtipo-pam50"><span class="header-section-number">2.3.1</span> Red por Subtipo PAM50</a>
  <ul>
  <li><a href="#tabla-métricas-de-red-pam50" id="toc-tabla-métricas-de-red-pam50" class="nav-link" data-scroll-target="#tabla-métricas-de-red-pam50"><span class="header-section-number">2.3.1.1</span> Tabla: Métricas de Red PAM50</a></li>
  <li><a href="#visualización-red-de-co-ocurrencia-pam50" id="toc-visualización-red-de-co-ocurrencia-pam50" class="nav-link" data-scroll-target="#visualización-red-de-co-ocurrencia-pam50"><span class="header-section-number">2.3.1.2</span> Visualización: Red de Co-ocurrencia PAM50</a></li>
  <li><a href="#tabla-top-nodos-centrales-pam50" id="toc-tabla-top-nodos-centrales-pam50" class="nav-link" data-scroll-target="#tabla-top-nodos-centrales-pam50"><span class="header-section-number">2.3.1.3</span> Tabla: Top Nodos Centrales (PAM50)</a></li>
  </ul></li>
  <li><a href="#red-por-clasificador-3-gene" id="toc-red-por-clasificador-3-gene" class="nav-link" data-scroll-target="#red-por-clasificador-3-gene"><span class="header-section-number">2.3.2</span> Red por Clasificador 3-Gene</a>
  <ul>
  <li><a href="#tabla-métricas-de-red-3-gene" id="toc-tabla-métricas-de-red-3-gene" class="nav-link" data-scroll-target="#tabla-métricas-de-red-3-gene"><span class="header-section-number">2.3.2.1</span> Tabla: Métricas de Red 3-Gene</a></li>
  <li><a href="#visualización-red-3-gene-classifier" id="toc-visualización-red-3-gene-classifier" class="nav-link" data-scroll-target="#visualización-red-3-gene-classifier"><span class="header-section-number">2.3.2.2</span> Visualización: Red 3-Gene Classifier</a></li>
  <li><a href="#tabla-top-nodos-centrales-3-gene" id="toc-tabla-top-nodos-centrales-3-gene" class="nav-link" data-scroll-target="#tabla-top-nodos-centrales-3-gene"><span class="header-section-number">2.3.2.3</span> Tabla: Top Nodos Centrales (3-Gene)</a></li>
  </ul></li>
  <li><a href="#red-por-integrative-cluster" id="toc-red-por-integrative-cluster" class="nav-link" data-scroll-target="#red-por-integrative-cluster"><span class="header-section-number">2.3.3</span> Red por Integrative Cluster</a>
  <ul>
  <li><a href="#tabla-métricas-de-red-integrative-cluster" id="toc-tabla-métricas-de-red-integrative-cluster" class="nav-link" data-scroll-target="#tabla-métricas-de-red-integrative-cluster"><span class="header-section-number">2.3.3.1</span> Tabla: Métricas de Red Integrative Cluster</a></li>
  <li><a href="#visualización-red-integrative-cluster" id="toc-visualización-red-integrative-cluster" class="nav-link" data-scroll-target="#visualización-red-integrative-cluster"><span class="header-section-number">2.3.3.2</span> Visualización: Red Integrative Cluster</a></li>
  <li><a href="#tabla-top-nodos-centrales-integrative-cluster" id="toc-tabla-top-nodos-centrales-integrative-cluster" class="nav-link" data-scroll-target="#tabla-top-nodos-centrales-integrative-cluster"><span class="header-section-number">2.3.3.3</span> Tabla: Top Nodos Centrales (Integrative Cluster)</a></li>
  </ul></li>
  </ul></li>
  <li><a href="#comparación-de-redes" id="toc-comparación-de-redes" class="nav-link" data-scroll-target="#comparación-de-redes"><span class="header-section-number">2.4</span> Comparación de Redes</a>
  <ul>
  <li><a href="#comparación-de-métricas-topológicas" id="toc-comparación-de-métricas-topológicas" class="nav-link" data-scroll-target="#comparación-de-métricas-topológicas"><span class="header-section-number">2.4.1</span> Comparación de Métricas Topológicas</a></li>
  <li><a href="#visualización-comparativa" id="toc-visualización-comparativa" class="nav-link" data-scroll-target="#visualización-comparativa"><span class="header-section-number">2.4.2</span> Visualización Comparativa</a></li>
  </ul></li>
  <li><a href="#análisis-de-comunidades" id="toc-análisis-de-comunidades" class="nav-link" data-scroll-target="#análisis-de-comunidades"><span class="header-section-number">2.5</span> Análisis de Comunidades</a>
  <ul>
  <li><a href="#comunidades-en-red-pam50" id="toc-comunidades-en-red-pam50" class="nav-link" data-scroll-target="#comunidades-en-red-pam50"><span class="header-section-number">2.5.1</span> Comunidades en Red PAM50</a></li>
  <li><a href="#comunidades-en-red-3-gene" id="toc-comunidades-en-red-3-gene" class="nav-link" data-scroll-target="#comunidades-en-red-3-gene"><span class="header-section-number">2.5.2</span> Comunidades en Red 3-Gene</a></li>
  <li><a href="#comunidades-en-red-integrative-cluster" id="toc-comunidades-en-red-integrative-cluster" class="nav-link" data-scroll-target="#comunidades-en-red-integrative-cluster"><span class="header-section-number">2.5.3</span> Comunidades en Red Integrative Cluster</a></li>
  </ul></li>
  <li><a href="#análisis-de-sub-redes-por-subtipo-específico" id="toc-análisis-de-sub-redes-por-subtipo-específico" class="nav-link" data-scroll-target="#análisis-de-sub-redes-por-subtipo-específico"><span class="header-section-number">2.6</span> Análisis de Sub-redes por Subtipo Específico</a>
  <ul>
  <li><a href="#comparación-luma-vs-lumb-pam50" id="toc-comparación-luma-vs-lumb-pam50" class="nav-link" data-scroll-target="#comparación-luma-vs-lumb-pam50"><span class="header-section-number">2.6.1</span> Comparación: LumA vs LumB (PAM50)</a></li>
  <li><a href="#comparación-triple-negativo-vs-her2" id="toc-comparación-triple-negativo-vs-her2" class="nav-link" data-scroll-target="#comparación-triple-negativo-vs-her2"><span class="header-section-number">2.6.2</span> Comparación: Triple Negativo vs HER2+</a></li>
  </ul></li>
  <li><a href="#análisis-de-k-core-1" id="toc-análisis-de-k-core-1" class="nav-link" data-scroll-target="#análisis-de-k-core-1"><span class="header-section-number">2.7</span> Análisis de K-Core</a>
  <ul>
  <li><a href="#k-core-en-red-pam50" id="toc-k-core-en-red-pam50" class="nav-link" data-scroll-target="#k-core-en-red-pam50"><span class="header-section-number">2.7.1</span> K-Core en Red PAM50</a></li>
  <li><a href="#visualización-k-core-pam50" id="toc-visualización-k-core-pam50" class="nav-link" data-scroll-target="#visualización-k-core-pam50"><span class="header-section-number">2.7.2</span> Visualización K-Core PAM50</a></li>
  </ul></li>
  <li><a href="#identificación-de-nodos-hub-y-puente-1" id="toc-identificación-de-nodos-hub-y-puente-1" class="nav-link" data-scroll-target="#identificación-de-nodos-hub-y-puente-1"><span class="header-section-number">2.8</span> Identificación de Nodos Hub y Puente</a>
  <ul>
  <li><a href="#gateway-score-nodos-puente" id="toc-gateway-score-nodos-puente" class="nav-link" data-scroll-target="#gateway-score-nodos-puente"><span class="header-section-number">2.8.1</span> Gateway Score (Nodos Puente)</a></li>
  <li><a href="#nodos-hub-alto-grado-y-centralidad" id="toc-nodos-hub-alto-grado-y-centralidad" class="nav-link" data-scroll-target="#nodos-hub-alto-grado-y-centralidad"><span class="header-section-number">2.8.2</span> Nodos Hub (Alto Grado y Centralidad)</a></li>
  </ul></li>
  <li><a href="#análisis-de-supervivencia-por-subtipo" id="toc-análisis-de-supervivencia-por-subtipo" class="nav-link" data-scroll-target="#análisis-de-supervivencia-por-subtipo"><span class="header-section-number">2.9</span> Análisis de Supervivencia por Subtipo</a>
  <ul>
  <li><a href="#supervivencia-por-pam50" id="toc-supervivencia-por-pam50" class="nav-link" data-scroll-target="#supervivencia-por-pam50"><span class="header-section-number">2.9.1</span> Supervivencia por PAM50</a></li>
  <li><a href="#recurrencia-por-subtipo" id="toc-recurrencia-por-subtipo" class="nav-link" data-scroll-target="#recurrencia-por-subtipo"><span class="header-section-number">2.9.2</span> Recurrencia por Subtipo</a></li>
  </ul></li>
  <li><a href="#resumen-ejecutivo" id="toc-resumen-ejecutivo" class="nav-link" data-scroll-target="#resumen-ejecutivo"><span class="header-section-number">2.10</span> Resumen Ejecutivo</a>
  <ul>
  <li><a href="#hallazgos-principales" id="toc-hallazgos-principales" class="nav-link" data-scroll-target="#hallazgos-principales"><span class="header-section-number">2.10.1</span> Hallazgos Principales</a></li>
  <li><a href="#implicaciones-clínicas" id="toc-implicaciones-clínicas" class="nav-link" data-scroll-target="#implicaciones-clínicas"><span class="header-section-number">2.10.2</span> Implicaciones Clínicas</a>
  <ul>
  <li><a href="#características-clave-identificadas" id="toc-características-clave-identificadas" class="nav-link" data-scroll-target="#características-clave-identificadas"><span class="header-section-number">2.10.2.1</span> Características clave identificadas:</a></li>
  </ul></li>
  <li><a href="#consideraciones-para-investigación-futura" id="toc-consideraciones-para-investigación-futura" class="nav-link" data-scroll-target="#consideraciones-para-investigación-futura"><span class="header-section-number">2.10.3</span> Consideraciones para investigación futura:</a></li>
  </ul></li>
  <li><a href="#información-de-sesión" id="toc-información-de-sesión" class="nav-link" data-scroll-target="#información-de-sesión"><span class="header-section-number">2.11</span> Información de Sesión</a></li>
  </ul></li>
  <li><a href="#resultados-1" id="toc-resultados-1" class="nav-link" data-scroll-target="#resultados-1"><span class="header-section-number">3</span> Resultados</a></li>
  <li><a href="#discusión" id="toc-discusión" class="nav-link" data-scroll-target="#discusión"><span class="header-section-number">4</span> Discusión</a></li>
  </ul>
</nav>
</div>
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">Análisis de Redes en Cáncer de Mama</h1>
</div>



<div class="quarto-title-meta">

    <div>
    <div class="quarto-title-meta-heading">Authors</div>
    <div class="quarto-title-meta-contents">
             <p>Luna Villalobos Araya <a href="mailto:llvillalobos@miuandes.cl" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-7336-1833" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
             <p>Amaru Simón Agüero Jiménez <a href="mailto:aaguero@miuandes.cl" class="quarto-title-author-email"><i class="bi bi-envelope"></i></a> <a href="https://orcid.org/0000-0001-7336-1833" class="quarto-title-author-orcid"> <img src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABAAAAAQCAYAAAAf8/9hAAAAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAA2ZpVFh0WE1MOmNvbS5hZG9iZS54bXAAAAAAADw/eHBhY2tldCBiZWdpbj0i77u/IiBpZD0iVzVNME1wQ2VoaUh6cmVTek5UY3prYzlkIj8+IDx4OnhtcG1ldGEgeG1sbnM6eD0iYWRvYmU6bnM6bWV0YS8iIHg6eG1wdGs9IkFkb2JlIFhNUCBDb3JlIDUuMC1jMDYwIDYxLjEzNDc3NywgMjAxMC8wMi8xMi0xNzozMjowMCAgICAgICAgIj4gPHJkZjpSREYgeG1sbnM6cmRmPSJodHRwOi8vd3d3LnczLm9yZy8xOTk5LzAyLzIyLXJkZi1zeW50YXgtbnMjIj4gPHJkZjpEZXNjcmlwdGlvbiByZGY6YWJvdXQ9IiIgeG1sbnM6eG1wTU09Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC9tbS8iIHhtbG5zOnN0UmVmPSJodHRwOi8vbnMuYWRvYmUuY29tL3hhcC8xLjAvc1R5cGUvUmVzb3VyY2VSZWYjIiB4bWxuczp4bXA9Imh0dHA6Ly9ucy5hZG9iZS5jb20veGFwLzEuMC8iIHhtcE1NOk9yaWdpbmFsRG9jdW1lbnRJRD0ieG1wLmRpZDo1N0NEMjA4MDI1MjA2ODExOTk0QzkzNTEzRjZEQTg1NyIgeG1wTU06RG9jdW1lbnRJRD0ieG1wLmRpZDozM0NDOEJGNEZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wTU06SW5zdGFuY2VJRD0ieG1wLmlpZDozM0NDOEJGM0ZGNTcxMUUxODdBOEVCODg2RjdCQ0QwOSIgeG1wOkNyZWF0b3JUb29sPSJBZG9iZSBQaG90b3Nob3AgQ1M1IE1hY2ludG9zaCI+IDx4bXBNTTpEZXJpdmVkRnJvbSBzdFJlZjppbnN0YW5jZUlEPSJ4bXAuaWlkOkZDN0YxMTc0MDcyMDY4MTE5NUZFRDc5MUM2MUUwNEREIiBzdFJlZjpkb2N1bWVudElEPSJ4bXAuZGlkOjU3Q0QyMDgwMjUyMDY4MTE5OTRDOTM1MTNGNkRBODU3Ii8+IDwvcmRmOkRlc2NyaXB0aW9uPiA8L3JkZjpSREY+IDwveDp4bXBtZXRhPiA8P3hwYWNrZXQgZW5kPSJyIj8+84NovQAAAR1JREFUeNpiZEADy85ZJgCpeCB2QJM6AMQLo4yOL0AWZETSqACk1gOxAQN+cAGIA4EGPQBxmJA0nwdpjjQ8xqArmczw5tMHXAaALDgP1QMxAGqzAAPxQACqh4ER6uf5MBlkm0X4EGayMfMw/Pr7Bd2gRBZogMFBrv01hisv5jLsv9nLAPIOMnjy8RDDyYctyAbFM2EJbRQw+aAWw/LzVgx7b+cwCHKqMhjJFCBLOzAR6+lXX84xnHjYyqAo5IUizkRCwIENQQckGSDGY4TVgAPEaraQr2a4/24bSuoExcJCfAEJihXkWDj3ZAKy9EJGaEo8T0QSxkjSwORsCAuDQCD+QILmD1A9kECEZgxDaEZhICIzGcIyEyOl2RkgwAAhkmC+eAm0TAAAAABJRU5ErkJggg=="></a></p>
          </div>
  </div>
    
    <div>
    <div class="quarto-title-meta-heading">Published</div>
    <div class="quarto-title-meta-contents">
      <p class="date">November 25, 2025</p>
    </div>
  </div>
  
    
  </div>
  


</header>


<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a>install_and_load_packages <span class="ot">&lt;-</span> <span class="cf">function</span>(packages) {</span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a>  <span class="cf">for</span> (package <span class="cf">in</span> packages) {</span>
<span id="cb1-3"><a href="#cb1-3" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span> (<span class="sc">!</span><span class="fu">require</span>(package, <span class="at">character.only =</span> <span class="cn">TRUE</span>)) {</span>
<span id="cb1-4"><a href="#cb1-4" aria-hidden="true" tabindex="-1"></a>      <span class="fu">install.packages</span>(package)</span>
<span id="cb1-5"><a href="#cb1-5" aria-hidden="true" tabindex="-1"></a>      <span class="fu">library</span>(package, <span class="at">character.only =</span> <span class="cn">TRUE</span>)</span>
<span id="cb1-6"><a href="#cb1-6" aria-hidden="true" tabindex="-1"></a>    }</span>
<span id="cb1-7"><a href="#cb1-7" aria-hidden="true" tabindex="-1"></a>  }</span>
<span id="cb1-8"><a href="#cb1-8" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb1-9"><a href="#cb1-9" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-10"><a href="#cb1-10" aria-hidden="true" tabindex="-1"></a>necessary_packages <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"knitr"</span>, <span class="st">"tidyverse"</span>, <span class="st">"naniar"</span>,<span class="st">"kableExtra"</span>, <span class="st">"DT"</span>, <span class="st">"readr"</span>, <span class="st">"naniar"</span>, <span class="st">"table1"</span>, <span class="st">"summarytools"</span>,<span class="st">"readxl"</span>,<span class="st">"writexl"</span>,<span class="st">"scales"</span>, <span class="st">"ggrepel"</span>)</span>
<span id="cb1-11"><a href="#cb1-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-12"><a href="#cb1-12" aria-hidden="true" tabindex="-1"></a><span class="fu">install_and_load_packages</span>(necessary_packages)</span>
<span id="cb1-13"><a href="#cb1-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-14"><a href="#cb1-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-15"><a href="#cb1-15" aria-hidden="true" tabindex="-1"></a>opts_chunk<span class="sc">$</span><span class="fu">set</span>(</span>
<span id="cb1-16"><a href="#cb1-16" aria-hidden="true" tabindex="-1"></a>  <span class="at">echo =</span> <span class="cn">TRUE</span>,</span>
<span id="cb1-17"><a href="#cb1-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">warning =</span> <span class="cn">FALSE</span>,</span>
<span id="cb1-18"><a href="#cb1-18" aria-hidden="true" tabindex="-1"></a>  <span class="at">message =</span> <span class="cn">FALSE</span>)</span>
<span id="cb1-19"><a href="#cb1-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb1-20"><a href="#cb1-20" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">warn =</span> <span class="sc">-</span><span class="dv">1</span>)  <span class="co"># Suprime advertencias</span></span>
<span id="cb1-21"><a href="#cb1-21" aria-hidden="true" tabindex="-1"></a><span class="fu">options</span>(<span class="at">readr.show_col_types =</span> <span class="cn">FALSE</span>)  <span class="co"># Evita mensajes de `read_csv()`</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb2"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Librerías principales</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(igraph)</span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggraph)</span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(DT)</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(viridis)</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(RColorBrewer)</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(gridExtra)</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(knitr)</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(scales)</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a><span class="co"># Configuración de tema para gráficos</span></span>
<span id="cb2-13"><a href="#cb2-13" aria-hidden="true" tabindex="-1"></a><span class="fu">theme_set</span>(<span class="fu">theme_minimal</span>(<span class="at">base_size =</span> <span class="dv">12</span>))</span>
<span id="cb2-14"><a href="#cb2-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-15"><a href="#cb2-15" aria-hidden="true" tabindex="-1"></a>datos <span class="ot">&lt;-</span> <span class="fu">read.delim</span>(<span class="fu">paste0</span>(<span class="fu">gsub</span>(<span class="st">"/docs"</span>, <span class="st">""</span>, <span class="fu">getwd</span>()), <span class="st">"/data/brca_metabric_clinical_data.tsv"</span>),</span>
<span id="cb2-16"><a href="#cb2-16" aria-hidden="true" tabindex="-1"></a>                    <span class="at">header =</span> <span class="cn">TRUE</span>,        <span class="co"># Si la primera fila tiene nombres de columnas</span></span>
<span id="cb2-17"><a href="#cb2-17" aria-hidden="true" tabindex="-1"></a>                    <span class="at">sep =</span> <span class="st">"</span><span class="sc">\t</span><span class="st">"</span>,           <span class="co"># Separador tabulación (por defecto en read.delim)</span></span>
<span id="cb2-18"><a href="#cb2-18" aria-hidden="true" tabindex="-1"></a>                    <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>,</span>
<span id="cb2-19"><a href="#cb2-19" aria-hidden="true" tabindex="-1"></a>                    <span class="at">encoding =</span> <span class="st">"UTF-8"</span>)   <span class="co"># Ajusta si usas otro encoding</span></span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<section id="metodología-análisis-de-redes-en-cáncer-de-mama" class="level1" data-number="1">
<h1 data-number="1"><span class="header-section-number">1</span> Metodología: Análisis de Redes en Cáncer de Mama</h1>
<section id="fuente-de-datos-y-características-del-estudio" class="level2" data-number="1.1">
<h2 data-number="1.1" class="anchored" data-anchor-id="fuente-de-datos-y-características-del-estudio"><span class="header-section-number">1.1</span> Fuente de Datos y Características del Estudio</h2>
<section id="dataset-metabric" class="level3" data-number="1.1.1">
<h3 data-number="1.1.1" class="anchored" data-anchor-id="dataset-metabric"><span class="header-section-number">1.1.1</span> Dataset METABRIC</h3>
<p>Los datos utilizados provienen del estudio Molecular Taxonomy of Breast Cancer International Consortium (METABRIC), un estudio de cohorte retrospectivo que caracteriza molecularmente 2,509 tumores de cáncer de mama primario. El dataset incluye:</p>
<ul>
<li><span class="math inline">\(N = 2{,}509\)</span> pacientes con cáncer de mama invasivo</li>
<li>39 variables clínicas y moleculares</li>
<li>Datos de seguimiento a largo plazo (supervivencia y recurrencia)</li>
<li>Tres sistemas de clasificación molecular principal:
<ul>
<li>PAM50 + Claudin-low subtype (7 subtipos)</li>
<li>3-Gene Classifier (4 subtipos)</li>
<li>Integrative Cluster (11 clusters)</li>
</ul></li>
</ul>
</section>
<section id="variables-analizadas" class="level3" data-number="1.1.2">
<h3 data-number="1.1.2" class="anchored" data-anchor-id="variables-analizadas"><span class="header-section-number">1.1.2</span> Variables Analizadas</h3>
<p>Se seleccionaron 22 variables para el análisis de redes, categorizadas en:</p>
<p><strong>Variables Moleculares:</strong> - Subtipo PAM50 (LumA, LumB, Her2, Basal, Normal, claudin-low, NC) - Clasificador 3-Gene (ER+/HER2- High Prolif, ER+/HER2- Low Prolif, ER-/HER2-, HER2+) - Cluster Integrativo (1-10, 4ER+, 4ER-)</p>
<p><strong>Receptores Hormonales:</strong> - Estado ER (Positive, Negative) - Estado PR (Positive, Negative) - Estado HER2 (Positive, Negative)</p>
<p><strong>Características Tumorales:</strong> - Grado histológico neoplásico (1, 2, 3) - Estadio tumoral (0, 1, 2, 3, 4) - Celularidad (Low, Moderate, High) - Subtipo histológico (Ductal/NST, Lobular, Mixed, etc.) - Tamaño tumoral categorizado (&lt;20mm, 20-30mm, 30-50mm, &gt;50mm) - Ganglios linfáticos positivos (N0, N1, N2-3, N&gt;3)</p>
<p><strong>Tratamiento:</strong> - Quimioterapia (YES, NO) - Terapia hormonal (YES, NO) - Radioterapia (YES, NO) - Tipo de cirugía (MASTECTOMY, BREAST CONSERVING)</p>
<p><strong>Características Demográficas y Clínicas:</strong> - Grupo etario al diagnóstico (&lt;40, 40-50, 50-60, 60-70, &gt;70 años) - Estado menopáusico inferido (Pre, Post) - Lateralidad del tumor primario (Left, Right)</p>
<p><strong>Desenlaces:</strong> - Estado de supervivencia global (LIVING, DECEASED) - Estado libre de recurrencia (Not Recurred, Recurred)</p>
<hr>
</section>
</section>
<section id="preparación-y-procesamiento-de-datos" class="level2" data-number="1.2">
<h2 data-number="1.2" class="anchored" data-anchor-id="preparación-y-procesamiento-de-datos"><span class="header-section-number">1.2</span> Preparación y Procesamiento de Datos</h2>
<section id="limpieza-de-datos" class="level3" data-number="1.2.1">
<h3 data-number="1.2.1" class="anchored" data-anchor-id="limpieza-de-datos"><span class="header-section-number">1.2.1</span> Limpieza de Datos</h3>
<ol type="1">
<li><p><strong>Manejo de valores perdidos:</strong> Los valores NA fueron reemplazados por la categoría “Unknown” para preservar información sobre patrones de datos faltantes.</p></li>
<li><p><strong>Estandarización de nombres:</strong> Las variables fueron renombradas para facilitar el manejo programático:</p>
<ul>
<li>Eliminación de caracteres especiales</li>
<li>Uso de convenciones consistentes</li>
<li>Nombres cortos y descriptivos</li>
</ul></li>
<li><p><strong>Categorización de variables continuas:</strong></p>
<ul>
<li><strong>Edad:</strong> Dividida en quintiles clínicamente relevantes</li>
<li><strong>Tamaño tumoral:</strong> Categorizada según clasificación T del sistema TNM</li>
<li><strong>Ganglios linfáticos:</strong> Agrupados según clasificación N</li>
</ul></li>
</ol>
</section>
<section id="transformación-de-datos-para-análisis-de-redes" class="level3" data-number="1.2.2">
<h3 data-number="1.2.2" class="anchored" data-anchor-id="transformación-de-datos-para-análisis-de-redes"><span class="header-section-number">1.2.2</span> Transformación de Datos para Análisis de Redes</h3>
<p>Para cada sistema de clasificación molecular (PAM50, 3-Gene, Integrative Cluster), se creó una matriz de características en formato largo:</p>
<ol type="1">
<li>Cada <strong>fila</strong> representa una combinación única de paciente-característica</li>
<li>Cada <strong>característica</strong> se codifica como “Variable: Valor” (ej. “ER: Positive”)</li>
<li>Se genera una <strong>matriz binaria</strong> paciente × característica donde:
<ul>
<li>Valor = 1 si el paciente presenta la característica</li>
<li>Valor = 0 si no la presenta</li>
</ul>
Formalmente: <span class="math inline">\(X \in \{0,1\}^{n \times m}\)</span> donde <span class="math inline">\(n\)</span> = pacientes, <span class="math inline">\(m\)</span> = características</li>
</ol>
<hr>
</section>
</section>
<section id="construcción-de-redes-de-co-ocurrencia" class="level2" data-number="1.3">
<h2 data-number="1.3" class="anchored" data-anchor-id="construcción-de-redes-de-co-ocurrencia"><span class="header-section-number">1.3</span> Construcción de Redes de Co-ocurrencia</h2>
<section id="principio-metodológico" class="level3" data-number="1.3.1">
<h3 data-number="1.3.1" class="anchored" data-anchor-id="principio-metodológico"><span class="header-section-number">1.3.1</span> Principio Metodológico</h3>
<p>Las redes de co-ocurrencia capturan relaciones entre características clínicas basándose en su tendencia a aparecer juntas en los mismos pacientes. Esta aproximación permite:</p>
<ul>
<li>Identificar <strong>patrones de multimorbilidad</strong> molecular y clínica</li>
<li>Detectar <strong>asociaciones no obvias</strong> entre características</li>
<li>Revelar <strong>estructura modular</strong> en los perfiles clínicos</li>
</ul>
</section>
<section id="algoritmo-de-construcción" class="level3" data-number="1.3.2">
<h3 data-number="1.3.2" class="anchored" data-anchor-id="algoritmo-de-construcción"><span class="header-section-number">1.3.2</span> Algoritmo de Construcción</h3>
<p><strong>Paso 1: Matriz de Co-ocurrencia</strong></p>
<p><span class="math display">\[C = X^T \times X\]</span></p>
<p>Donde: - <span class="math inline">\(X\)</span> = matriz binaria (<span class="math inline">\(n\)</span> pacientes × <span class="math inline">\(m\)</span> características) - <span class="math inline">\(C\)</span> = matriz de co-ocurrencia (<span class="math inline">\(m \times m\)</span>) - <span class="math inline">\(C_{ij}\)</span> = número de pacientes que presentan ambas características <span class="math inline">\(i\)</span> y <span class="math inline">\(j\)</span></p>
<p><strong>Paso 2: Normalización</strong> Para corregir por diferencias en frecuencia de características individuales, se aplicó normalización tipo Jaccard:</p>
<p><span class="math display">\[C_{\text{norm}}[i,j] = \frac{C_{ij}}{\sqrt{C_{ii} \times C_{jj}}}\]</span></p>
<p>Esta normalización produce valores entre 0 y 1, donde: - 0 = co-ocurrencia nula - 1 = co-ocurrencia perfecta</p>
<p><strong>Paso 3: Filtrado de Aristas</strong> Para reducir ruido y enfocarse en asociaciones fuertes: - Se calculó el <strong>percentil 75</strong> (<span class="math inline">\(P_{75}\)</span>) de todos los pesos de co-ocurrencia - Se eliminaron aristas con peso <span class="math inline">\(&lt; P_{75}\)</span> - Esto resulta en redes más densas y biológicamente interpretables</p>
<p><strong>Paso 4: Eliminación de Nodos Aislados</strong> Nodos sin conexiones (grado = 0) fueron eliminados del análisis, ya que no aportan información sobre patrones de co-ocurrencia.</p>
</section>
<section id="redes-construidas" class="level3" data-number="1.3.3">
<h3 data-number="1.3.3" class="anchored" data-anchor-id="redes-construidas"><span class="header-section-number">1.3.3</span> Redes Construidas</h3>
<p>Se generaron <strong>tres redes independientes</strong>, una para cada sistema de clasificación:</p>
<ol type="1">
<li><strong>Red PAM50:</strong> Incluye los 7 subtipos PAM50 + características clínicas</li>
<li><strong>Red 3-Gene:</strong> Incluye los 4 subtipos del clasificador 3-Gene + características clínicas</li>
<li><strong>Red Integrative Cluster:</strong> Incluye los 11 clusters integrativos + características clínicas</li>
</ol>
<hr>
</section>
</section>
<section id="métricas-de-topología-de-redes" class="level2" data-number="1.4">
<h2 data-number="1.4" class="anchored" data-anchor-id="métricas-de-topología-de-redes"><span class="header-section-number">1.4</span> Métricas de Topología de Redes</h2>
<section id="métricas-globales" class="level3" data-number="1.4.1">
<h3 data-number="1.4.1" class="anchored" data-anchor-id="métricas-globales"><span class="header-section-number">1.4.1</span> Métricas Globales</h3>
<p>Para caracterizar la estructura general de cada red, se calcularon:</p>
<p><strong>Nodos y Aristas:</strong> - Número total de nodos (características) - Número total de aristas (co-ocurrencias significativas)</p>
<p><strong>Densidad (D):</strong></p>
<p><span class="math display">\[D = \frac{2E}{N(N-1)}\]</span></p>
<p>Donde <span class="math inline">\(E\)</span> = número de aristas, <span class="math inline">\(N\)</span> = número de nodos</p>
<p>Interpretación: Proporción de conexiones observadas respecto al máximo posible. Valores altos indican redes altamente interconectadas.</p>
<p><strong>Coeficiente de Clustering Global (C):</strong></p>
<p><span class="math display">\[C = \frac{\text{Número de triángulos} \times 3}{\text{Número de tríadas conectadas}}\]</span></p>
<p>Interpretación: Tendencia de los nodos a formar grupos densos. Valores altos sugieren modularidad.</p>
<p><strong>Diámetro (d):</strong></p>
<p><span class="math display">\[d = \max_{i,j} \{\text{distancia}_{\text{mínima}}(i,j)\}\]</span></p>
<p>Interpretación: Distancia máxima entre dos nodos cualesquiera. Refleja la “extensión” de la red.</p>
<p><strong>Modularidad (Q):</strong> Métrica que cuantifica la calidad de la partición en comunidades:</p>
<p><span class="math display">\[Q = \frac{1}{2m} \sum_{ij} \left[A_{ij} - \frac{k_i k_j}{2m}\right] \delta(c_i, c_j)\]</span></p>
<p>Donde: - <span class="math inline">\(m\)</span> = número de aristas - <span class="math inline">\(A_{ij}\)</span> = peso de la arista entre <span class="math inline">\(i\)</span> y <span class="math inline">\(j\)</span> - <span class="math inline">\(k_i, k_j\)</span> = grados de los nodos <span class="math inline">\(i\)</span> y <span class="math inline">\(j\)</span> - <span class="math inline">\(\delta(c_i, c_j) = 1\)</span> si <span class="math inline">\(i\)</span> y <span class="math inline">\(j\)</span> están en la misma comunidad, 0 en caso contrario</p>
<p>Valores <span class="math inline">\(Q &gt; 0.3\)</span> indican estructura comunitaria significativa.</p>
</section>
<section id="métricas-de-centralidad-nodal" class="level3" data-number="1.4.2">
<h3 data-number="1.4.2" class="anchored" data-anchor-id="métricas-de-centralidad-nodal"><span class="header-section-number">1.4.2</span> Métricas de Centralidad Nodal</h3>
<p>Para identificar características “importantes” en la red:</p>
<p><strong>Grado (Degree):</strong></p>
<p><span class="math display">\[k_i = \sum_j A_{ij}\]</span></p>
<p>Número de conexiones directas de un nodo. Identifica características que co-ocurren con muchas otras.</p>
<p><strong>Intermediación (Betweenness Centrality):</strong></p>
<p><span class="math display">\[BC_i = \sum_{s \neq i \neq t} \frac{\sigma_{st}(i)}{\sigma_{st}}\]</span></p>
<p>Donde <span class="math inline">\(\sigma_{st}\)</span> = número de caminos más cortos entre <span class="math inline">\(s\)</span> y <span class="math inline">\(t\)</span>, y <span class="math inline">\(\sigma_{st}(i)\)</span> = número que pasan por <span class="math inline">\(i\)</span>.</p>
<p>Identifica “puentes” entre diferentes regiones de la red.</p>
<p><strong>Cercanía (Closeness Centrality):</strong></p>
<p><span class="math display">\[CC_i = \frac{1}{\sum_j d(i,j)}\]</span></p>
<p>Inverso de la suma de distancias a todos los demás nodos. Identifica nodos “centrales”.</p>
<p><strong>Eigenvector Centrality:</strong> Centralidad basada en la importancia de los vecinos:</p>
<p><span class="math display">\[x_i = \frac{1}{\lambda} \sum_j A_{ij} x_j\]</span></p>
<p>Identifica nodos conectados a otros nodos importantes.</p>
<hr>
</section>
</section>
<section id="detección-de-comunidades" class="level2" data-number="1.5">
<h2 data-number="1.5" class="anchored" data-anchor-id="detección-de-comunidades"><span class="header-section-number">1.5</span> Detección de Comunidades</h2>
<section id="algoritmo-de-louvain" class="level3" data-number="1.5.1">
<h3 data-number="1.5.1" class="anchored" data-anchor-id="algoritmo-de-louvain"><span class="header-section-number">1.5.1</span> Algoritmo de Louvain</h3>
<p>Se utilizó el algoritmo de Louvain para detectar comunidades (módulos) en cada red. Este es un método de optimización de modularidad que opera en dos fases:</p>
<p><strong>Fase 1:</strong> Optimización local - Cada nodo se asigna a la comunidad de un vecino si esto aumenta la modularidad</p>
<p><strong>Fase 2:</strong> Agregación - Se construye una nueva red donde cada comunidad se convierte en un nodo</p>
<p>El proceso se repite iterativamente hasta convergencia.</p>
<p><strong>Ventajas del método:</strong> - Computacionalmente eficiente - No requiere especificar número de comunidades a priori - Detecta comunidades a múltiples escalas</p>
</section>
<section id="caracterización-de-comunidades" class="level3" data-number="1.5.2">
<h3 data-number="1.5.2" class="anchored" data-anchor-id="caracterización-de-comunidades"><span class="header-section-number">1.5.2</span> Caracterización de Comunidades</h3>
<p>Para cada comunidad detectada se calculó:</p>
<ol type="1">
<li><strong>Tamaño:</strong> Número de nodos</li>
<li><strong>Densidad interna:</strong> Proporción de conexiones internas vs.&nbsp;posibles</li>
<li><strong>Grado promedio:</strong> Conectividad media de los nodos</li>
<li><strong>Características principales:</strong> Los 3 nodos de mayor grado en la comunidad</li>
<li><strong>Score de estabilidad:</strong> Combinación ponderada de densidad interna y grado medio</li>
</ol>
<hr>
</section>
</section>
<section id="análisis-de-k-core" class="level2" data-number="1.6">
<h2 data-number="1.6" class="anchored" data-anchor-id="análisis-de-k-core"><span class="header-section-number">1.6</span> Análisis de K-Core</h2>
<section id="definición" class="level3" data-number="1.6.1">
<h3 data-number="1.6.1" class="anchored" data-anchor-id="definición"><span class="header-section-number">1.6.1</span> Definición</h3>
<p>El <span class="math inline">\(k\)</span>-core de una red es el subgrafo maximal donde cada nodo tiene al menos <span class="math inline">\(k\)</span> conexiones dentro del subgrafo.</p>
</section>
<section id="procedimiento" class="level3" data-number="1.6.2">
<h3 data-number="1.6.2" class="anchored" data-anchor-id="procedimiento"><span class="header-section-number">1.6.2</span> Procedimiento</h3>
<ol type="1">
<li>Para cada nodo, se calculó su <strong>coreness</strong> (<span class="math inline">\(k\)</span> máximo para el cual pertenece al <span class="math inline">\(k\)</span>-core)</li>
<li>Se identificaron nodos en el núcleo más denso (mayor <span class="math inline">\(k\)</span>-core)</li>
<li>Se visualizaron capas de <span class="math inline">\(k\)</span>-core como indicador de jerarquía en la red</li>
</ol>
</section>
<section id="interpretación-biológica" class="level3" data-number="1.6.3">
<h3 data-number="1.6.3" class="anchored" data-anchor-id="interpretación-biológica"><span class="header-section-number">1.6.3</span> Interpretación Biológica</h3>
<ul>
<li><strong>Alto <span class="math inline">\(k\)</span>-core:</strong> Características en el “núcleo” del perfil de cáncer de mama</li>
<li><strong>Bajo <span class="math inline">\(k\)</span>-core:</strong> Características periféricas o específicas de subtipos</li>
<li>Revela <strong>jerarquía de importancia</strong> clínica</li>
</ul>
<hr>
</section>
</section>
<section id="identificación-de-nodos-hub-y-puente" class="level2" data-number="1.7">
<h2 data-number="1.7" class="anchored" data-anchor-id="identificación-de-nodos-hub-y-puente"><span class="header-section-number">1.7</span> Identificación de Nodos Hub y Puente</h2>
<section id="nodos-hub" class="level3" data-number="1.7.1">
<h3 data-number="1.7.1" class="anchored" data-anchor-id="nodos-hub"><span class="header-section-number">1.7.1</span> Nodos Hub</h3>
<p>Definidos como nodos que cumplen simultáneamente: - Grado <span class="math inline">\(\geq\)</span> percentil 90 - Eigenvector centrality <span class="math inline">\(\geq\)</span> percentil 90</p>
<p><strong>Interpretación:</strong> Características centrales que co-ocurren con muchas otras características importantes.</p>
</section>
<section id="nodos-puente-gateway-nodes" class="level3" data-number="1.7.2">
<h3 data-number="1.7.2" class="anchored" data-anchor-id="nodos-puente-gateway-nodes"><span class="header-section-number">1.7.2</span> Nodos Puente (Gateway Nodes)</h3>
<p>Para cada nodo <span class="math inline">\(i\)</span>, se calculó el <strong>Gateway Score (GS)</strong>:</p>
<p><span class="math display">\[GS_i = N_{\text{communities}} \times P_{\text{external}} \times k_i\]</span></p>
<p>Donde: - <span class="math inline">\(N_{\text{communities}}\)</span> = número de comunidades diferentes a las que se conecta - <span class="math inline">\(P_{\text{external}}\)</span> = proporción de vecinos en otras comunidades - <span class="math inline">\(k_i\)</span> = grado del nodo</p>
<p><strong>Interpretación:</strong> - Alto GS indica nodos que conectan diferentes módulos funcionales - Potenciales características de transición entre subtipos - Relevantes para entender heterogeneidad intratumoral</p>
<hr>
</section>
</section>
<section id="análisis-comparativos" class="level2" data-number="1.8">
<h2 data-number="1.8" class="anchored" data-anchor-id="análisis-comparativos"><span class="header-section-number">1.8</span> Análisis Comparativos</h2>
<section id="comparación-entre-sistemas-de-clasificación" class="level3" data-number="1.8.1">
<h3 data-number="1.8.1" class="anchored" data-anchor-id="comparación-entre-sistemas-de-clasificación"><span class="header-section-number">1.8.1</span> Comparación Entre Sistemas de Clasificación</h3>
<p>Se compararon las tres redes (PAM50, 3-Gene, IntCluster) en términos de: - Métricas topológicas globales - Número y tamaño de comunidades - Características más centrales</p>
</section>
<section id="análisis-de-subgrupos-específicos" class="level3" data-number="1.8.2">
<h3 data-number="1.8.2" class="anchored" data-anchor-id="análisis-de-subgrupos-específicos"><span class="header-section-number">1.8.2</span> Análisis de Subgrupos Específicos</h3>
<p><strong>Luminal A vs.&nbsp;Luminal B:</strong> - Comparación de frecuencias de características clínicas - Identificación de marcadores diferenciales - Análisis de patrones de tratamiento</p>
<p><strong>Triple Negativo vs.&nbsp;HER2+:</strong> - Comparación de agresividad (grado, estadio) - Patrones de tratamiento - Desenlaces clínicos (supervivencia, recurrencia)</p>
</section>
<section id="análisis-de-supervivencia-y-recurrencia" class="level3" data-number="1.8.3">
<h3 data-number="1.8.3" class="anchored" data-anchor-id="análisis-de-supervivencia-y-recurrencia"><span class="header-section-number">1.8.3</span> Análisis de Supervivencia y Recurrencia</h3>
<p>Para cada subtipo molecular se calculó: - Distribución de estado vital (vivo/fallecido) - Tasa de recurrencia - Tablas de contingencia con pruebas de significancia</p>
<hr>
</section>
</section>
<section id="software-y-herramientas-computacionales" class="level2" data-number="1.9">
<h2 data-number="1.9" class="anchored" data-anchor-id="software-y-herramientas-computacionales"><span class="header-section-number">1.9</span> Software y Herramientas Computacionales</h2>
<section id="lenguaje-y-entorno" class="level3" data-number="1.9.1">
<h3 data-number="1.9.1" class="anchored" data-anchor-id="lenguaje-y-entorno"><span class="header-section-number">1.9.1</span> Lenguaje y Entorno</h3>
<ul>
<li><strong>R version 4.3+</strong></li>
<li><strong>RStudio</strong> para análisis interactivo</li>
<li>Documentos R Markdown (.Rmd) para reproducibilidad</li>
</ul>
</section>
<section id="paquetes-utilizados" class="level3" data-number="1.9.2">
<h3 data-number="1.9.2" class="anchored" data-anchor-id="paquetes-utilizados"><span class="header-section-number">1.9.2</span> Paquetes Utilizados</h3>
<p><strong>Análisis de Redes:</strong> - <code>igraph</code> (v1.5+): Construcción y análisis de redes - <code>ggraph</code> (v2.1+): Visualización avanzada de redes</p>
<p><strong>Manipulación de Datos:</strong> - <code>tidyverse</code> (v2.0+): Manipulación y transformación de datos - <code>dplyr</code>: Operaciones de datos - <code>tidyr</code>: Reestructuración de datos - <code>ggplot2</code>: Visualización base</p>
<p><strong>Visualización:</strong> - <code>viridis</code>: Paletas de colores accesibles - <code>RColorBrewer</code>: Paletas categóricas - <code>gridExtra</code>: Composición de gráficos múltiples - <code>scales</code>: Formateo de escalas</p>
<p><strong>Tablas Interactivas:</strong> - <code>DT</code> (v0.28+): Tablas HTML interactivas con funcionalidad de exportación</p>
</section>
<section id="parámetros-de-visualización" class="level3" data-number="1.9.3">
<h3 data-number="1.9.3" class="anchored" data-anchor-id="parámetros-de-visualización"><span class="header-section-number">1.9.3</span> Parámetros de Visualización</h3>
<ul>
<li>Resolución de figuras: 300 DPI</li>
<li>Formato de salida: PNG con fondo blanco</li>
<li>Dimensiones estándar: 14” × 12” para redes, 14” × 8” para comparaciones</li>
<li>Layouts de red: Fruchterman-Reingold con seed fijo para reproducibilidad</li>
</ul>
<hr>
</section>
</section>
<section id="consideraciones-éticas-y-de-datos" class="level2" data-number="1.10">
<h2 data-number="1.10" class="anchored" data-anchor-id="consideraciones-éticas-y-de-datos"><span class="header-section-number">1.10</span> Consideraciones Éticas y de Datos</h2>
<section id="datos-públicos" class="level3" data-number="1.10.1">
<h3 data-number="1.10.1" class="anchored" data-anchor-id="datos-públicos"><span class="header-section-number">1.10.1</span> Datos Públicos</h3>
<p>El dataset METABRIC es de acceso público y ha sido anonimizado completamente: - No contiene identificadores personales - Cumple con regulaciones de privacidad (GDPR, HIPAA) - Datos disponibles en repositorios científicos reconocidos</p>
</section>
<section id="reproducibilidad" class="level3" data-number="1.10.2">
<h3 data-number="1.10.2" class="anchored" data-anchor-id="reproducibilidad"><span class="header-section-number">1.10.2</span> Reproducibilidad</h3>
<p>Todo el análisis es completamente reproducible: - Código documentado en R Markdown - Seeds fijos para procesos estocásticos - Versionado de paquetes especificado - Datos de entrada y salida preservados</p>
</section>
<section id="limitaciones-metodológicas" class="level3" data-number="1.10.3">
<h3 data-number="1.10.3" class="anchored" data-anchor-id="limitaciones-metodológicas"><span class="header-section-number">1.10.3</span> Limitaciones Metodológicas</h3>
<p><strong>Sesgos de Selección:</strong> - Cohorte retrospectiva (no prospectiva) - Pacientes de sistema de salud británico (generalizabilidad limitada) - Sesgo de supervivencia en datos de seguimiento</p>
<p><strong>Limitaciones del Análisis de Redes:</strong> - Redes no ponderadas pierden información cuantitativa - Filtrado de aristas (percentil 75) es arbitrario pero estándar - Correlación ≠ causalidad</p>
<p><strong>Limitaciones de Datos:</strong> - Datos faltantes (NA) manejados como categoría “Unknown” - Variables numéricas discretizadas (pérdida de resolución) - Clasificaciones moleculares no siempre concordantes</p>
<hr>
</section>
</section>
<section id="validación-y-control-de-calidad" class="level2" data-number="1.11">
<h2 data-number="1.11" class="anchored" data-anchor-id="validación-y-control-de-calidad"><span class="header-section-number">1.11</span> Validación y Control de Calidad</h2>
<section id="validación-de-redes" class="level3" data-number="1.11.1">
<h3 data-number="1.11.1" class="anchored" data-anchor-id="validación-de-redes"><span class="header-section-number">1.11.1</span> Validación de Redes</h3>
<ul>
<li>Comparación de métricas entre redes para detectar anomalías</li>
<li>Verificación de distribución de grados (evitar hubs artificiales)</li>
<li>Inspección manual de subgrafos críticos</li>
</ul>
</section>
<section id="control-de-calidad-de-datos" class="level3" data-number="1.11.2">
<h3 data-number="1.11.2" class="anchored" data-anchor-id="control-de-calidad-de-datos"><span class="header-section-number">1.11.2</span> Control de Calidad de Datos</h3>
<ul>
<li>Verificación de rangos de variables numéricas</li>
<li>Conteo de valores faltantes por variable</li>
<li>Detección de outliers en métricas de centralidad</li>
</ul>
</section>
<section id="sensibilidad-de-parámetros" class="level3" data-number="1.11.3">
<h3 data-number="1.11.3" class="anchored" data-anchor-id="sensibilidad-de-parámetros"><span class="header-section-number">1.11.3</span> Sensibilidad de Parámetros</h3>
<p>Se evaluó sensibilidad a: - Umbral de filtrado de aristas (percentiles 70, 75, 80) - Métodos de normalización alternativos - Algoritmos de detección de comunidades (Louvain vs.&nbsp;Walktrap)</p>
</section>
</section>
<section id="criterios-de-interpretación" class="level2" data-number="1.12">
<h2 data-number="1.12" class="anchored" data-anchor-id="criterios-de-interpretación"><span class="header-section-number">1.12</span> Criterios de Interpretación</h2>
<section id="significancia-de-métricas" class="level3" data-number="1.12.1">
<h3 data-number="1.12.1" class="anchored" data-anchor-id="significancia-de-métricas"><span class="header-section-number">1.12.1</span> Significancia de Métricas</h3>
<p><strong>Densidad:</strong> - <span class="math inline">\(D &lt; 0.1\)</span>: Red dispersa - <span class="math inline">\(0.1 \leq D \leq 0.3\)</span>: Densidad moderada - <span class="math inline">\(D &gt; 0.3\)</span>: Red densa</p>
<p><strong>Modularidad:</strong> - <span class="math inline">\(Q &lt; 0.3\)</span>: Sin estructura comunitaria clara - <span class="math inline">\(0.3 \leq Q \leq 0.7\)</span>: Estructura comunitaria moderada - <span class="math inline">\(Q &gt; 0.7\)</span>: Comunidades muy definidas</p>
<p><strong>Grado de Nodo:</strong> - Alto: Característica con múltiples asociaciones - Bajo: Característica específica o poco conectada</p>
</section>
<section id="umbrales-de-centralidad" class="level3" data-number="1.12.2">
<h3 data-number="1.12.2" class="anchored" data-anchor-id="umbrales-de-centralidad"><span class="header-section-number">1.12.2</span> Umbrales de Centralidad</h3>
<p>Se consideraron nodos “importantes” aquellos en: - Percentil <span class="math inline">\(\geq 90\)</span> de grado - Percentil <span class="math inline">\(\geq 90\)</span> de eigenvector - Percentil <span class="math inline">\(\geq 80\)</span> de betweenness - Top 10 de gateway score</p>
</section>
<section id="relevancia-clínica" class="level3" data-number="1.12.3">
<h3 data-number="1.12.3" class="anchored" data-anchor-id="relevancia-clínica"><span class="header-section-number">1.12.3</span> Relevancia Clínica</h3>
<p>Un hallazgo se consideró clínicamente relevante si: 1. Era reproducible en múltiples redes 2. Tenía soporte en literatura biomédica 3. Mostraba asociación con desenlaces clínicos 4. Era accionable desde perspectiva terapéutica</p>
<hr>
</section>
</section>
<section id="referencias-metodológicas" class="level2" data-number="1.13">
<h2 data-number="1.13" class="anchored" data-anchor-id="referencias-metodológicas"><span class="header-section-number">1.13</span> Referencias Metodológicas</h2>
<p><strong>Análisis de Redes:</strong> - Barabási AL, et al.&nbsp;Network medicine: a network-based approach to human disease. Nat Rev Genet. 2011;12(1):56-68. - Newman MEJ. Networks: An Introduction. Oxford University Press; 2010. - Blondel VD, et al.&nbsp;Fast unfolding of communities in large networks. J Stat Mech. 2008;P10008.</p>
<p><strong>Cáncer de Mama:</strong> - Perou CM, et al.&nbsp;Molecular portraits of human breast tumours. Nature. 2000;406:747-752. - Curtis C, et al.&nbsp;The genomic and transcriptomic architecture of 2,000 breast tumours reveals novel subgroups. Nature. 2012;486:346-352.</p>
<p><strong>Software:</strong> - R Core Team. R: A language and environment for statistical computing. R Foundation for Statistical Computing; 2023. - Csardi G, Nepusz T. The igraph software package for complex network research. InterJournal Complex Systems. 2006;1695.</p>
</section>
</section>
<section id="resultados" class="level1" data-number="2">
<h1 data-number="2"><span class="header-section-number">2</span> Resultados</h1>
<section id="exploración-de-clasificaciones-génicas" class="level2" data-number="2.1">
<h2 data-number="2.1" class="anchored" data-anchor-id="exploración-de-clasificaciones-génicas"><span class="header-section-number">2.1</span> Exploración de Clasificaciones Génicas</h2>
<section id="distribución-de-subtipos-moleculares" class="level3" data-number="2.1.1">
<h3 data-number="2.1.1" class="anchored" data-anchor-id="distribución-de-subtipos-moleculares"><span class="header-section-number">2.1.1</span> Distribución de Subtipos Moleculares</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="co"># PAM50 + Claudin-low</span></span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a>pam50_dist <span class="ot">&lt;-</span> datos <span class="sc">%&gt;%</span></span>
<span id="cb3-3"><a href="#cb3-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(Pam50...Claudin.low.subtype) <span class="sc">%&gt;%</span></span>
<span id="cb3-4"><a href="#cb3-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(Pam50...Claudin.low.subtype)) <span class="sc">%&gt;%</span></span>
<span id="cb3-5"><a href="#cb3-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(n)) <span class="sc">%&gt;%</span></span>
<span id="cb3-6"><a href="#cb3-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb3-7"><a href="#cb3-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">Porcentaje =</span> <span class="fu">round</span>(n <span class="sc">/</span> <span class="fu">sum</span>(n) <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>),</span>
<span id="cb3-8"><a href="#cb3-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Subtipo =</span> Pam50...Claudin.low.subtype</span>
<span id="cb3-9"><a href="#cb3-9" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb3-10"><a href="#cb3-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-11"><a href="#cb3-11" aria-hidden="true" tabindex="-1"></a><span class="co"># 3-Gene classifier</span></span>
<span id="cb3-12"><a href="#cb3-12" aria-hidden="true" tabindex="-1"></a>gene3_dist <span class="ot">&lt;-</span> datos <span class="sc">%&gt;%</span></span>
<span id="cb3-13"><a href="#cb3-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(X3.Gene.classifier.subtype) <span class="sc">%&gt;%</span></span>
<span id="cb3-14"><a href="#cb3-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(X3.Gene.classifier.subtype)) <span class="sc">%&gt;%</span></span>
<span id="cb3-15"><a href="#cb3-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(n)) <span class="sc">%&gt;%</span></span>
<span id="cb3-16"><a href="#cb3-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb3-17"><a href="#cb3-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">Porcentaje =</span> <span class="fu">round</span>(n <span class="sc">/</span> <span class="fu">sum</span>(n) <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>),</span>
<span id="cb3-18"><a href="#cb3-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">Subtipo =</span> X3.Gene.classifier.subtype</span>
<span id="cb3-19"><a href="#cb3-19" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb3-20"><a href="#cb3-20" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb3-21"><a href="#cb3-21" aria-hidden="true" tabindex="-1"></a><span class="co"># Integrative Cluster</span></span>
<span id="cb3-22"><a href="#cb3-22" aria-hidden="true" tabindex="-1"></a>intclust_dist <span class="ot">&lt;-</span> datos <span class="sc">%&gt;%</span></span>
<span id="cb3-23"><a href="#cb3-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">count</span>(Integrative.Cluster) <span class="sc">%&gt;%</span></span>
<span id="cb3-24"><a href="#cb3-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(Integrative.Cluster)) <span class="sc">%&gt;%</span></span>
<span id="cb3-25"><a href="#cb3-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(n)) <span class="sc">%&gt;%</span></span>
<span id="cb3-26"><a href="#cb3-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb3-27"><a href="#cb3-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">Porcentaje =</span> <span class="fu">round</span>(n <span class="sc">/</span> <span class="fu">sum</span>(n) <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>),</span>
<span id="cb3-28"><a href="#cb3-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">Cluster =</span> Integrative.Cluster</span>
<span id="cb3-29"><a href="#cb3-29" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<section id="tabla-distribución-pam50-claudin-low" class="level4" data-number="2.1.1.1">
<h4 data-number="2.1.1.1" class="anchored" data-anchor-id="tabla-distribución-pam50-claudin-low"><span class="header-section-number">2.1.1.1</span> Tabla: Distribución PAM50 + Claudin-low</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb4"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb4-2"><a href="#cb4-2" aria-hidden="true" tabindex="-1"></a>  pam50_dist <span class="sc">%&gt;%</span> <span class="fu">select</span>(Subtipo, n, Porcentaje),</span>
<span id="cb4-3"><a href="#cb4-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Subtipo PAM50"</span>, <span class="st">"N° Casos"</span>, <span class="st">"Porcentaje (%)"</span>),</span>
<span id="cb4-4"><a href="#cb4-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb4-5"><a href="#cb4-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb4-6"><a href="#cb4-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb4-7"><a href="#cb4-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb4-8"><a href="#cb4-8" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb4-9"><a href="#cb4-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb4-10"><a href="#cb4-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Distribución de subtipos moleculares PAM50 + Claudin-low"</span></span>
<span id="cb4-11"><a href="#cb4-11" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb4-12"><a href="#cb4-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="st">'Porcentaje'</span>, <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-6560649287804b38a395" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-6560649287804b38a395">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Distribución de subtipos moleculares PAM50 + Claudin-low<\/caption>","data":[["1","2","3","4","5","6","7"],["LumA","LumB","Her2","claudin-low","Basal","Normal","NC"],[700,475,224,218,209,148,6],[35.35,23.99,11.31,11.01,10.56,7.47,0.3]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Subtipo PAM50<\/th>\n      <th>N° Casos<\/th>\n      <th>Porcentaje (%)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Subtipo","targets":1},{"name":"n","targets":2},{"name":"Porcentaje","targets":3}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="tabla-distribución-3-gene-classifier" class="level4" data-number="2.1.1.2">
<h4 data-number="2.1.1.2" class="anchored" data-anchor-id="tabla-distribución-3-gene-classifier"><span class="header-section-number">2.1.1.2</span> Tabla: Distribución 3-Gene Classifier</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb5"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb5-2"><a href="#cb5-2" aria-hidden="true" tabindex="-1"></a>  gene3_dist <span class="sc">%&gt;%</span> <span class="fu">select</span>(Subtipo, n, Porcentaje),</span>
<span id="cb5-3"><a href="#cb5-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Subtipo 3-Gene"</span>, <span class="st">"N° Casos"</span>, <span class="st">"Porcentaje (%)"</span>),</span>
<span id="cb5-4"><a href="#cb5-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb5-5"><a href="#cb5-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb5-6"><a href="#cb5-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb5-7"><a href="#cb5-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb5-8"><a href="#cb5-8" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb5-9"><a href="#cb5-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb5-10"><a href="#cb5-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Distribución de subtipos según clasificador de 3 genes"</span></span>
<span id="cb5-11"><a href="#cb5-11" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb5-12"><a href="#cb5-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="st">'Porcentaje'</span>, <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-358fb0b9b65ca20cef46" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-358fb0b9b65ca20cef46">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Distribución de subtipos según clasificador de 3 genes<\/caption>","data":[["1","2","3","4"],["ER+/HER2- Low Prolif","ER+/HER2- High Prolif","ER-/HER2-","HER2+"],[640,617,309,198],[36.28,34.98,17.52,11.22]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Subtipo 3-Gene<\/th>\n      <th>N° Casos<\/th>\n      <th>Porcentaje (%)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Subtipo","targets":1},{"name":"n","targets":2},{"name":"Porcentaje","targets":3}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="tabla-distribución-integrative-cluster" class="level4" data-number="2.1.1.3">
<h4 data-number="2.1.1.3" class="anchored" data-anchor-id="tabla-distribución-integrative-cluster"><span class="header-section-number">2.1.1.3</span> Tabla: Distribución Integrative Cluster</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb6"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a>  intclust_dist <span class="sc">%&gt;%</span> <span class="fu">select</span>(Cluster, n, Porcentaje),</span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Cluster Integrativo"</span>, <span class="st">"N° Casos"</span>, <span class="st">"Porcentaje (%)"</span>),</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb6-5"><a href="#cb6-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb6-6"><a href="#cb6-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb6-7"><a href="#cb6-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb6-8"><a href="#cb6-8" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb6-9"><a href="#cb6-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb6-10"><a href="#cb6-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Distribución de clusters integrativos"</span></span>
<span id="cb6-11"><a href="#cb6-11" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb6-12"><a href="#cb6-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="st">'Porcentaje'</span>, <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-84bfd4ded55bbf9317fd" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-84bfd4ded55bbf9317fd">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Distribución de clusters integrativos<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11"],["8","3","4ER+","10","5","7","9","1","6","4ER-","2"],[299,290,260,226,190,190,146,139,85,83,72],[15.1,14.65,13.13,11.41,9.6,9.6,7.37,7.02,4.29,4.19,3.64]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Cluster Integrativo<\/th>\n      <th>N° Casos<\/th>\n      <th>Porcentaje (%)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Cluster","targets":1},{"name":"n","targets":2},{"name":"Porcentaje","targets":3}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="visualización-de-distribuciones" class="level3" data-number="2.1.2">
<h3 data-number="2.1.2" class="anchored" data-anchor-id="visualización-de-distribuciones"><span class="header-section-number">2.1.2</span> Visualización de Distribuciones</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a>p1 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(pam50_dist, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Subtipo, n), <span class="at">y =</span> n, <span class="at">fill =</span> Subtipo)) <span class="sc">+</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb7-3"><a href="#cb7-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> <span class="fu">paste0</span>(n, <span class="st">" ("</span>, Porcentaje, <span class="st">"%)"</span>)), </span>
<span id="cb7-4"><a href="#cb7-4" aria-hidden="true" tabindex="-1"></a>            <span class="at">hjust =</span> <span class="sc">-</span><span class="fl">0.1</span>, <span class="at">size =</span> <span class="fl">3.5</span>) <span class="sc">+</span></span>
<span id="cb7-5"><a href="#cb7-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>() <span class="sc">+</span></span>
<span id="cb7-6"><a href="#cb7-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"plasma"</span>) <span class="sc">+</span></span>
<span id="cb7-7"><a href="#cb7-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb7-8"><a href="#cb7-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Distribución de Subtipos PAM50 + Claudin-low"</span>,</span>
<span id="cb7-9"><a href="#cb7-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">"Subtipo Molecular"</span>,</span>
<span id="cb7-10"><a href="#cb7-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Número de Casos"</span></span>
<span id="cb7-11"><a href="#cb7-11" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb7-12"><a href="#cb7-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb7-13"><a href="#cb7-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">expand =</span> <span class="fu">expansion</span>(<span class="at">mult =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.15</span>)))</span>
<span id="cb7-14"><a href="#cb7-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-15"><a href="#cb7-15" aria-hidden="true" tabindex="-1"></a>p2 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(gene3_dist, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Subtipo, n), <span class="at">y =</span> n, <span class="at">fill =</span> Subtipo)) <span class="sc">+</span></span>
<span id="cb7-16"><a href="#cb7-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb7-17"><a href="#cb7-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> <span class="fu">paste0</span>(n, <span class="st">" ("</span>, Porcentaje, <span class="st">"%)"</span>)), </span>
<span id="cb7-18"><a href="#cb7-18" aria-hidden="true" tabindex="-1"></a>            <span class="at">hjust =</span> <span class="sc">-</span><span class="fl">0.1</span>, <span class="at">size =</span> <span class="fl">3.5</span>) <span class="sc">+</span></span>
<span id="cb7-19"><a href="#cb7-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>() <span class="sc">+</span></span>
<span id="cb7-20"><a href="#cb7-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"mako"</span>) <span class="sc">+</span></span>
<span id="cb7-21"><a href="#cb7-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb7-22"><a href="#cb7-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Distribución de Subtipos 3-Gene Classifier"</span>,</span>
<span id="cb7-23"><a href="#cb7-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">"Subtipo"</span>,</span>
<span id="cb7-24"><a href="#cb7-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Número de Casos"</span></span>
<span id="cb7-25"><a href="#cb7-25" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb7-26"><a href="#cb7-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb7-27"><a href="#cb7-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">expand =</span> <span class="fu">expansion</span>(<span class="at">mult =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.15</span>)))</span>
<span id="cb7-28"><a href="#cb7-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-29"><a href="#cb7-29" aria-hidden="true" tabindex="-1"></a>p3 <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(intclust_dist, <span class="fu">aes</span>(<span class="at">x =</span> <span class="fu">reorder</span>(Cluster, n), <span class="at">y =</span> n, <span class="at">fill =</span> Cluster)) <span class="sc">+</span></span>
<span id="cb7-30"><a href="#cb7-30" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb7-31"><a href="#cb7-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> <span class="fu">paste0</span>(n, <span class="st">" ("</span>, Porcentaje, <span class="st">"%)"</span>)), </span>
<span id="cb7-32"><a href="#cb7-32" aria-hidden="true" tabindex="-1"></a>            <span class="at">hjust =</span> <span class="sc">-</span><span class="fl">0.1</span>, <span class="at">size =</span> <span class="fl">3.5</span>) <span class="sc">+</span></span>
<span id="cb7-33"><a href="#cb7-33" aria-hidden="true" tabindex="-1"></a>  <span class="fu">coord_flip</span>() <span class="sc">+</span></span>
<span id="cb7-34"><a href="#cb7-34" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"rocket"</span>) <span class="sc">+</span></span>
<span id="cb7-35"><a href="#cb7-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb7-36"><a href="#cb7-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Distribución de Clusters Integrativos"</span>,</span>
<span id="cb7-37"><a href="#cb7-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">"Cluster"</span>,</span>
<span id="cb7-38"><a href="#cb7-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Número de Casos"</span></span>
<span id="cb7-39"><a href="#cb7-39" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb7-40"><a href="#cb7-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(<span class="at">legend.position =</span> <span class="st">"none"</span>) <span class="sc">+</span></span>
<span id="cb7-41"><a href="#cb7-41" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_y_continuous</span>(<span class="at">expand =</span> <span class="fu">expansion</span>(<span class="at">mult =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="fl">0.15</span>)))</span>
<span id="cb7-42"><a href="#cb7-42" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb7-43"><a href="#cb7-43" aria-hidden="true" tabindex="-1"></a><span class="fu">grid.arrange</span>(p1, p2, p3, <span class="at">ncol =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/viz_distribuciones-1.png" class="img-fluid figure-img" width="1344"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb8"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"figuras_redes/distribucion_subtipos.png"</span>, </span>
<span id="cb8-2"><a href="#cb8-2" aria-hidden="true" tabindex="-1"></a>       <span class="fu">arrangeGrob</span>(p1, p2, p3, <span class="at">ncol =</span> <span class="dv">1</span>),</span>
<span id="cb8-3"><a href="#cb8-3" aria-hidden="true" tabindex="-1"></a>       <span class="at">width =</span> <span class="dv">14</span>, <span class="at">height =</span> <span class="dv">10</span>, <span class="at">dpi =</span> <span class="dv">300</span>, <span class="at">bg =</span> <span class="st">"white"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
</section>
<section id="preparación-de-datos-para-análisis-de-redes" class="level2" data-number="2.2">
<h2 data-number="2.2" class="anchored" data-anchor-id="preparación-de-datos-para-análisis-de-redes"><span class="header-section-number">2.2</span> Preparación de Datos para Análisis de Redes</h2>
<section id="selección-y-limpieza-de-variables-clínicas" class="level3" data-number="2.2.1">
<h3 data-number="2.2.1" class="anchored" data-anchor-id="selección-y-limpieza-de-variables-clínicas"><span class="header-section-number">2.2.1</span> Selección y Limpieza de Variables Clínicas</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb9"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Seleccionar variables clínicas relevantes</span></span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a>datos_red <span class="ot">&lt;-</span> datos <span class="sc">%&gt;%</span></span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(</span>
<span id="cb9-4"><a href="#cb9-4" aria-hidden="true" tabindex="-1"></a>    Patient.ID,</span>
<span id="cb9-5"><a href="#cb9-5" aria-hidden="true" tabindex="-1"></a>    Pam50...Claudin.low.subtype,</span>
<span id="cb9-6"><a href="#cb9-6" aria-hidden="true" tabindex="-1"></a>    X3.Gene.classifier.subtype,</span>
<span id="cb9-7"><a href="#cb9-7" aria-hidden="true" tabindex="-1"></a>    Integrative.Cluster,</span>
<span id="cb9-8"><a href="#cb9-8" aria-hidden="true" tabindex="-1"></a>    ER.Status,</span>
<span id="cb9-9"><a href="#cb9-9" aria-hidden="true" tabindex="-1"></a>    PR.Status,</span>
<span id="cb9-10"><a href="#cb9-10" aria-hidden="true" tabindex="-1"></a>    HER2.Status,</span>
<span id="cb9-11"><a href="#cb9-11" aria-hidden="true" tabindex="-1"></a>    Neoplasm.Histologic.Grade,</span>
<span id="cb9-12"><a href="#cb9-12" aria-hidden="true" tabindex="-1"></a>    Tumor.Stage,</span>
<span id="cb9-13"><a href="#cb9-13" aria-hidden="true" tabindex="-1"></a>    Cellularity,</span>
<span id="cb9-14"><a href="#cb9-14" aria-hidden="true" tabindex="-1"></a>    Chemotherapy,</span>
<span id="cb9-15"><a href="#cb9-15" aria-hidden="true" tabindex="-1"></a>    Hormone.Therapy,</span>
<span id="cb9-16"><a href="#cb9-16" aria-hidden="true" tabindex="-1"></a>    Radio.Therapy,</span>
<span id="cb9-17"><a href="#cb9-17" aria-hidden="true" tabindex="-1"></a>    Inferred.Menopausal.State,</span>
<span id="cb9-18"><a href="#cb9-18" aria-hidden="true" tabindex="-1"></a>    Cancer.Type.Detailed,</span>
<span id="cb9-19"><a href="#cb9-19" aria-hidden="true" tabindex="-1"></a>    Overall.Survival.Status,</span>
<span id="cb9-20"><a href="#cb9-20" aria-hidden="true" tabindex="-1"></a>    Relapse.Free.Status,</span>
<span id="cb9-21"><a href="#cb9-21" aria-hidden="true" tabindex="-1"></a>    Type.of.Breast.Surgery,</span>
<span id="cb9-22"><a href="#cb9-22" aria-hidden="true" tabindex="-1"></a>    Primary.Tumor.Laterality,</span>
<span id="cb9-23"><a href="#cb9-23" aria-hidden="true" tabindex="-1"></a>    Tumor.Other.Histologic.Subtype,</span>
<span id="cb9-24"><a href="#cb9-24" aria-hidden="true" tabindex="-1"></a>    Age.at.Diagnosis,</span>
<span id="cb9-25"><a href="#cb9-25" aria-hidden="true" tabindex="-1"></a>    Tumor.Size,</span>
<span id="cb9-26"><a href="#cb9-26" aria-hidden="true" tabindex="-1"></a>    Lymph.nodes.examined.positive</span>
<span id="cb9-27"><a href="#cb9-27" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb9-28"><a href="#cb9-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">rename</span>(</span>
<span id="cb9-29"><a href="#cb9-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">Patient =</span> Patient.ID,</span>
<span id="cb9-30"><a href="#cb9-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">PAM50 =</span> Pam50...Claudin.low.subtype,</span>
<span id="cb9-31"><a href="#cb9-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">Gene3 =</span> X3.Gene.classifier.subtype,</span>
<span id="cb9-32"><a href="#cb9-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">IntCluster =</span> Integrative.Cluster,</span>
<span id="cb9-33"><a href="#cb9-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">ER =</span> ER.Status,</span>
<span id="cb9-34"><a href="#cb9-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">PR =</span> PR.Status,</span>
<span id="cb9-35"><a href="#cb9-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">HER2 =</span> HER2.Status,</span>
<span id="cb9-36"><a href="#cb9-36" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grade =</span> Neoplasm.Histologic.Grade,</span>
<span id="cb9-37"><a href="#cb9-37" aria-hidden="true" tabindex="-1"></a>    <span class="at">Stage =</span> Tumor.Stage,</span>
<span id="cb9-38"><a href="#cb9-38" aria-hidden="true" tabindex="-1"></a>    <span class="at">Cell =</span> Cellularity,</span>
<span id="cb9-39"><a href="#cb9-39" aria-hidden="true" tabindex="-1"></a>    <span class="at">Chemo =</span> Chemotherapy,</span>
<span id="cb9-40"><a href="#cb9-40" aria-hidden="true" tabindex="-1"></a>    <span class="at">Hormone =</span> Hormone.Therapy,</span>
<span id="cb9-41"><a href="#cb9-41" aria-hidden="true" tabindex="-1"></a>    <span class="at">Radio =</span> Radio.Therapy,</span>
<span id="cb9-42"><a href="#cb9-42" aria-hidden="true" tabindex="-1"></a>    <span class="at">Menopause =</span> Inferred.Menopausal.State,</span>
<span id="cb9-43"><a href="#cb9-43" aria-hidden="true" tabindex="-1"></a>    <span class="at">CancerType =</span> Cancer.Type.Detailed,</span>
<span id="cb9-44"><a href="#cb9-44" aria-hidden="true" tabindex="-1"></a>    <span class="at">Survival =</span> Overall.Survival.Status,</span>
<span id="cb9-45"><a href="#cb9-45" aria-hidden="true" tabindex="-1"></a>    <span class="at">Relapse =</span> Relapse.Free.Status,</span>
<span id="cb9-46"><a href="#cb9-46" aria-hidden="true" tabindex="-1"></a>    <span class="at">Surgery =</span> Type.of.Breast.Surgery,</span>
<span id="cb9-47"><a href="#cb9-47" aria-hidden="true" tabindex="-1"></a>    <span class="at">Laterality =</span> Primary.Tumor.Laterality,</span>
<span id="cb9-48"><a href="#cb9-48" aria-hidden="true" tabindex="-1"></a>    <span class="at">HistoSubtype =</span> Tumor.Other.Histologic.Subtype,</span>
<span id="cb9-49"><a href="#cb9-49" aria-hidden="true" tabindex="-1"></a>    <span class="at">Age =</span> Age.at.Diagnosis,</span>
<span id="cb9-50"><a href="#cb9-50" aria-hidden="true" tabindex="-1"></a>    <span class="at">TumorSize =</span> Tumor.Size,</span>
<span id="cb9-51"><a href="#cb9-51" aria-hidden="true" tabindex="-1"></a>    <span class="at">LymphNodes =</span> Lymph.nodes.examined.positive</span>
<span id="cb9-52"><a href="#cb9-52" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb9-53"><a href="#cb9-53" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb9-54"><a href="#cb9-54" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Convertir a categorías</span></span>
<span id="cb9-55"><a href="#cb9-55" aria-hidden="true" tabindex="-1"></a>    <span class="at">ER =</span> <span class="fu">factor</span>(ER),</span>
<span id="cb9-56"><a href="#cb9-56" aria-hidden="true" tabindex="-1"></a>    <span class="at">PR =</span> <span class="fu">factor</span>(PR),</span>
<span id="cb9-57"><a href="#cb9-57" aria-hidden="true" tabindex="-1"></a>    <span class="at">HER2 =</span> <span class="fu">factor</span>(HER2),</span>
<span id="cb9-58"><a href="#cb9-58" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grade =</span> <span class="fu">factor</span>(Grade),</span>
<span id="cb9-59"><a href="#cb9-59" aria-hidden="true" tabindex="-1"></a>    <span class="at">Stage =</span> <span class="fu">factor</span>(Stage),</span>
<span id="cb9-60"><a href="#cb9-60" aria-hidden="true" tabindex="-1"></a>    <span class="at">Chemo =</span> <span class="fu">factor</span>(Chemo),</span>
<span id="cb9-61"><a href="#cb9-61" aria-hidden="true" tabindex="-1"></a>    <span class="at">Hormone =</span> <span class="fu">factor</span>(Hormone),</span>
<span id="cb9-62"><a href="#cb9-62" aria-hidden="true" tabindex="-1"></a>    <span class="at">Radio =</span> <span class="fu">factor</span>(Radio),</span>
<span id="cb9-63"><a href="#cb9-63" aria-hidden="true" tabindex="-1"></a>    <span class="at">Cell =</span> <span class="fu">factor</span>(Cell),</span>
<span id="cb9-64"><a href="#cb9-64" aria-hidden="true" tabindex="-1"></a>    <span class="at">Menopause =</span> <span class="fu">factor</span>(Menopause),</span>
<span id="cb9-65"><a href="#cb9-65" aria-hidden="true" tabindex="-1"></a>    <span class="at">Survival =</span> <span class="fu">factor</span>(Survival),</span>
<span id="cb9-66"><a href="#cb9-66" aria-hidden="true" tabindex="-1"></a>    <span class="at">Relapse =</span> <span class="fu">factor</span>(Relapse),</span>
<span id="cb9-67"><a href="#cb9-67" aria-hidden="true" tabindex="-1"></a>    <span class="at">Surgery =</span> <span class="fu">factor</span>(Surgery),</span>
<span id="cb9-68"><a href="#cb9-68" aria-hidden="true" tabindex="-1"></a>    <span class="at">Laterality =</span> <span class="fu">factor</span>(Laterality),</span>
<span id="cb9-69"><a href="#cb9-69" aria-hidden="true" tabindex="-1"></a>    <span class="at">HistoSubtype =</span> <span class="fu">factor</span>(HistoSubtype),</span>
<span id="cb9-70"><a href="#cb9-70" aria-hidden="true" tabindex="-1"></a>    <span class="at">CancerType =</span> <span class="fu">factor</span>(CancerType),</span>
<span id="cb9-71"><a href="#cb9-71" aria-hidden="true" tabindex="-1"></a>    <span class="co"># Categorizar variables numéricas para análisis de red</span></span>
<span id="cb9-72"><a href="#cb9-72" aria-hidden="true" tabindex="-1"></a>    <span class="at">AgeGroup =</span> <span class="fu">cut</span>(Age, <span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">40</span>, <span class="dv">50</span>, <span class="dv">60</span>, <span class="dv">70</span>, <span class="dv">100</span>), </span>
<span id="cb9-73"><a href="#cb9-73" aria-hidden="true" tabindex="-1"></a>                   <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"&lt;40"</span>, <span class="st">"40-50"</span>, <span class="st">"50-60"</span>, <span class="st">"60-70"</span>, <span class="st">"&gt;70"</span>),</span>
<span id="cb9-74"><a href="#cb9-74" aria-hidden="true" tabindex="-1"></a>                   <span class="at">include.lowest =</span> <span class="cn">TRUE</span>),</span>
<span id="cb9-75"><a href="#cb9-75" aria-hidden="true" tabindex="-1"></a>    <span class="at">SizeGroup =</span> <span class="fu">cut</span>(TumorSize, <span class="at">breaks =</span> <span class="fu">c</span>(<span class="dv">0</span>, <span class="dv">20</span>, <span class="dv">30</span>, <span class="dv">50</span>, <span class="dv">200</span>),</span>
<span id="cb9-76"><a href="#cb9-76" aria-hidden="true" tabindex="-1"></a>                    <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"Small(&lt;20mm)"</span>, <span class="st">"Medium(20-30)"</span>, <span class="st">"Large(30-50)"</span>, <span class="st">"VeryLarge(&gt;50)"</span>),</span>
<span id="cb9-77"><a href="#cb9-77" aria-hidden="true" tabindex="-1"></a>                    <span class="at">include.lowest =</span> <span class="cn">TRUE</span>),</span>
<span id="cb9-78"><a href="#cb9-78" aria-hidden="true" tabindex="-1"></a>    <span class="at">LymphGroup =</span> <span class="fu">cut</span>(LymphNodes, <span class="at">breaks =</span> <span class="fu">c</span>(<span class="sc">-</span><span class="dv">1</span>, <span class="dv">0</span>, <span class="dv">1</span>, <span class="dv">3</span>, <span class="dv">50</span>),</span>
<span id="cb9-79"><a href="#cb9-79" aria-hidden="true" tabindex="-1"></a>                     <span class="at">labels =</span> <span class="fu">c</span>(<span class="st">"N0"</span>, <span class="st">"N1"</span>, <span class="st">"N2-3"</span>, <span class="st">"N&gt;3"</span>),</span>
<span id="cb9-80"><a href="#cb9-80" aria-hidden="true" tabindex="-1"></a>                     <span class="at">include.lowest =</span> <span class="cn">TRUE</span>)</span>
<span id="cb9-81"><a href="#cb9-81" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb9-82"><a href="#cb9-82" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb9-83"><a href="#cb9-83" aria-hidden="true" tabindex="-1"></a>summary_clean <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb9-84"><a href="#cb9-84" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb9-85"><a href="#cb9-85" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total_Pacientes =</span> <span class="fu">n</span>(),</span>
<span id="cb9-86"><a href="#cb9-86" aria-hidden="true" tabindex="-1"></a>    <span class="at">PAM50_disponible =</span> <span class="fu">sum</span>(<span class="sc">!</span><span class="fu">is.na</span>(PAM50)),</span>
<span id="cb9-87"><a href="#cb9-87" aria-hidden="true" tabindex="-1"></a>    <span class="at">Gene3_disponible =</span> <span class="fu">sum</span>(<span class="sc">!</span><span class="fu">is.na</span>(Gene3)),</span>
<span id="cb9-88"><a href="#cb9-88" aria-hidden="true" tabindex="-1"></a>    <span class="at">IntCluster_disponible =</span> <span class="fu">sum</span>(<span class="sc">!</span><span class="fu">is.na</span>(IntCluster)),</span>
<span id="cb9-89"><a href="#cb9-89" aria-hidden="true" tabindex="-1"></a>    <span class="at">ER_disponible =</span> <span class="fu">sum</span>(<span class="sc">!</span><span class="fu">is.na</span>(ER)),</span>
<span id="cb9-90"><a href="#cb9-90" aria-hidden="true" tabindex="-1"></a>    <span class="at">HER2_disponible =</span> <span class="fu">sum</span>(<span class="sc">!</span><span class="fu">is.na</span>(HER2))</span>
<span id="cb9-91"><a href="#cb9-91" aria-hidden="true" tabindex="-1"></a>  )</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<p><strong>Resumen de datos procesados:</strong></p>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a>  summary_clean,</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(<span class="at">dom =</span> <span class="st">'t'</span>),</span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Disponibilidad de variables clave para análisis"</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-d1b271de98dd1b79ab54" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-d1b271de98dd1b79ab54">{"x":{"filter":"none","vertical":false,"caption":"<caption>Disponibilidad de variables clave para análisis<\/caption>","data":[["1"],[2509],[1980],[1764],[1980],[2469],[1980]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Total_Pacientes<\/th>\n      <th>PAM50_disponible<\/th>\n      <th>Gene3_disponible<\/th>\n      <th>IntCluster_disponible<\/th>\n      <th>ER_disponible<\/th>\n      <th>HER2_disponible<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","columnDefs":[{"className":"dt-right","targets":[1,2,3,4,5,6]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Total_Pacientes","targets":1},{"name":"PAM50_disponible","targets":2},{"name":"Gene3_disponible","targets":3},{"name":"IntCluster_disponible","targets":4},{"name":"ER_disponible","targets":5},{"name":"HER2_disponible","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="construcción-de-redes-de-co-ocurrencia-1" class="level2" data-number="2.3">
<h2 data-number="2.3" class="anchored" data-anchor-id="construcción-de-redes-de-co-ocurrencia-1"><span class="header-section-number">2.3</span> Construcción de Redes de Co-ocurrencia</h2>
<section id="red-por-subtipo-pam50" class="level3" data-number="2.3.1">
<h3 data-number="2.3.1" class="anchored" data-anchor-id="red-por-subtipo-pam50"><span class="header-section-number">2.3.1</span> Red por Subtipo PAM50</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar datos con PAM50 disponible</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>datos_pam50 <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(PAM50))</span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a><span class="co"># Crear matriz de características</span></span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>matriz_caracteristicas <span class="ot">&lt;-</span> datos_pam50 <span class="sc">%&gt;%</span></span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(</span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a>    PAM50, ER, PR, HER2, Grade, Stage, Cell, Chemo, </span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a>    Hormone, Radio, Menopause, Surgery, Laterality, </span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>    HistoSubtype, AgeGroup, SizeGroup, LymphGroup,</span>
<span id="cb11-11"><a href="#cb11-11" aria-hidden="true" tabindex="-1"></a>    Survival, Relapse</span>
<span id="cb11-12"><a href="#cb11-12" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb11-13"><a href="#cb11-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), as.character)) <span class="sc">%&gt;%</span></span>
<span id="cb11-14"><a href="#cb11-14" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), <span class="sc">~</span><span class="fu">replace_na</span>(., <span class="st">"Unknown"</span>)))</span>
<span id="cb11-15"><a href="#cb11-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-16"><a href="#cb11-16" aria-hidden="true" tabindex="-1"></a><span class="co"># Formato largo para análisis de red</span></span>
<span id="cb11-17"><a href="#cb11-17" aria-hidden="true" tabindex="-1"></a>datos_largo <span class="ot">&lt;-</span> matriz_caracteristicas <span class="sc">%&gt;%</span></span>
<span id="cb11-18"><a href="#cb11-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">id =</span> <span class="fu">row_number</span>()) <span class="sc">%&gt;%</span></span>
<span id="cb11-19"><a href="#cb11-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(</span>
<span id="cb11-20"><a href="#cb11-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">cols =</span> <span class="sc">-</span>id,</span>
<span id="cb11-21"><a href="#cb11-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_to =</span> <span class="st">"Variable"</span>,</span>
<span id="cb11-22"><a href="#cb11-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_to =</span> <span class="st">"Valor"</span></span>
<span id="cb11-23"><a href="#cb11-23" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb11-24"><a href="#cb11-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Caracteristica =</span> <span class="fu">paste</span>(Variable, Valor, <span class="at">sep =</span> <span class="st">": "</span>))</span>
<span id="cb11-25"><a href="#cb11-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-26"><a href="#cb11-26" aria-hidden="true" tabindex="-1"></a><span class="co"># Matriz binaria: paciente x característica</span></span>
<span id="cb11-27"><a href="#cb11-27" aria-hidden="true" tabindex="-1"></a>matriz_bin <span class="ot">&lt;-</span> datos_largo <span class="sc">%&gt;%</span></span>
<span id="cb11-28"><a href="#cb11-28" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">presente =</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span></span>
<span id="cb11-29"><a href="#cb11-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(</span>
<span id="cb11-30"><a href="#cb11-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">id_cols =</span> id,</span>
<span id="cb11-31"><a href="#cb11-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_from =</span> Caracteristica,</span>
<span id="cb11-32"><a href="#cb11-32" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_from =</span> presente,</span>
<span id="cb11-33"><a href="#cb11-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_fill =</span> <span class="dv">0</span></span>
<span id="cb11-34"><a href="#cb11-34" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb11-35"><a href="#cb11-35" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="sc">-</span>id) <span class="sc">%&gt;%</span></span>
<span id="cb11-36"><a href="#cb11-36" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.matrix</span>()</span>
<span id="cb11-37"><a href="#cb11-37" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-38"><a href="#cb11-38" aria-hidden="true" tabindex="-1"></a><span class="co"># Matriz de co-ocurrencia: t(X) %*% X</span></span>
<span id="cb11-39"><a href="#cb11-39" aria-hidden="true" tabindex="-1"></a>coocurrencia <span class="ot">&lt;-</span> <span class="fu">t</span>(matriz_bin) <span class="sc">%*%</span> matriz_bin</span>
<span id="cb11-40"><a href="#cb11-40" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-41"><a href="#cb11-41" aria-hidden="true" tabindex="-1"></a><span class="co"># Normalizar (Jaccard-like)</span></span>
<span id="cb11-42"><a href="#cb11-42" aria-hidden="true" tabindex="-1"></a>freq_individual <span class="ot">&lt;-</span> <span class="fu">diag</span>(coocurrencia)</span>
<span id="cb11-43"><a href="#cb11-43" aria-hidden="true" tabindex="-1"></a>cooc_norm <span class="ot">&lt;-</span> coocurrencia <span class="sc">/</span> <span class="fu">outer</span>(<span class="fu">sqrt</span>(freq_individual), <span class="fu">sqrt</span>(freq_individual))</span>
<span id="cb11-44"><a href="#cb11-44" aria-hidden="true" tabindex="-1"></a><span class="fu">diag</span>(cooc_norm) <span class="ot">&lt;-</span> <span class="dv">0</span></span>
<span id="cb11-45"><a href="#cb11-45" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-46"><a href="#cb11-46" aria-hidden="true" tabindex="-1"></a><span class="co"># Filtrar aristas débiles (percentil 75)</span></span>
<span id="cb11-47"><a href="#cb11-47" aria-hidden="true" tabindex="-1"></a>umbral <span class="ot">&lt;-</span> <span class="fu">quantile</span>(cooc_norm[<span class="fu">upper.tri</span>(cooc_norm)], <span class="fl">0.75</span>)</span>
<span id="cb11-48"><a href="#cb11-48" aria-hidden="true" tabindex="-1"></a>cooc_filtrada <span class="ot">&lt;-</span> cooc_norm <span class="sc">*</span> (cooc_norm <span class="sc">&gt;=</span> umbral)</span>
<span id="cb11-49"><a href="#cb11-49" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-50"><a href="#cb11-50" aria-hidden="true" tabindex="-1"></a><span class="co"># Crear grafo</span></span>
<span id="cb11-51"><a href="#cb11-51" aria-hidden="true" tabindex="-1"></a>red_pam50 <span class="ot">&lt;-</span> <span class="fu">graph_from_adjacency_matrix</span>(</span>
<span id="cb11-52"><a href="#cb11-52" aria-hidden="true" tabindex="-1"></a>  cooc_filtrada,</span>
<span id="cb11-53"><a href="#cb11-53" aria-hidden="true" tabindex="-1"></a>  <span class="at">mode =</span> <span class="st">"undirected"</span>,</span>
<span id="cb11-54"><a href="#cb11-54" aria-hidden="true" tabindex="-1"></a>  <span class="at">weighted =</span> <span class="cn">TRUE</span>,</span>
<span id="cb11-55"><a href="#cb11-55" aria-hidden="true" tabindex="-1"></a>  <span class="at">diag =</span> <span class="cn">FALSE</span></span>
<span id="cb11-56"><a href="#cb11-56" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb11-57"><a href="#cb11-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-58"><a href="#cb11-58" aria-hidden="true" tabindex="-1"></a><span class="co"># Eliminar nodos aislados</span></span>
<span id="cb11-59"><a href="#cb11-59" aria-hidden="true" tabindex="-1"></a>red_pam50 <span class="ot">&lt;-</span> <span class="fu">delete.vertices</span>(red_pam50, <span class="fu">degree</span>(red_pam50) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb11-60"><a href="#cb11-60" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-61"><a href="#cb11-61" aria-hidden="true" tabindex="-1"></a><span class="co"># Métricas de centralidad</span></span>
<span id="cb11-62"><a href="#cb11-62" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>degree <span class="ot">&lt;-</span> <span class="fu">degree</span>(red_pam50)</span>
<span id="cb11-63"><a href="#cb11-63" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>betweenness <span class="ot">&lt;-</span> <span class="fu">betweenness</span>(red_pam50, <span class="at">normalized =</span> <span class="cn">TRUE</span>)</span>
<span id="cb11-64"><a href="#cb11-64" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>eigenvector <span class="ot">&lt;-</span> <span class="fu">eigen_centrality</span>(red_pam50)<span class="sc">$</span>vector</span>
<span id="cb11-65"><a href="#cb11-65" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>closeness <span class="ot">&lt;-</span> <span class="fu">closeness</span>(red_pam50, <span class="at">normalized =</span> <span class="cn">TRUE</span>)</span>
<span id="cb11-66"><a href="#cb11-66" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-67"><a href="#cb11-67" aria-hidden="true" tabindex="-1"></a><span class="co"># Detección de comunidades</span></span>
<span id="cb11-68"><a href="#cb11-68" aria-hidden="true" tabindex="-1"></a>comunidades_pam50 <span class="ot">&lt;-</span> <span class="fu">cluster_louvain</span>(red_pam50)</span>
<span id="cb11-69"><a href="#cb11-69" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>community <span class="ot">&lt;-</span> <span class="fu">membership</span>(comunidades_pam50)</span>
<span id="cb11-70"><a href="#cb11-70" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-71"><a href="#cb11-71" aria-hidden="true" tabindex="-1"></a><span class="co"># Métricas de la red</span></span>
<span id="cb11-72"><a href="#cb11-72" aria-hidden="true" tabindex="-1"></a>metricas_pam50 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb11-73"><a href="#cb11-73" aria-hidden="true" tabindex="-1"></a>  <span class="at">Metrica =</span> <span class="fu">c</span>(</span>
<span id="cb11-74"><a href="#cb11-74" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Nodos"</span>,</span>
<span id="cb11-75"><a href="#cb11-75" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Aristas"</span>,</span>
<span id="cb11-76"><a href="#cb11-76" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Densidad"</span>,</span>
<span id="cb11-77"><a href="#cb11-77" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Clustering Global"</span>,</span>
<span id="cb11-78"><a href="#cb11-78" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Diámetro"</span>,</span>
<span id="cb11-79"><a href="#cb11-79" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Comunidades"</span>,</span>
<span id="cb11-80"><a href="#cb11-80" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Modularidad"</span></span>
<span id="cb11-81"><a href="#cb11-81" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb11-82"><a href="#cb11-82" aria-hidden="true" tabindex="-1"></a>  <span class="at">Valor =</span> <span class="fu">c</span>(</span>
<span id="cb11-83"><a href="#cb11-83" aria-hidden="true" tabindex="-1"></a>    <span class="fu">vcount</span>(red_pam50),</span>
<span id="cb11-84"><a href="#cb11-84" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ecount</span>(red_pam50),</span>
<span id="cb11-85"><a href="#cb11-85" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">edge_density</span>(red_pam50), <span class="dv">4</span>),</span>
<span id="cb11-86"><a href="#cb11-86" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">transitivity</span>(red_pam50, <span class="at">type =</span> <span class="st">"global"</span>), <span class="dv">4</span>),</span>
<span id="cb11-87"><a href="#cb11-87" aria-hidden="true" tabindex="-1"></a>    <span class="fu">diameter</span>(red_pam50, <span class="at">directed =</span> <span class="cn">FALSE</span>),</span>
<span id="cb11-88"><a href="#cb11-88" aria-hidden="true" tabindex="-1"></a>    <span class="fu">length</span>(<span class="fu">unique</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>community)),</span>
<span id="cb11-89"><a href="#cb11-89" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">modularity</span>(comunidades_pam50), <span class="dv">4</span>)</span>
<span id="cb11-90"><a href="#cb11-90" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb11-91"><a href="#cb11-91" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<section id="tabla-métricas-de-red-pam50" class="level4" data-number="2.3.1.1">
<h4 data-number="2.3.1.1" class="anchored" data-anchor-id="tabla-métricas-de-red-pam50"><span class="header-section-number">2.3.1.1</span> Tabla: Métricas de Red PAM50</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>  metricas_pam50,</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Métrica de Red"</span>, <span class="st">"Valor"</span>),</span>
<span id="cb12-4"><a href="#cb12-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb12-5"><a href="#cb12-5" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'t'</span>,</span>
<span id="cb12-6"><a href="#cb12-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span></span>
<span id="cb12-7"><a href="#cb12-7" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb12-8"><a href="#cb12-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Métricas topológicas de la red de características clínicas (PAM50)"</span></span>
<span id="cb12-9"><a href="#cb12-9" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-11830db2e5f90b751ad0" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-11830db2e5f90b751ad0">{"x":{"filter":"none","vertical":false,"caption":"<caption>Métricas topológicas de la red de características clínicas (PAM50)<\/caption>","data":[["1","2","3","4","5","6","7"],["Nodos","Aristas","Densidad","Clustering Global","Diámetro","Comunidades","Modularidad"],[57,604,0.3784,0.76,1.863028277382139,5,0.1022]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Métrica de Red<\/th>\n      <th>Valor<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","pageLength":10,"columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Metrica","targets":1},{"name":"Valor","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="visualización-red-de-co-ocurrencia-pam50" class="level4" data-number="2.3.1.2">
<h4 data-number="2.3.1.2" class="anchored" data-anchor-id="visualización-red-de-co-ocurrencia-pam50"><span class="header-section-number">2.3.1.2</span> Visualización: Red de Co-ocurrencia PAM50</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb13"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb13-1"><a href="#cb13-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">123</span>)</span>
<span id="cb13-2"><a href="#cb13-2" aria-hidden="true" tabindex="-1"></a>layout_fr <span class="ot">&lt;-</span> <span class="fu">layout_with_fr</span>(red_pam50)</span>
<span id="cb13-3"><a href="#cb13-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-4"><a href="#cb13-4" aria-hidden="true" tabindex="-1"></a>p_red_pam50 <span class="ot">&lt;-</span> <span class="fu">ggraph</span>(red_pam50, <span class="at">layout =</span> layout_fr) <span class="sc">+</span></span>
<span id="cb13-5"><a href="#cb13-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_edge_link</span>(<span class="fu">aes</span>(<span class="at">width =</span> weight, <span class="at">alpha =</span> weight), <span class="at">color =</span> <span class="st">"gray70"</span>) <span class="sc">+</span></span>
<span id="cb13-6"><a href="#cb13-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_point</span>(<span class="fu">aes</span>(<span class="at">size =</span> degree, <span class="at">color =</span> <span class="fu">factor</span>(community)), <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb13-7"><a href="#cb13-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> name), <span class="at">size =</span> <span class="fl">2.5</span>, <span class="at">repel =</span> <span class="cn">TRUE</span>, <span class="at">max.overlaps =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb13-8"><a href="#cb13-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_edge_width</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="fl">0.3</span>, <span class="dv">2</span>)) <span class="sc">+</span></span>
<span id="cb13-9"><a href="#cb13-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_edge_alpha</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="fl">0.3</span>, <span class="fl">0.8</span>)) <span class="sc">+</span></span>
<span id="cb13-10"><a href="#cb13-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_continuous</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="dv">3</span>, <span class="dv">12</span>)) <span class="sc">+</span></span>
<span id="cb13-11"><a href="#cb13-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"turbo"</span>) <span class="sc">+</span></span>
<span id="cb13-12"><a href="#cb13-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb13-13"><a href="#cb13-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Red de Co-ocurrencia de Características Clínicas"</span>,</span>
<span id="cb13-14"><a href="#cb13-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">"Análisis por Subtipo PAM50 + Claudin-low"</span>,</span>
<span id="cb13-15"><a href="#cb13-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="st">"Grado"</span>,</span>
<span id="cb13-16"><a href="#cb13-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"Comunidad"</span></span>
<span id="cb13-17"><a href="#cb13-17" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb13-18"><a href="#cb13-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_graph</span>() <span class="sc">+</span></span>
<span id="cb13-19"><a href="#cb13-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb13-20"><a href="#cb13-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">16</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb13-21"><a href="#cb13-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>)</span>
<span id="cb13-22"><a href="#cb13-22" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb13-23"><a href="#cb13-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb13-24"><a href="#cb13-24" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(p_red_pam50)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/viz_red_pam50-1.png" class="img-fluid figure-img" width="1344"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"figuras_redes/red_pam50_completa.png"</span>, p_red_pam50,</span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>       <span class="at">width =</span> <span class="dv">14</span>, <span class="at">height =</span> <span class="dv">12</span>, <span class="at">dpi =</span> <span class="dv">300</span>, <span class="at">bg =</span> <span class="st">"white"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="tabla-top-nodos-centrales-pam50" class="level4" data-number="2.3.1.3">
<h4 data-number="2.3.1.3" class="anchored" data-anchor-id="tabla-top-nodos-centrales-pam50"><span class="header-section-number">2.3.1.3</span> Tabla: Top Nodos Centrales (PAM50)</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a>centralidad_pam50 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>name,</span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>degree,</span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Intermediacion =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>betweenness, <span class="dv">4</span>),</span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">Cercania =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>closeness, <span class="dv">4</span>),</span>
<span id="cb15-6"><a href="#cb15-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">Eigenvector =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>eigenvector, <span class="dv">4</span>),</span>
<span id="cb15-7"><a href="#cb15-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>community</span>
<span id="cb15-8"><a href="#cb15-8" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb15-9"><a href="#cb15-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(Grado)) <span class="sc">%&gt;%</span></span>
<span id="cb15-10"><a href="#cb15-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">head</span>(<span class="dv">20</span>)</span>
<span id="cb15-11"><a href="#cb15-11" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-12"><a href="#cb15-12" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb15-13"><a href="#cb15-13" aria-hidden="true" tabindex="-1"></a>  centralidad_pam50,</span>
<span id="cb15-14"><a href="#cb15-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Característica"</span>, <span class="st">"Grado"</span>, <span class="st">"Intermediación"</span>, <span class="st">"Cercanía"</span>, <span class="st">"Eigenvector"</span>, <span class="st">"Comunidad"</span>),</span>
<span id="cb15-15"><a href="#cb15-15" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb15-16"><a href="#cb15-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">20</span>,</span>
<span id="cb15-17"><a href="#cb15-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb15-18"><a href="#cb15-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb15-19"><a href="#cb15-19" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb15-20"><a href="#cb15-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb15-21"><a href="#cb15-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Top 20 características más centrales en la red PAM50"</span></span>
<span id="cb15-22"><a href="#cb15-22" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb15-23"><a href="#cb15-23" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">c</span>(<span class="st">'Intermediacion'</span>, <span class="st">'Cercania'</span>, <span class="st">'Eigenvector'</span>), <span class="dv">4</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-3d086dc0e5b36813ff72" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-3d086dc0e5b36813ff72">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Top 20 características más centrales en la red PAM50<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20"],["HistoSubtype: Ductal/NST","HER2: Negative","ER: Positive","Radio: YES","Surgery: MASTECTOMY","Grade: 3","Menopause: Post","Chemo: NO","PR: Negative","Hormone: YES","Relapse: 0:Not Recurred","PR: Positive","Cell: High","Survival: 1:DECEASED","Laterality: Left","Laterality: Right","Survival: 0:LIVING","LymphGroup: N0","Relapse: 1:Recurred","Stage: 2"],[44,43,38,38,38,37,37,36,35,35,35,34,34,34,33,32,32,32,31,30],[0.026,0.0195,0.0234,0.0188,0.0169,0.0403,0.0019,0.0019,0.0442,0.0013,0.0045,0.0227,0.0019,0.0058,0.0065,0.0091,0.0136,0.0052,0.0221,0.0156],[1.7431,1.6842,1.6093,1.7144,1.7597,1.8169,1.5624,1.4508,1.7894,1.5793,1.6113,1.6418,1.7094,1.6851,1.6576,1.7097,1.7182,1.592,1.7587,1.6515],[0.9298999999999999,1,0.9318,0.7977,0.795,0.6928,0.9442,0.9418,0.6535,0.8191000000000001,0.8048,0.7683,0.7205,0.7741,0.7163,0.6743,0.6511,0.7375,0.6082,0.6185],[1,2,2,1,2,1,2,2,1,2,3,2,2,2,2,3,3,3,2,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Característica<\/th>\n      <th>Grado<\/th>\n      <th>Intermediación<\/th>\n      <th>Cercanía<\/th>\n      <th>Eigenvector<\/th>\n      <th>Comunidad<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":20,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Caracteristica","targets":1},{"name":"Grado","targets":2},{"name":"Intermediacion","targets":3},{"name":"Cercania","targets":4},{"name":"Eigenvector","targets":5},{"name":"Comunidad","targets":6}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="red-por-clasificador-3-gene" class="level3" data-number="2.3.2">
<h3 data-number="2.3.2" class="anchored" data-anchor-id="red-por-clasificador-3-gene"><span class="header-section-number">2.3.2</span> Red por Clasificador 3-Gene</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a>datos_3gene <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(Gene3))</span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-4"><a href="#cb16-4" aria-hidden="true" tabindex="-1"></a>matriz_caracteristicas_3g <span class="ot">&lt;-</span> datos_3gene <span class="sc">%&gt;%</span></span>
<span id="cb16-5"><a href="#cb16-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(</span>
<span id="cb16-6"><a href="#cb16-6" aria-hidden="true" tabindex="-1"></a>    Gene3, ER, PR, HER2, Grade, Stage, Cell, Chemo, </span>
<span id="cb16-7"><a href="#cb16-7" aria-hidden="true" tabindex="-1"></a>    Hormone, Radio, Menopause, Surgery, Laterality, </span>
<span id="cb16-8"><a href="#cb16-8" aria-hidden="true" tabindex="-1"></a>    HistoSubtype, AgeGroup, SizeGroup, LymphGroup,</span>
<span id="cb16-9"><a href="#cb16-9" aria-hidden="true" tabindex="-1"></a>    Survival, Relapse</span>
<span id="cb16-10"><a href="#cb16-10" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb16-11"><a href="#cb16-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), as.character)) <span class="sc">%&gt;%</span></span>
<span id="cb16-12"><a href="#cb16-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), <span class="sc">~</span><span class="fu">replace_na</span>(., <span class="st">"Unknown"</span>)))</span>
<span id="cb16-13"><a href="#cb16-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-14"><a href="#cb16-14" aria-hidden="true" tabindex="-1"></a>datos_largo_3g <span class="ot">&lt;-</span> matriz_caracteristicas_3g <span class="sc">%&gt;%</span></span>
<span id="cb16-15"><a href="#cb16-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">id =</span> <span class="fu">row_number</span>()) <span class="sc">%&gt;%</span></span>
<span id="cb16-16"><a href="#cb16-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(</span>
<span id="cb16-17"><a href="#cb16-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">cols =</span> <span class="sc">-</span>id,</span>
<span id="cb16-18"><a href="#cb16-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_to =</span> <span class="st">"Variable"</span>,</span>
<span id="cb16-19"><a href="#cb16-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_to =</span> <span class="st">"Valor"</span></span>
<span id="cb16-20"><a href="#cb16-20" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb16-21"><a href="#cb16-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Caracteristica =</span> <span class="fu">paste</span>(Variable, Valor, <span class="at">sep =</span> <span class="st">": "</span>))</span>
<span id="cb16-22"><a href="#cb16-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-23"><a href="#cb16-23" aria-hidden="true" tabindex="-1"></a>matriz_bin_3g <span class="ot">&lt;-</span> datos_largo_3g <span class="sc">%&gt;%</span></span>
<span id="cb16-24"><a href="#cb16-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">presente =</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span></span>
<span id="cb16-25"><a href="#cb16-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(</span>
<span id="cb16-26"><a href="#cb16-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">id_cols =</span> id,</span>
<span id="cb16-27"><a href="#cb16-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_from =</span> Caracteristica,</span>
<span id="cb16-28"><a href="#cb16-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_from =</span> presente,</span>
<span id="cb16-29"><a href="#cb16-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_fill =</span> <span class="dv">0</span></span>
<span id="cb16-30"><a href="#cb16-30" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb16-31"><a href="#cb16-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="sc">-</span>id) <span class="sc">%&gt;%</span></span>
<span id="cb16-32"><a href="#cb16-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.matrix</span>()</span>
<span id="cb16-33"><a href="#cb16-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-34"><a href="#cb16-34" aria-hidden="true" tabindex="-1"></a>coocurrencia_3g <span class="ot">&lt;-</span> <span class="fu">t</span>(matriz_bin_3g) <span class="sc">%*%</span> matriz_bin_3g</span>
<span id="cb16-35"><a href="#cb16-35" aria-hidden="true" tabindex="-1"></a>freq_individual_3g <span class="ot">&lt;-</span> <span class="fu">diag</span>(coocurrencia_3g)</span>
<span id="cb16-36"><a href="#cb16-36" aria-hidden="true" tabindex="-1"></a>cooc_norm_3g <span class="ot">&lt;-</span> coocurrencia_3g <span class="sc">/</span> <span class="fu">outer</span>(<span class="fu">sqrt</span>(freq_individual_3g), <span class="fu">sqrt</span>(freq_individual_3g))</span>
<span id="cb16-37"><a href="#cb16-37" aria-hidden="true" tabindex="-1"></a><span class="fu">diag</span>(cooc_norm_3g) <span class="ot">&lt;-</span> <span class="dv">0</span></span>
<span id="cb16-38"><a href="#cb16-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-39"><a href="#cb16-39" aria-hidden="true" tabindex="-1"></a>umbral_3g <span class="ot">&lt;-</span> <span class="fu">quantile</span>(cooc_norm_3g[<span class="fu">upper.tri</span>(cooc_norm_3g)], <span class="fl">0.75</span>)</span>
<span id="cb16-40"><a href="#cb16-40" aria-hidden="true" tabindex="-1"></a>cooc_filtrada_3g <span class="ot">&lt;-</span> cooc_norm_3g <span class="sc">*</span> (cooc_norm_3g <span class="sc">&gt;=</span> umbral_3g)</span>
<span id="cb16-41"><a href="#cb16-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-42"><a href="#cb16-42" aria-hidden="true" tabindex="-1"></a>red_3gene <span class="ot">&lt;-</span> <span class="fu">graph_from_adjacency_matrix</span>(</span>
<span id="cb16-43"><a href="#cb16-43" aria-hidden="true" tabindex="-1"></a>  cooc_filtrada_3g,</span>
<span id="cb16-44"><a href="#cb16-44" aria-hidden="true" tabindex="-1"></a>  <span class="at">mode =</span> <span class="st">"undirected"</span>,</span>
<span id="cb16-45"><a href="#cb16-45" aria-hidden="true" tabindex="-1"></a>  <span class="at">weighted =</span> <span class="cn">TRUE</span>,</span>
<span id="cb16-46"><a href="#cb16-46" aria-hidden="true" tabindex="-1"></a>  <span class="at">diag =</span> <span class="cn">FALSE</span></span>
<span id="cb16-47"><a href="#cb16-47" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb16-48"><a href="#cb16-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-49"><a href="#cb16-49" aria-hidden="true" tabindex="-1"></a>red_3gene <span class="ot">&lt;-</span> <span class="fu">delete.vertices</span>(red_3gene, <span class="fu">degree</span>(red_3gene) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb16-50"><a href="#cb16-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-51"><a href="#cb16-51" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_3gene)<span class="sc">$</span>degree <span class="ot">&lt;-</span> <span class="fu">degree</span>(red_3gene)</span>
<span id="cb16-52"><a href="#cb16-52" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_3gene)<span class="sc">$</span>betweenness <span class="ot">&lt;-</span> <span class="fu">betweenness</span>(red_3gene, <span class="at">normalized =</span> <span class="cn">TRUE</span>)</span>
<span id="cb16-53"><a href="#cb16-53" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_3gene)<span class="sc">$</span>eigenvector <span class="ot">&lt;-</span> <span class="fu">eigen_centrality</span>(red_3gene)<span class="sc">$</span>vector</span>
<span id="cb16-54"><a href="#cb16-54" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-55"><a href="#cb16-55" aria-hidden="true" tabindex="-1"></a>comunidades_3gene <span class="ot">&lt;-</span> <span class="fu">cluster_louvain</span>(red_3gene)</span>
<span id="cb16-56"><a href="#cb16-56" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_3gene)<span class="sc">$</span>community <span class="ot">&lt;-</span> <span class="fu">membership</span>(comunidades_3gene)</span>
<span id="cb16-57"><a href="#cb16-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb16-58"><a href="#cb16-58" aria-hidden="true" tabindex="-1"></a>metricas_3gene <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb16-59"><a href="#cb16-59" aria-hidden="true" tabindex="-1"></a>  <span class="at">Metrica =</span> <span class="fu">c</span>(</span>
<span id="cb16-60"><a href="#cb16-60" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Nodos"</span>,</span>
<span id="cb16-61"><a href="#cb16-61" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Aristas"</span>,</span>
<span id="cb16-62"><a href="#cb16-62" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Densidad"</span>,</span>
<span id="cb16-63"><a href="#cb16-63" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Clustering Global"</span>,</span>
<span id="cb16-64"><a href="#cb16-64" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Diámetro"</span>,</span>
<span id="cb16-65"><a href="#cb16-65" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Comunidades"</span>,</span>
<span id="cb16-66"><a href="#cb16-66" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Modularidad"</span></span>
<span id="cb16-67"><a href="#cb16-67" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb16-68"><a href="#cb16-68" aria-hidden="true" tabindex="-1"></a>  <span class="at">Valor =</span> <span class="fu">c</span>(</span>
<span id="cb16-69"><a href="#cb16-69" aria-hidden="true" tabindex="-1"></a>    <span class="fu">vcount</span>(red_3gene),</span>
<span id="cb16-70"><a href="#cb16-70" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ecount</span>(red_3gene),</span>
<span id="cb16-71"><a href="#cb16-71" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">edge_density</span>(red_3gene), <span class="dv">4</span>),</span>
<span id="cb16-72"><a href="#cb16-72" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">transitivity</span>(red_3gene, <span class="at">type =</span> <span class="st">"global"</span>), <span class="dv">4</span>),</span>
<span id="cb16-73"><a href="#cb16-73" aria-hidden="true" tabindex="-1"></a>    <span class="fu">diameter</span>(red_3gene, <span class="at">directed =</span> <span class="cn">FALSE</span>),</span>
<span id="cb16-74"><a href="#cb16-74" aria-hidden="true" tabindex="-1"></a>    <span class="fu">length</span>(<span class="fu">unique</span>(<span class="fu">V</span>(red_3gene)<span class="sc">$</span>community)),</span>
<span id="cb16-75"><a href="#cb16-75" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">modularity</span>(comunidades_3gene), <span class="dv">4</span>)</span>
<span id="cb16-76"><a href="#cb16-76" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb16-77"><a href="#cb16-77" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<section id="tabla-métricas-de-red-3-gene" class="level4" data-number="2.3.2.1">
<h4 data-number="2.3.2.1" class="anchored" data-anchor-id="tabla-métricas-de-red-3-gene"><span class="header-section-number">2.3.2.1</span> Tabla: Métricas de Red 3-Gene</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a>  metricas_3gene,</span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Métrica de Red"</span>, <span class="st">"Valor"</span>),</span>
<span id="cb17-4"><a href="#cb17-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(<span class="at">dom =</span> <span class="st">'t'</span>),</span>
<span id="cb17-5"><a href="#cb17-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Métricas topológicas de la red 3-Gene Classifier"</span></span>
<span id="cb17-6"><a href="#cb17-6" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-2abb9242f60569722e78" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2abb9242f60569722e78">{"x":{"filter":"none","vertical":false,"caption":"<caption>Métricas topológicas de la red 3-Gene Classifier<\/caption>","data":[["1","2","3","4","5","6","7"],["Nodos","Aristas","Densidad","Clustering Global","Diámetro","Comunidades","Modularidad"],[53,553,0.4013,0.7579,2.116271356185363,4,0.1047]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Métrica de Red<\/th>\n      <th>Valor<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Metrica","targets":1},{"name":"Valor","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="visualización-red-3-gene-classifier" class="level4" data-number="2.3.2.2">
<h4 data-number="2.3.2.2" class="anchored" data-anchor-id="visualización-red-3-gene-classifier"><span class="header-section-number">2.3.2.2</span> Visualización: Red 3-Gene Classifier</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">456</span>)</span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a>layout_fr_3g <span class="ot">&lt;-</span> <span class="fu">layout_with_fr</span>(red_3gene)</span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-4"><a href="#cb18-4" aria-hidden="true" tabindex="-1"></a>p_red_3gene <span class="ot">&lt;-</span> <span class="fu">ggraph</span>(red_3gene, <span class="at">layout =</span> layout_fr_3g) <span class="sc">+</span></span>
<span id="cb18-5"><a href="#cb18-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_edge_link</span>(<span class="fu">aes</span>(<span class="at">width =</span> weight, <span class="at">alpha =</span> weight), <span class="at">color =</span> <span class="st">"gray70"</span>) <span class="sc">+</span></span>
<span id="cb18-6"><a href="#cb18-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_point</span>(<span class="fu">aes</span>(<span class="at">size =</span> degree, <span class="at">color =</span> <span class="fu">factor</span>(community)), <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb18-7"><a href="#cb18-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> name), <span class="at">size =</span> <span class="fl">2.5</span>, <span class="at">repel =</span> <span class="cn">TRUE</span>, <span class="at">max.overlaps =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb18-8"><a href="#cb18-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_edge_width</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="fl">0.3</span>, <span class="dv">2</span>)) <span class="sc">+</span></span>
<span id="cb18-9"><a href="#cb18-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_edge_alpha</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="fl">0.3</span>, <span class="fl">0.8</span>)) <span class="sc">+</span></span>
<span id="cb18-10"><a href="#cb18-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_continuous</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="dv">3</span>, <span class="dv">12</span>)) <span class="sc">+</span></span>
<span id="cb18-11"><a href="#cb18-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"plasma"</span>) <span class="sc">+</span></span>
<span id="cb18-12"><a href="#cb18-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb18-13"><a href="#cb18-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Red de Co-ocurrencia: Clasificador 3-Gene"</span>,</span>
<span id="cb18-14"><a href="#cb18-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">"Relaciones entre características clínicas y subtipos moleculares"</span>,</span>
<span id="cb18-15"><a href="#cb18-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="st">"Grado"</span>,</span>
<span id="cb18-16"><a href="#cb18-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"Comunidad"</span></span>
<span id="cb18-17"><a href="#cb18-17" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb18-18"><a href="#cb18-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_graph</span>() <span class="sc">+</span></span>
<span id="cb18-19"><a href="#cb18-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb18-20"><a href="#cb18-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">16</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb18-21"><a href="#cb18-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>)</span>
<span id="cb18-22"><a href="#cb18-22" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb18-23"><a href="#cb18-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb18-24"><a href="#cb18-24" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(p_red_3gene)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/viz_red_3gene-1.png" class="img-fluid figure-img" width="1344"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"figuras_redes/red_3gene_completa.png"</span>, p_red_3gene,</span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>       <span class="at">width =</span> <span class="dv">14</span>, <span class="at">height =</span> <span class="dv">12</span>, <span class="at">dpi =</span> <span class="dv">300</span>, <span class="at">bg =</span> <span class="st">"white"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="tabla-top-nodos-centrales-3-gene" class="level4" data-number="2.3.2.3">
<h4 data-number="2.3.2.3" class="anchored" data-anchor-id="tabla-top-nodos-centrales-3-gene"><span class="header-section-number">2.3.2.3</span> Tabla: Top Nodos Centrales (3-Gene)</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb20"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb20-1"><a href="#cb20-1" aria-hidden="true" tabindex="-1"></a>centralidad_3gene <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb20-2"><a href="#cb20-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_3gene)<span class="sc">$</span>name,</span>
<span id="cb20-3"><a href="#cb20-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_3gene)<span class="sc">$</span>degree,</span>
<span id="cb20-4"><a href="#cb20-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Intermediacion =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_3gene)<span class="sc">$</span>betweenness, <span class="dv">4</span>),</span>
<span id="cb20-5"><a href="#cb20-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">Eigenvector =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_3gene)<span class="sc">$</span>eigenvector, <span class="dv">4</span>),</span>
<span id="cb20-6"><a href="#cb20-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_3gene)<span class="sc">$</span>community</span>
<span id="cb20-7"><a href="#cb20-7" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb20-8"><a href="#cb20-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(Grado)) <span class="sc">%&gt;%</span></span>
<span id="cb20-9"><a href="#cb20-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">head</span>(<span class="dv">20</span>)</span>
<span id="cb20-10"><a href="#cb20-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-11"><a href="#cb20-11" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb20-12"><a href="#cb20-12" aria-hidden="true" tabindex="-1"></a>  centralidad_3gene,</span>
<span id="cb20-13"><a href="#cb20-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Característica"</span>, <span class="st">"Grado"</span>, <span class="st">"Intermediación"</span>, <span class="st">"Eigenvector"</span>, <span class="st">"Comunidad"</span>),</span>
<span id="cb20-14"><a href="#cb20-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb20-15"><a href="#cb20-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">20</span>,</span>
<span id="cb20-16"><a href="#cb20-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb20-17"><a href="#cb20-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb20-18"><a href="#cb20-18" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb20-19"><a href="#cb20-19" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb20-20"><a href="#cb20-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Top 20 características más centrales en la red 3-Gene"</span></span>
<span id="cb20-21"><a href="#cb20-21" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb20-22"><a href="#cb20-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">c</span>(<span class="st">'Intermediacion'</span>, <span class="st">'Eigenvector'</span>), <span class="dv">4</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-fc85235cfc94750fbe1a" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-fc85235cfc94750fbe1a">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Top 20 características más centrales en la red 3-Gene<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20"],["HistoSubtype: Ductal/NST","HER2: Negative","ER: Positive","Menopause: Post","Grade: 3","Hormone: YES","Radio: YES","Chemo: NO","Surgery: MASTECTOMY","Relapse: 0:Not Recurred","Survival: 1:DECEASED","PR: Negative","Laterality: Right","PR: Positive","Cell: High","LymphGroup: N0","Laterality: Left","Stage: 2","Survival: 0:LIVING","Radio: NO"],[44,43,36,36,35,35,35,34,34,34,33,32,31,31,31,31,31,29,29,29],[0.0513,0.04,0.0151,0.0023,0.0633,0.0143,0.0166,0.0113,0.0038,0.0143,0.0113,0.0234,0.0264,0.0053,0.0151,0.006,0.0015,0.0339,0.008999999999999999,0.0098],[0.9245,1,0.9371,0.9411,0.6778,0.8267,0.7728,0.9416,0.7796999999999999,0.8053,0.7668,0.6391,0.6744,0.7621,0.7002,0.7396,0.7072000000000001,0.6041,0.6324,0.6435],[1,3,2,2,1,3,1,2,3,2,3,1,2,2,3,2,2,3,2,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Característica<\/th>\n      <th>Grado<\/th>\n      <th>Intermediación<\/th>\n      <th>Eigenvector<\/th>\n      <th>Comunidad<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":20,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Caracteristica","targets":1},{"name":"Grado","targets":2},{"name":"Intermediacion","targets":3},{"name":"Eigenvector","targets":4},{"name":"Comunidad","targets":5}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="red-por-integrative-cluster" class="level3" data-number="2.3.3">
<h3 data-number="2.3.3" class="anchored" data-anchor-id="red-por-integrative-cluster"><span class="header-section-number">2.3.3</span> Red por Integrative Cluster</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a>datos_intclust <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(IntCluster))</span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>matriz_caracteristicas_ic <span class="ot">&lt;-</span> datos_intclust <span class="sc">%&gt;%</span></span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>    IntCluster, ER, PR, HER2, Grade, Stage, Cell, Chemo, </span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a>    Hormone, Radio, Menopause, Surgery, Laterality, </span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a>    HistoSubtype, AgeGroup, SizeGroup, LymphGroup,</span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a>    Survival, Relapse</span>
<span id="cb21-10"><a href="#cb21-10" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb21-11"><a href="#cb21-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), as.character)) <span class="sc">%&gt;%</span></span>
<span id="cb21-12"><a href="#cb21-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(<span class="fu">everything</span>(), <span class="sc">~</span><span class="fu">replace_na</span>(., <span class="st">"Unknown"</span>)))</span>
<span id="cb21-13"><a href="#cb21-13" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-14"><a href="#cb21-14" aria-hidden="true" tabindex="-1"></a>datos_largo_ic <span class="ot">&lt;-</span> matriz_caracteristicas_ic <span class="sc">%&gt;%</span></span>
<span id="cb21-15"><a href="#cb21-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">id =</span> <span class="fu">row_number</span>()) <span class="sc">%&gt;%</span></span>
<span id="cb21-16"><a href="#cb21-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(</span>
<span id="cb21-17"><a href="#cb21-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">cols =</span> <span class="sc">-</span>id,</span>
<span id="cb21-18"><a href="#cb21-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_to =</span> <span class="st">"Variable"</span>,</span>
<span id="cb21-19"><a href="#cb21-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_to =</span> <span class="st">"Valor"</span></span>
<span id="cb21-20"><a href="#cb21-20" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb21-21"><a href="#cb21-21" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">Caracteristica =</span> <span class="fu">paste</span>(Variable, Valor, <span class="at">sep =</span> <span class="st">": "</span>))</span>
<span id="cb21-22"><a href="#cb21-22" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-23"><a href="#cb21-23" aria-hidden="true" tabindex="-1"></a>matriz_bin_ic <span class="ot">&lt;-</span> datos_largo_ic <span class="sc">%&gt;%</span></span>
<span id="cb21-24"><a href="#cb21-24" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">presente =</span> <span class="dv">1</span>) <span class="sc">%&gt;%</span></span>
<span id="cb21-25"><a href="#cb21-25" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(</span>
<span id="cb21-26"><a href="#cb21-26" aria-hidden="true" tabindex="-1"></a>    <span class="at">id_cols =</span> id,</span>
<span id="cb21-27"><a href="#cb21-27" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_from =</span> Caracteristica,</span>
<span id="cb21-28"><a href="#cb21-28" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_from =</span> presente,</span>
<span id="cb21-29"><a href="#cb21-29" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_fill =</span> <span class="dv">0</span></span>
<span id="cb21-30"><a href="#cb21-30" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb21-31"><a href="#cb21-31" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(<span class="sc">-</span>id) <span class="sc">%&gt;%</span></span>
<span id="cb21-32"><a href="#cb21-32" aria-hidden="true" tabindex="-1"></a>  <span class="fu">as.matrix</span>()</span>
<span id="cb21-33"><a href="#cb21-33" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-34"><a href="#cb21-34" aria-hidden="true" tabindex="-1"></a>coocurrencia_ic <span class="ot">&lt;-</span> <span class="fu">t</span>(matriz_bin_ic) <span class="sc">%*%</span> matriz_bin_ic</span>
<span id="cb21-35"><a href="#cb21-35" aria-hidden="true" tabindex="-1"></a>freq_individual_ic <span class="ot">&lt;-</span> <span class="fu">diag</span>(coocurrencia_ic)</span>
<span id="cb21-36"><a href="#cb21-36" aria-hidden="true" tabindex="-1"></a>cooc_norm_ic <span class="ot">&lt;-</span> coocurrencia_ic <span class="sc">/</span> <span class="fu">outer</span>(<span class="fu">sqrt</span>(freq_individual_ic), <span class="fu">sqrt</span>(freq_individual_ic))</span>
<span id="cb21-37"><a href="#cb21-37" aria-hidden="true" tabindex="-1"></a><span class="fu">diag</span>(cooc_norm_ic) <span class="ot">&lt;-</span> <span class="dv">0</span></span>
<span id="cb21-38"><a href="#cb21-38" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-39"><a href="#cb21-39" aria-hidden="true" tabindex="-1"></a>umbral_ic <span class="ot">&lt;-</span> <span class="fu">quantile</span>(cooc_norm_ic[<span class="fu">upper.tri</span>(cooc_norm_ic)], <span class="fl">0.75</span>)</span>
<span id="cb21-40"><a href="#cb21-40" aria-hidden="true" tabindex="-1"></a>cooc_filtrada_ic <span class="ot">&lt;-</span> cooc_norm_ic <span class="sc">*</span> (cooc_norm_ic <span class="sc">&gt;=</span> umbral_ic)</span>
<span id="cb21-41"><a href="#cb21-41" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-42"><a href="#cb21-42" aria-hidden="true" tabindex="-1"></a>red_intclust <span class="ot">&lt;-</span> <span class="fu">graph_from_adjacency_matrix</span>(</span>
<span id="cb21-43"><a href="#cb21-43" aria-hidden="true" tabindex="-1"></a>  cooc_filtrada_ic,</span>
<span id="cb21-44"><a href="#cb21-44" aria-hidden="true" tabindex="-1"></a>  <span class="at">mode =</span> <span class="st">"undirected"</span>,</span>
<span id="cb21-45"><a href="#cb21-45" aria-hidden="true" tabindex="-1"></a>  <span class="at">weighted =</span> <span class="cn">TRUE</span>,</span>
<span id="cb21-46"><a href="#cb21-46" aria-hidden="true" tabindex="-1"></a>  <span class="at">diag =</span> <span class="cn">FALSE</span></span>
<span id="cb21-47"><a href="#cb21-47" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb21-48"><a href="#cb21-48" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-49"><a href="#cb21-49" aria-hidden="true" tabindex="-1"></a>red_intclust <span class="ot">&lt;-</span> <span class="fu">delete.vertices</span>(red_intclust, <span class="fu">degree</span>(red_intclust) <span class="sc">==</span> <span class="dv">0</span>)</span>
<span id="cb21-50"><a href="#cb21-50" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-51"><a href="#cb21-51" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_intclust)<span class="sc">$</span>degree <span class="ot">&lt;-</span> <span class="fu">degree</span>(red_intclust)</span>
<span id="cb21-52"><a href="#cb21-52" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_intclust)<span class="sc">$</span>betweenness <span class="ot">&lt;-</span> <span class="fu">betweenness</span>(red_intclust, <span class="at">normalized =</span> <span class="cn">TRUE</span>)</span>
<span id="cb21-53"><a href="#cb21-53" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_intclust)<span class="sc">$</span>eigenvector <span class="ot">&lt;-</span> <span class="fu">eigen_centrality</span>(red_intclust)<span class="sc">$</span>vector</span>
<span id="cb21-54"><a href="#cb21-54" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-55"><a href="#cb21-55" aria-hidden="true" tabindex="-1"></a>comunidades_intclust <span class="ot">&lt;-</span> <span class="fu">cluster_louvain</span>(red_intclust)</span>
<span id="cb21-56"><a href="#cb21-56" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_intclust)<span class="sc">$</span>community <span class="ot">&lt;-</span> <span class="fu">membership</span>(comunidades_intclust)</span>
<span id="cb21-57"><a href="#cb21-57" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-58"><a href="#cb21-58" aria-hidden="true" tabindex="-1"></a>metricas_intclust <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb21-59"><a href="#cb21-59" aria-hidden="true" tabindex="-1"></a>  <span class="at">Metrica =</span> <span class="fu">c</span>(</span>
<span id="cb21-60"><a href="#cb21-60" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Nodos"</span>,</span>
<span id="cb21-61"><a href="#cb21-61" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Aristas"</span>,</span>
<span id="cb21-62"><a href="#cb21-62" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Densidad"</span>,</span>
<span id="cb21-63"><a href="#cb21-63" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Clustering Global"</span>,</span>
<span id="cb21-64"><a href="#cb21-64" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Diámetro"</span>,</span>
<span id="cb21-65"><a href="#cb21-65" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Comunidades"</span>,</span>
<span id="cb21-66"><a href="#cb21-66" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Modularidad"</span></span>
<span id="cb21-67"><a href="#cb21-67" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb21-68"><a href="#cb21-68" aria-hidden="true" tabindex="-1"></a>  <span class="at">Valor =</span> <span class="fu">c</span>(</span>
<span id="cb21-69"><a href="#cb21-69" aria-hidden="true" tabindex="-1"></a>    <span class="fu">vcount</span>(red_intclust),</span>
<span id="cb21-70"><a href="#cb21-70" aria-hidden="true" tabindex="-1"></a>    <span class="fu">ecount</span>(red_intclust),</span>
<span id="cb21-71"><a href="#cb21-71" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">edge_density</span>(red_intclust), <span class="dv">4</span>),</span>
<span id="cb21-72"><a href="#cb21-72" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">transitivity</span>(red_intclust, <span class="at">type =</span> <span class="st">"global"</span>), <span class="dv">4</span>),</span>
<span id="cb21-73"><a href="#cb21-73" aria-hidden="true" tabindex="-1"></a>    <span class="fu">diameter</span>(red_intclust, <span class="at">directed =</span> <span class="cn">FALSE</span>),</span>
<span id="cb21-74"><a href="#cb21-74" aria-hidden="true" tabindex="-1"></a>    <span class="fu">length</span>(<span class="fu">unique</span>(<span class="fu">V</span>(red_intclust)<span class="sc">$</span>community)),</span>
<span id="cb21-75"><a href="#cb21-75" aria-hidden="true" tabindex="-1"></a>    <span class="fu">round</span>(<span class="fu">modularity</span>(comunidades_intclust), <span class="dv">4</span>)</span>
<span id="cb21-76"><a href="#cb21-76" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb21-77"><a href="#cb21-77" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
<section id="tabla-métricas-de-red-integrative-cluster" class="level4" data-number="2.3.3.1">
<h4 data-number="2.3.3.1" class="anchored" data-anchor-id="tabla-métricas-de-red-integrative-cluster"><span class="header-section-number">2.3.3.1</span> Tabla: Métricas de Red Integrative Cluster</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb22"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb22-1"><a href="#cb22-1" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb22-2"><a href="#cb22-2" aria-hidden="true" tabindex="-1"></a>  metricas_intclust,</span>
<span id="cb22-3"><a href="#cb22-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Métrica de Red"</span>, <span class="st">"Valor"</span>),</span>
<span id="cb22-4"><a href="#cb22-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(<span class="at">dom =</span> <span class="st">'t'</span>),</span>
<span id="cb22-5"><a href="#cb22-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Métricas topológicas de la red Integrative Cluster"</span></span>
<span id="cb22-6"><a href="#cb22-6" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-44496f11d8538761c4cf" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-44496f11d8538761c4cf">{"x":{"filter":"none","vertical":false,"caption":"<caption>Métricas topológicas de la red Integrative Cluster<\/caption>","data":[["1","2","3","4","5","6","7"],["Nodos","Aristas","Densidad","Clustering Global","Diámetro","Comunidades","Modularidad"],[61,676,0.3694,0.7498,1.872602017294557,6,0.1008]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Métrica de Red<\/th>\n      <th>Valor<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","columnDefs":[{"className":"dt-right","targets":2},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Metrica","targets":1},{"name":"Valor","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="visualización-red-integrative-cluster" class="level4" data-number="2.3.3.2">
<h4 data-number="2.3.3.2" class="anchored" data-anchor-id="visualización-red-integrative-cluster"><span class="header-section-number">2.3.3.2</span> Visualización: Red Integrative Cluster</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">789</span>)</span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>layout_fr_ic <span class="ot">&lt;-</span> <span class="fu">layout_with_fr</span>(red_intclust)</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a>p_red_intclust <span class="ot">&lt;-</span> <span class="fu">ggraph</span>(red_intclust, <span class="at">layout =</span> layout_fr_ic) <span class="sc">+</span></span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_edge_link</span>(<span class="fu">aes</span>(<span class="at">width =</span> weight, <span class="at">alpha =</span> weight), <span class="at">color =</span> <span class="st">"gray70"</span>) <span class="sc">+</span></span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_point</span>(<span class="fu">aes</span>(<span class="at">size =</span> degree, <span class="at">color =</span> <span class="fu">factor</span>(community)), <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_text</span>(<span class="fu">aes</span>(<span class="at">label =</span> name), <span class="at">size =</span> <span class="fl">2.5</span>, <span class="at">repel =</span> <span class="cn">TRUE</span>, <span class="at">max.overlaps =</span> <span class="dv">20</span>) <span class="sc">+</span></span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_edge_width</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="fl">0.3</span>, <span class="dv">2</span>)) <span class="sc">+</span></span>
<span id="cb23-9"><a href="#cb23-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_edge_alpha</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="fl">0.3</span>, <span class="fl">0.8</span>)) <span class="sc">+</span></span>
<span id="cb23-10"><a href="#cb23-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_continuous</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="dv">3</span>, <span class="dv">12</span>)) <span class="sc">+</span></span>
<span id="cb23-11"><a href="#cb23-11" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_d</span>(<span class="at">option =</span> <span class="st">"mako"</span>) <span class="sc">+</span></span>
<span id="cb23-12"><a href="#cb23-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb23-13"><a href="#cb23-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Red de Co-ocurrencia: Integrative Cluster"</span>,</span>
<span id="cb23-14"><a href="#cb23-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">"Análisis integrado de características genómicas y clínicas"</span>,</span>
<span id="cb23-15"><a href="#cb23-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="st">"Grado"</span>,</span>
<span id="cb23-16"><a href="#cb23-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"Comunidad"</span></span>
<span id="cb23-17"><a href="#cb23-17" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb23-18"><a href="#cb23-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_graph</span>() <span class="sc">+</span></span>
<span id="cb23-19"><a href="#cb23-19" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb23-20"><a href="#cb23-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">16</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb23-21"><a href="#cb23-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>)</span>
<span id="cb23-22"><a href="#cb23-22" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb23-23"><a href="#cb23-23" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-24"><a href="#cb23-24" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(p_red_intclust)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/viz_red_intclust-1.png" class="img-fluid figure-img" width="1344"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb24"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb24-1"><a href="#cb24-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"figuras_redes/red_intcluster_completa.png"</span>, p_red_intclust,</span>
<span id="cb24-2"><a href="#cb24-2" aria-hidden="true" tabindex="-1"></a>       <span class="at">width =</span> <span class="dv">14</span>, <span class="at">height =</span> <span class="dv">12</span>, <span class="at">dpi =</span> <span class="dv">300</span>, <span class="at">bg =</span> <span class="st">"white"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
<section id="tabla-top-nodos-centrales-integrative-cluster" class="level4" data-number="2.3.3.3">
<h4 data-number="2.3.3.3" class="anchored" data-anchor-id="tabla-top-nodos-centrales-integrative-cluster"><span class="header-section-number">2.3.3.3</span> Tabla: Top Nodos Centrales (Integrative Cluster)</h4>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a>centralidad_intclust <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_intclust)<span class="sc">$</span>name,</span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_intclust)<span class="sc">$</span>degree,</span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Intermediacion =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_intclust)<span class="sc">$</span>betweenness, <span class="dv">4</span>),</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">Eigenvector =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_intclust)<span class="sc">$</span>eigenvector, <span class="dv">4</span>),</span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_intclust)<span class="sc">$</span>community</span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(Grado)) <span class="sc">%&gt;%</span></span>
<span id="cb25-9"><a href="#cb25-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">head</span>(<span class="dv">20</span>)</span>
<span id="cb25-10"><a href="#cb25-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb25-11"><a href="#cb25-11" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb25-12"><a href="#cb25-12" aria-hidden="true" tabindex="-1"></a>  centralidad_intclust,</span>
<span id="cb25-13"><a href="#cb25-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Característica"</span>, <span class="st">"Grado"</span>, <span class="st">"Intermediación"</span>, <span class="st">"Eigenvector"</span>, <span class="st">"Comunidad"</span>),</span>
<span id="cb25-14"><a href="#cb25-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb25-15"><a href="#cb25-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">20</span>,</span>
<span id="cb25-16"><a href="#cb25-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb25-17"><a href="#cb25-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb25-18"><a href="#cb25-18" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb25-19"><a href="#cb25-19" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb25-20"><a href="#cb25-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Top 20 características más centrales en la red Integrative Cluster"</span></span>
<span id="cb25-21"><a href="#cb25-21" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb25-22"><a href="#cb25-22" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">c</span>(<span class="st">'Intermediacion'</span>, <span class="st">'Eigenvector'</span>), <span class="dv">4</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-e4542e95645e2bb45c0a" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-e4542e95645e2bb45c0a">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Top 20 características más centrales en la red Integrative Cluster<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20"],["HER2: Negative","ER: Positive","HistoSubtype: Ductal/NST","Menopause: Post","Grade: 3","Chemo: NO","PR: Negative","Surgery: MASTECTOMY","Cell: High","Radio: YES","Relapse: 0:Not Recurred","PR: Positive","Survival: 1:DECEASED","Hormone: YES","LymphGroup: N0","Relapse: 1:Recurred","Laterality: Left","Stage: 2","Laterality: Right","Grade: 2"],[47,43,43,41,40,40,39,39,39,38,38,38,37,36,36,35,35,34,34,34],[0.0186,0.008500000000000001,0.0005999999999999999,0.0147,0.0198,0.0192,0.0254,0.0056,0.0282,0.0023,0.0136,0.0333,0.009599999999999999,0.0011,0.013,0.0153,0.007900000000000001,0.0119,0.0023,0.0328],[1,0.9294,0.9227,0.9382,0.7167,0.9288999999999999,0.7036,0.7877,0.734,0.79,0.7991,0.759,0.7759,0.804,0.7503,0.6339,0.7161999999999999,0.6571,0.6804,0.6439],[1,1,2,1,2,1,2,2,2,2,1,1,2,1,3,2,1,2,3,1]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Característica<\/th>\n      <th>Grado<\/th>\n      <th>Intermediación<\/th>\n      <th>Eigenvector<\/th>\n      <th>Comunidad<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":20,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Caracteristica","targets":1},{"name":"Grado","targets":2},{"name":"Intermediacion","targets":3},{"name":"Eigenvector","targets":4},{"name":"Comunidad","targets":5}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,20,25,50,100]}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
</section>
<section id="comparación-de-redes" class="level2" data-number="2.4">
<h2 data-number="2.4" class="anchored" data-anchor-id="comparación-de-redes"><span class="header-section-number">2.4</span> Comparación de Redes</h2>
<section id="comparación-de-métricas-topológicas" class="level3" data-number="2.4.1">
<h3 data-number="2.4.1" class="anchored" data-anchor-id="comparación-de-métricas-topológicas"><span class="header-section-number">2.4.1</span> Comparación de Métricas Topológicas</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb26"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb26-1"><a href="#cb26-1" aria-hidden="true" tabindex="-1"></a>comparacion <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb26-2"><a href="#cb26-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Red =</span> <span class="fu">c</span>(<span class="st">"PAM50"</span>, <span class="st">"3-Gene"</span>, <span class="st">"IntCluster"</span>),</span>
<span id="cb26-3"><a href="#cb26-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Nodos =</span> <span class="fu">c</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">1</span>], metricas_3gene<span class="sc">$</span>Valor[<span class="dv">1</span>], metricas_intclust<span class="sc">$</span>Valor[<span class="dv">1</span>]),</span>
<span id="cb26-4"><a href="#cb26-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Aristas =</span> <span class="fu">c</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">2</span>], metricas_3gene<span class="sc">$</span>Valor[<span class="dv">2</span>], metricas_intclust<span class="sc">$</span>Valor[<span class="dv">2</span>]),</span>
<span id="cb26-5"><a href="#cb26-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">Densidad =</span> <span class="fu">c</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">3</span>], metricas_3gene<span class="sc">$</span>Valor[<span class="dv">3</span>], metricas_intclust<span class="sc">$</span>Valor[<span class="dv">3</span>]),</span>
<span id="cb26-6"><a href="#cb26-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">Clustering =</span> <span class="fu">c</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">4</span>], metricas_3gene<span class="sc">$</span>Valor[<span class="dv">4</span>], metricas_intclust<span class="sc">$</span>Valor[<span class="dv">4</span>]),</span>
<span id="cb26-7"><a href="#cb26-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidades =</span> <span class="fu">c</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">6</span>], metricas_3gene<span class="sc">$</span>Valor[<span class="dv">6</span>], metricas_intclust<span class="sc">$</span>Valor[<span class="dv">6</span>]),</span>
<span id="cb26-8"><a href="#cb26-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">Modularidad =</span> <span class="fu">c</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">7</span>], metricas_3gene<span class="sc">$</span>Valor[<span class="dv">7</span>], metricas_intclust<span class="sc">$</span>Valor[<span class="dv">7</span>])</span>
<span id="cb26-9"><a href="#cb26-9" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb26-10"><a href="#cb26-10" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb26-11"><a href="#cb26-11" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb26-12"><a href="#cb26-12" aria-hidden="true" tabindex="-1"></a>  comparacion,</span>
<span id="cb26-13"><a href="#cb26-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Red"</span>, <span class="st">"Nodos"</span>, <span class="st">"Aristas"</span>, <span class="st">"Densidad"</span>, <span class="st">"Clustering"</span>, <span class="st">"Comunidades"</span>, <span class="st">"Modularidad"</span>),</span>
<span id="cb26-14"><a href="#cb26-14" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb26-15"><a href="#cb26-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'t'</span>,</span>
<span id="cb26-16"><a href="#cb26-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">5</span></span>
<span id="cb26-17"><a href="#cb26-17" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb26-18"><a href="#cb26-18" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Comparación de métricas topológicas entre las tres redes"</span></span>
<span id="cb26-19"><a href="#cb26-19" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb26-20"><a href="#cb26-20" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">c</span>(<span class="st">'Densidad'</span>, <span class="st">'Clustering'</span>, <span class="st">'Modularidad'</span>), <span class="dv">4</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-228c3188fd654cd31936" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-228c3188fd654cd31936">{"x":{"filter":"none","vertical":false,"caption":"<caption>Comparación de métricas topológicas entre las tres redes<\/caption>","data":[["1","2","3"],["PAM50","3-Gene","IntCluster"],[57,53,61],[604,553,676],[0.3784,0.4013,0.3694],[0.76,0.7579,0.7498],[5,4,6],[0.1022,0.1047,0.1008]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Red<\/th>\n      <th>Nodos<\/th>\n      <th>Aristas<\/th>\n      <th>Densidad<\/th>\n      <th>Clustering<\/th>\n      <th>Comunidades<\/th>\n      <th>Modularidad<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","pageLength":5,"columnDefs":[{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Red","targets":1},{"name":"Nodos","targets":2},{"name":"Aristas","targets":3},{"name":"Densidad","targets":4},{"name":"Clustering","targets":5},{"name":"Comunidades","targets":6},{"name":"Modularidad","targets":7}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[5,10,25,50,100]}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="visualización-comparativa" class="level3" data-number="2.4.2">
<h3 data-number="2.4.2" class="anchored" data-anchor-id="visualización-comparativa"><span class="header-section-number">2.4.2</span> Visualización Comparativa</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a>comparacion_long <span class="ot">&lt;-</span> comparacion <span class="sc">%&gt;%</span></span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Red, Densidad, Clustering, Modularidad) <span class="sc">%&gt;%</span></span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_longer</span>(<span class="at">cols =</span> <span class="sc">-</span>Red, <span class="at">names_to =</span> <span class="st">"Metrica"</span>, <span class="at">values_to =</span> <span class="st">"Valor"</span>)</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>p_comp <span class="ot">&lt;-</span> <span class="fu">ggplot</span>(comparacion_long, <span class="fu">aes</span>(<span class="at">x =</span> Red, <span class="at">y =</span> Valor, <span class="at">fill =</span> Red)) <span class="sc">+</span></span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_bar</span>(<span class="at">stat =</span> <span class="st">"identity"</span>, <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">facet_wrap</span>(<span class="sc">~</span>Metrica, <span class="at">scales =</span> <span class="st">"free_y"</span>, <span class="at">ncol =</span> <span class="dv">3</span>) <span class="sc">+</span></span>
<span id="cb27-8"><a href="#cb27-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_fill_viridis_d</span>(<span class="at">option =</span> <span class="st">"rocket"</span>) <span class="sc">+</span></span>
<span id="cb27-9"><a href="#cb27-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb27-10"><a href="#cb27-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"Comparación de Métricas entre Redes"</span>,</span>
<span id="cb27-11"><a href="#cb27-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">"Densidad, Clustering Global y Modularidad"</span>,</span>
<span id="cb27-12"><a href="#cb27-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">x =</span> <span class="st">"Tipo de Red"</span>,</span>
<span id="cb27-13"><a href="#cb27-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">y =</span> <span class="st">"Valor"</span></span>
<span id="cb27-14"><a href="#cb27-14" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb27-15"><a href="#cb27-15" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_minimal</span>() <span class="sc">+</span></span>
<span id="cb27-16"><a href="#cb27-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb27-17"><a href="#cb27-17" aria-hidden="true" tabindex="-1"></a>    <span class="at">legend.position =</span> <span class="st">"none"</span>,</span>
<span id="cb27-18"><a href="#cb27-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">strip.text =</span> <span class="fu">element_text</span>(<span class="at">face =</span> <span class="st">"bold"</span>, <span class="at">size =</span> <span class="dv">11</span>),</span>
<span id="cb27-19"><a href="#cb27-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">14</span>, <span class="at">face =</span> <span class="st">"bold"</span>)</span>
<span id="cb27-20"><a href="#cb27-20" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb27-21"><a href="#cb27-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb27-22"><a href="#cb27-22" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(p_comp)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/viz_comparacion-1.png" class="img-fluid figure-img" width="1344"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb28"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb28-1"><a href="#cb28-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"figuras_redes/comparacion_metricas.png"</span>, p_comp,</span>
<span id="cb28-2"><a href="#cb28-2" aria-hidden="true" tabindex="-1"></a>       <span class="at">width =</span> <span class="dv">14</span>, <span class="at">height =</span> <span class="dv">8</span>, <span class="at">dpi =</span> <span class="dv">300</span>, <span class="at">bg =</span> <span class="st">"white"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
</section>
<section id="análisis-de-comunidades" class="level2" data-number="2.5">
<h2 data-number="2.5" class="anchored" data-anchor-id="análisis-de-comunidades"><span class="header-section-number">2.5</span> Análisis de Comunidades</h2>
<section id="comunidades-en-red-pam50" class="level3" data-number="2.5.1">
<h3 data-number="2.5.1" class="anchored" data-anchor-id="comunidades-en-red-pam50"><span class="header-section-number">2.5.1</span> Comunidades en Red PAM50</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb29"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb29-1"><a href="#cb29-1" aria-hidden="true" tabindex="-1"></a>comunidades_df_pam50 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb29-2"><a href="#cb29-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>name,</span>
<span id="cb29-3"><a href="#cb29-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>community,</span>
<span id="cb29-4"><a href="#cb29-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>degree</span>
<span id="cb29-5"><a href="#cb29-5" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb29-6"><a href="#cb29-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(Comunidad, <span class="fu">desc</span>(Grado))</span>
<span id="cb29-7"><a href="#cb29-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-8"><a href="#cb29-8" aria-hidden="true" tabindex="-1"></a>resumen_comunidades_pam50 <span class="ot">&lt;-</span> comunidades_df_pam50 <span class="sc">%&gt;%</span></span>
<span id="cb29-9"><a href="#cb29-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Comunidad) <span class="sc">%&gt;%</span></span>
<span id="cb29-10"><a href="#cb29-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb29-11"><a href="#cb29-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">N_Nodos =</span> <span class="fu">n</span>(),</span>
<span id="cb29-12"><a href="#cb29-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grado_Promedio =</span> <span class="fu">round</span>(<span class="fu">mean</span>(Grado), <span class="dv">2</span>),</span>
<span id="cb29-13"><a href="#cb29-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grado_Max =</span> <span class="fu">max</span>(Grado),</span>
<span id="cb29-14"><a href="#cb29-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">Principales =</span> <span class="fu">paste</span>(<span class="fu">head</span>(Caracteristica, <span class="dv">3</span>), <span class="at">collapse =</span> <span class="st">" | "</span>)</span>
<span id="cb29-15"><a href="#cb29-15" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb29-16"><a href="#cb29-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(N_Nodos))</span>
<span id="cb29-17"><a href="#cb29-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb29-18"><a href="#cb29-18" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb29-19"><a href="#cb29-19" aria-hidden="true" tabindex="-1"></a>  resumen_comunidades_pam50,</span>
<span id="cb29-20"><a href="#cb29-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Comunidad"</span>, <span class="st">"N° Nodos"</span>, <span class="st">"Grado Promedio"</span>, <span class="st">"Grado Máximo"</span>, <span class="st">"Principales Características"</span>),</span>
<span id="cb29-21"><a href="#cb29-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb29-22"><a href="#cb29-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb29-23"><a href="#cb29-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb29-24"><a href="#cb29-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb29-25"><a href="#cb29-25" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb29-26"><a href="#cb29-26" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb29-27"><a href="#cb29-27" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Análisis de comunidades detectadas en red PAM50"</span></span>
<span id="cb29-28"><a href="#cb29-28" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-6d47d2af2883d77a510c" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-6d47d2af2883d77a510c">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Análisis de comunidades detectadas en red PAM50<\/caption>","data":[["1","2","3","4","5"],[2,1,3,4,5],[22,18,12,3,2],[26.95,16.56,25.92,1.33,1],[43,44,35,2,1],["HER2: Negative | ER: Positive | Surgery: MASTECTOMY","HistoSubtype: Ductal/NST | Radio: YES | Grade: 3","Relapse: 0:Not Recurred | Laterality: Right | Survival: 0:LIVING","HistoSubtype: Unknown | LymphGroup: Unknown | Stage: 0","Surgery: Unknown | SizeGroup: Unknown"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Comunidad<\/th>\n      <th>N° Nodos<\/th>\n      <th>Grado Promedio<\/th>\n      <th>Grado Máximo<\/th>\n      <th>Principales Características<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Comunidad","targets":1},{"name":"N_Nodos","targets":2},{"name":"Grado_Promedio","targets":3},{"name":"Grado_Max","targets":4},{"name":"Principales","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="comunidades-en-red-3-gene" class="level3" data-number="2.5.2">
<h3 data-number="2.5.2" class="anchored" data-anchor-id="comunidades-en-red-3-gene"><span class="header-section-number">2.5.2</span> Comunidades en Red 3-Gene</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb30"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb30-1"><a href="#cb30-1" aria-hidden="true" tabindex="-1"></a>comunidades_df_3gene <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb30-2"><a href="#cb30-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_3gene)<span class="sc">$</span>name,</span>
<span id="cb30-3"><a href="#cb30-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_3gene)<span class="sc">$</span>community,</span>
<span id="cb30-4"><a href="#cb30-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_3gene)<span class="sc">$</span>degree</span>
<span id="cb30-5"><a href="#cb30-5" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb30-6"><a href="#cb30-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(Comunidad, <span class="fu">desc</span>(Grado))</span>
<span id="cb30-7"><a href="#cb30-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-8"><a href="#cb30-8" aria-hidden="true" tabindex="-1"></a>resumen_comunidades_3gene <span class="ot">&lt;-</span> comunidades_df_3gene <span class="sc">%&gt;%</span></span>
<span id="cb30-9"><a href="#cb30-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Comunidad) <span class="sc">%&gt;%</span></span>
<span id="cb30-10"><a href="#cb30-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb30-11"><a href="#cb30-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">N_Nodos =</span> <span class="fu">n</span>(),</span>
<span id="cb30-12"><a href="#cb30-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grado_Promedio =</span> <span class="fu">round</span>(<span class="fu">mean</span>(Grado), <span class="dv">2</span>),</span>
<span id="cb30-13"><a href="#cb30-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grado_Max =</span> <span class="fu">max</span>(Grado),</span>
<span id="cb30-14"><a href="#cb30-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">Principales =</span> <span class="fu">paste</span>(<span class="fu">head</span>(Caracteristica, <span class="dv">3</span>), <span class="at">collapse =</span> <span class="st">" | "</span>)</span>
<span id="cb30-15"><a href="#cb30-15" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb30-16"><a href="#cb30-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(N_Nodos))</span>
<span id="cb30-17"><a href="#cb30-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb30-18"><a href="#cb30-18" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb30-19"><a href="#cb30-19" aria-hidden="true" tabindex="-1"></a>  resumen_comunidades_3gene,</span>
<span id="cb30-20"><a href="#cb30-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Comunidad"</span>, <span class="st">"N° Nodos"</span>, <span class="st">"Grado Promedio"</span>, <span class="st">"Grado Máximo"</span>, <span class="st">"Principales Características"</span>),</span>
<span id="cb30-21"><a href="#cb30-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb30-22"><a href="#cb30-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb30-23"><a href="#cb30-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb30-24"><a href="#cb30-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb30-25"><a href="#cb30-25" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb30-26"><a href="#cb30-26" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb30-27"><a href="#cb30-27" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Análisis de comunidades detectadas en red 3-Gene"</span></span>
<span id="cb30-28"><a href="#cb30-28" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-4422ef0708f7efbb5e59" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-4422ef0708f7efbb5e59">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Análisis de comunidades detectadas en red 3-Gene<\/caption>","data":[["1","2","3","4"],[2,3,1,4],[20,18,13,2],[25.55,19.61,18.46,1],[36,43,44,1],["ER: Positive | Menopause: Post | Chemo: NO","HER2: Negative | Hormone: YES | Surgery: MASTECTOMY","HistoSubtype: Ductal/NST | Grade: 3 | Radio: YES","LymphGroup: Unknown | HistoSubtype: Unknown"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Comunidad<\/th>\n      <th>N° Nodos<\/th>\n      <th>Grado Promedio<\/th>\n      <th>Grado Máximo<\/th>\n      <th>Principales Características<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Comunidad","targets":1},{"name":"N_Nodos","targets":2},{"name":"Grado_Promedio","targets":3},{"name":"Grado_Max","targets":4},{"name":"Principales","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="comunidades-en-red-integrative-cluster" class="level3" data-number="2.5.3">
<h3 data-number="2.5.3" class="anchored" data-anchor-id="comunidades-en-red-integrative-cluster"><span class="header-section-number">2.5.3</span> Comunidades en Red Integrative Cluster</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb31"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb31-1"><a href="#cb31-1" aria-hidden="true" tabindex="-1"></a>comunidades_df_intclust <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb31-2"><a href="#cb31-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_intclust)<span class="sc">$</span>name,</span>
<span id="cb31-3"><a href="#cb31-3" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_intclust)<span class="sc">$</span>community,</span>
<span id="cb31-4"><a href="#cb31-4" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_intclust)<span class="sc">$</span>degree</span>
<span id="cb31-5"><a href="#cb31-5" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb31-6"><a href="#cb31-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(Comunidad, <span class="fu">desc</span>(Grado))</span>
<span id="cb31-7"><a href="#cb31-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-8"><a href="#cb31-8" aria-hidden="true" tabindex="-1"></a>resumen_comunidades_intclust <span class="ot">&lt;-</span> comunidades_df_intclust <span class="sc">%&gt;%</span></span>
<span id="cb31-9"><a href="#cb31-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Comunidad) <span class="sc">%&gt;%</span></span>
<span id="cb31-10"><a href="#cb31-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb31-11"><a href="#cb31-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">N_Nodos =</span> <span class="fu">n</span>(),</span>
<span id="cb31-12"><a href="#cb31-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grado_Promedio =</span> <span class="fu">round</span>(<span class="fu">mean</span>(Grado), <span class="dv">2</span>),</span>
<span id="cb31-13"><a href="#cb31-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grado_Max =</span> <span class="fu">max</span>(Grado),</span>
<span id="cb31-14"><a href="#cb31-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">Principales =</span> <span class="fu">paste</span>(<span class="fu">head</span>(Caracteristica, <span class="dv">3</span>), <span class="at">collapse =</span> <span class="st">" | "</span>)</span>
<span id="cb31-15"><a href="#cb31-15" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb31-16"><a href="#cb31-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(N_Nodos))</span>
<span id="cb31-17"><a href="#cb31-17" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb31-18"><a href="#cb31-18" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb31-19"><a href="#cb31-19" aria-hidden="true" tabindex="-1"></a>  resumen_comunidades_intclust,</span>
<span id="cb31-20"><a href="#cb31-20" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Comunidad"</span>, <span class="st">"N° Nodos"</span>, <span class="st">"Grado Promedio"</span>, <span class="st">"Grado Máximo"</span>, <span class="st">"Principales Características"</span>),</span>
<span id="cb31-21"><a href="#cb31-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb31-22"><a href="#cb31-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb31-23"><a href="#cb31-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb31-24"><a href="#cb31-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb31-25"><a href="#cb31-25" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb31-26"><a href="#cb31-26" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb31-27"><a href="#cb31-27" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Análisis de comunidades detectadas en red Integrative Cluster"</span></span>
<span id="cb31-28"><a href="#cb31-28" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-937099887b8c9ff72293" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-937099887b8c9ff72293">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Análisis de comunidades detectadas en red Integrative Cluster<\/caption>","data":[["1","2","3","4","5","6"],[1,2,3,5,4,6],[23,19,11,4,2,2],[24.83,25.42,26.09,1.5,1.5,1],[47,43,36,2,2,1],["HER2: Negative | ER: Positive | Menopause: Post","HistoSubtype: Ductal/NST | Grade: 3 | PR: Negative","LymphGroup: N0 | Laterality: Right | Hormone: NO","HistoSubtype: Unknown | Stage: 0 | LymphGroup: Unknown","Stage: 3 | SizeGroup: VeryLarge(&gt;50)","Surgery: Unknown | SizeGroup: Unknown"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Comunidad<\/th>\n      <th>N° Nodos<\/th>\n      <th>Grado Promedio<\/th>\n      <th>Grado Máximo<\/th>\n      <th>Principales Características<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"className":"dt-right","targets":[1,2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Comunidad","targets":1},{"name":"N_Nodos","targets":2},{"name":"Grado_Promedio","targets":3},{"name":"Grado_Max","targets":4},{"name":"Principales","targets":5}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="análisis-de-sub-redes-por-subtipo-específico" class="level2" data-number="2.6">
<h2 data-number="2.6" class="anchored" data-anchor-id="análisis-de-sub-redes-por-subtipo-específico"><span class="header-section-number">2.6</span> Análisis de Sub-redes por Subtipo Específico</h2>
<section id="comparación-luma-vs-lumb-pam50" class="level3" data-number="2.6.1">
<h3 data-number="2.6.1" class="anchored" data-anchor-id="comparación-luma-vs-lumb-pam50"><span class="header-section-number">2.6.1</span> Comparación: LumA vs LumB (PAM50)</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb32"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb32-1"><a href="#cb32-1" aria-hidden="true" tabindex="-1"></a>datos_lum <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb32-2"><a href="#cb32-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(PAM50 <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"LumA"</span>, <span class="st">"LumB"</span>))</span>
<span id="cb32-3"><a href="#cb32-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-4"><a href="#cb32-4" aria-hidden="true" tabindex="-1"></a>comp_lum <span class="ot">&lt;-</span> datos_lum <span class="sc">%&gt;%</span></span>
<span id="cb32-5"><a href="#cb32-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(PAM50, ER, PR, HER2, Grade, Stage, Chemo, Hormone, Radio) <span class="sc">%&gt;%</span></span>
<span id="cb32-6"><a href="#cb32-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(PAM50) <span class="sc">%&gt;%</span></span>
<span id="cb32-7"><a href="#cb32-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb32-8"><a href="#cb32-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">N =</span> <span class="fu">n</span>(),</span>
<span id="cb32-9"><a href="#cb32-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">ER_Pos =</span> <span class="fu">sum</span>(ER <span class="sc">==</span> <span class="st">"Positive"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-10"><a href="#cb32-10" aria-hidden="true" tabindex="-1"></a>    <span class="at">PR_Pos =</span> <span class="fu">sum</span>(PR <span class="sc">==</span> <span class="st">"Positive"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-11"><a href="#cb32-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">HER2_Pos =</span> <span class="fu">sum</span>(HER2 <span class="sc">==</span> <span class="st">"Positive"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-12"><a href="#cb32-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grade3 =</span> <span class="fu">sum</span>(Grade <span class="sc">==</span> <span class="st">"3"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-13"><a href="#cb32-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">Stage2 =</span> <span class="fu">sum</span>(Stage <span class="sc">==</span> <span class="st">"2"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-14"><a href="#cb32-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">Chemo_Yes =</span> <span class="fu">sum</span>(Chemo <span class="sc">==</span> <span class="st">"YES"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-15"><a href="#cb32-15" aria-hidden="true" tabindex="-1"></a>    <span class="at">Hormone_Yes =</span> <span class="fu">sum</span>(Hormone <span class="sc">==</span> <span class="st">"YES"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>),</span>
<span id="cb32-16"><a href="#cb32-16" aria-hidden="true" tabindex="-1"></a>    <span class="at">Radio_Yes =</span> <span class="fu">sum</span>(Radio <span class="sc">==</span> <span class="st">"YES"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>)</span>
<span id="cb32-17"><a href="#cb32-17" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb32-18"><a href="#cb32-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="fu">across</span>(ER_Pos<span class="sc">:</span>Radio_Yes, <span class="sc">~</span><span class="fu">round</span>(.<span class="sc">/</span>N <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>)))</span>
<span id="cb32-19"><a href="#cb32-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb32-20"><a href="#cb32-20" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb32-21"><a href="#cb32-21" aria-hidden="true" tabindex="-1"></a>  comp_lum,</span>
<span id="cb32-22"><a href="#cb32-22" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Subtipo"</span>, <span class="st">"N"</span>, <span class="st">"ER+ (%)"</span>, <span class="st">"PR+ (%)"</span>, <span class="st">"HER2+ (%)"</span>, </span>
<span id="cb32-23"><a href="#cb32-23" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Grado 3 (%)"</span>, <span class="st">"Estadio 2 (%)"</span>, <span class="st">"Quimio (%)"</span>, <span class="st">"Hormonal (%)"</span>, <span class="st">"Radio (%)"</span>),</span>
<span id="cb32-24"><a href="#cb32-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb32-25"><a href="#cb32-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'t'</span></span>
<span id="cb32-26"><a href="#cb32-26" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb32-27"><a href="#cb32-27" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Comparación de características entre subtipos Luminales A y B"</span></span>
<span id="cb32-28"><a href="#cb32-28" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb32-29"><a href="#cb32-29" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="dv">3</span><span class="sc">:</span><span class="dv">10</span>, <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-7327dddd40a9a7ad49ff" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-7327dddd40a9a7ad49ff">{"x":{"filter":"none","vertical":false,"caption":"<caption>Comparación de características entre subtipos Luminales A y B<\/caption>","data":[["1","2"],["LumA","LumB"],[700,475],[99.43000000000001,100],[77,62.95],[3,9.470000000000001],[24.57,53.26],[40.86,44.21],[8,9.890000000000001],[68.14,79.79000000000001],[54,62.95]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Subtipo<\/th>\n      <th>N<\/th>\n      <th>ER+ (%)<\/th>\n      <th>PR+ (%)<\/th>\n      <th>HER2+ (%)<\/th>\n      <th>Grado 3 (%)<\/th>\n      <th>Estadio 2 (%)<\/th>\n      <th>Quimio (%)<\/th>\n      <th>Hormonal (%)<\/th>\n      <th>Radio (%)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":8,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":9,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":10,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7,8,9,10]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"PAM50","targets":1},{"name":"N","targets":2},{"name":"ER_Pos","targets":3},{"name":"PR_Pos","targets":4},{"name":"HER2_Pos","targets":5},{"name":"Grade3","targets":6},{"name":"Stage2","targets":7},{"name":"Chemo_Yes","targets":8},{"name":"Hormone_Yes","targets":9},{"name":"Radio_Yes","targets":10}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render","options.columnDefs.6.render","options.columnDefs.7.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="comparación-triple-negativo-vs-her2" class="level3" data-number="2.6.2">
<h3 data-number="2.6.2" class="anchored" data-anchor-id="comparación-triple-negativo-vs-her2"><span class="header-section-number">2.6.2</span> Comparación: Triple Negativo vs HER2+</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb33"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb33-1"><a href="#cb33-1" aria-hidden="true" tabindex="-1"></a>datos_agresivos <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb33-2"><a href="#cb33-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb33-3"><a href="#cb33-3" aria-hidden="true" tabindex="-1"></a>    <span class="at">Triple_Neg =</span> (ER <span class="sc">==</span> <span class="st">"Negative"</span> <span class="sc">&amp;</span> PR <span class="sc">==</span> <span class="st">"Negative"</span> <span class="sc">&amp;</span> HER2 <span class="sc">==</span> <span class="st">"Negative"</span>),</span>
<span id="cb33-4"><a href="#cb33-4" aria-hidden="true" tabindex="-1"></a>    <span class="at">HER2_Pos =</span> (HER2 <span class="sc">==</span> <span class="st">"Positive"</span>)</span>
<span id="cb33-5"><a href="#cb33-5" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb33-6"><a href="#cb33-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Triple_Neg <span class="sc">|</span> HER2_Pos) <span class="sc">%&gt;%</span></span>
<span id="cb33-7"><a href="#cb33-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb33-8"><a href="#cb33-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Subtipo_Agresivo =</span> <span class="fu">case_when</span>(</span>
<span id="cb33-9"><a href="#cb33-9" aria-hidden="true" tabindex="-1"></a>      Triple_Neg <span class="sc">~</span> <span class="st">"Triple Negativo"</span>,</span>
<span id="cb33-10"><a href="#cb33-10" aria-hidden="true" tabindex="-1"></a>      HER2_Pos <span class="sc">~</span> <span class="st">"HER2+"</span>,</span>
<span id="cb33-11"><a href="#cb33-11" aria-hidden="true" tabindex="-1"></a>      <span class="cn">TRUE</span> <span class="sc">~</span> <span class="st">"Otro"</span></span>
<span id="cb33-12"><a href="#cb33-12" aria-hidden="true" tabindex="-1"></a>    )</span>
<span id="cb33-13"><a href="#cb33-13" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb33-14"><a href="#cb33-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-15"><a href="#cb33-15" aria-hidden="true" tabindex="-1"></a>comp_agresivos <span class="ot">&lt;-</span> datos_agresivos <span class="sc">%&gt;%</span></span>
<span id="cb33-16"><a href="#cb33-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">select</span>(Subtipo_Agresivo, Grade, Stage, Chemo, Radio, Survival, Relapse) <span class="sc">%&gt;%</span></span>
<span id="cb33-17"><a href="#cb33-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(Subtipo_Agresivo) <span class="sc">%&gt;%</span></span>
<span id="cb33-18"><a href="#cb33-18" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(</span>
<span id="cb33-19"><a href="#cb33-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">N =</span> <span class="fu">n</span>(),</span>
<span id="cb33-20"><a href="#cb33-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">Grade3 =</span> <span class="fu">sum</span>(Grade <span class="sc">==</span> <span class="st">"3"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>) <span class="sc">/</span> <span class="fu">n</span>() <span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb33-21"><a href="#cb33-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">Stage3_4 =</span> <span class="fu">sum</span>(Stage <span class="sc">%in%</span> <span class="fu">c</span>(<span class="st">"3"</span>, <span class="st">"4"</span>), <span class="at">na.rm =</span> <span class="cn">TRUE</span>) <span class="sc">/</span> <span class="fu">n</span>() <span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb33-22"><a href="#cb33-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">Chemo =</span> <span class="fu">sum</span>(Chemo <span class="sc">==</span> <span class="st">"YES"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>) <span class="sc">/</span> <span class="fu">n</span>() <span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb33-23"><a href="#cb33-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">Radio =</span> <span class="fu">sum</span>(Radio <span class="sc">==</span> <span class="st">"YES"</span>, <span class="at">na.rm =</span> <span class="cn">TRUE</span>) <span class="sc">/</span> <span class="fu">n</span>() <span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb33-24"><a href="#cb33-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">Deceased =</span> <span class="fu">sum</span>(<span class="fu">grepl</span>(<span class="st">"DECEASED"</span>, Survival), <span class="at">na.rm =</span> <span class="cn">TRUE</span>) <span class="sc">/</span> <span class="fu">n</span>() <span class="sc">*</span> <span class="dv">100</span>,</span>
<span id="cb33-25"><a href="#cb33-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">Recurred =</span> <span class="fu">sum</span>(<span class="fu">grepl</span>(<span class="st">"Recurred"</span>, Relapse), <span class="at">na.rm =</span> <span class="cn">TRUE</span>) <span class="sc">/</span> <span class="fu">n</span>() <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb33-26"><a href="#cb33-26" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb33-27"><a href="#cb33-27" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb33-28"><a href="#cb33-28" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb33-29"><a href="#cb33-29" aria-hidden="true" tabindex="-1"></a>  comp_agresivos,</span>
<span id="cb33-30"><a href="#cb33-30" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Subtipo"</span>, <span class="st">"N"</span>, <span class="st">"Grado 3 (%)"</span>, <span class="st">"Estadio 3-4 (%)"</span>, </span>
<span id="cb33-31"><a href="#cb33-31" aria-hidden="true" tabindex="-1"></a>               <span class="st">"Quimio (%)"</span>, <span class="st">"Radio (%)"</span>, <span class="st">"Fallecidos (%)"</span>, <span class="st">"Recurrencia (%)"</span>),</span>
<span id="cb33-32"><a href="#cb33-32" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb33-33"><a href="#cb33-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'t'</span></span>
<span id="cb33-34"><a href="#cb33-34" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb33-35"><a href="#cb33-35" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Comparación entre subtipos agresivos: Triple Negativo vs HER2+"</span></span>
<span id="cb33-36"><a href="#cb33-36" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb33-37"><a href="#cb33-37" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="dv">3</span><span class="sc">:</span><span class="dv">8</span>, <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-82c766b90854284292b8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-82c766b90854284292b8">{"x":{"filter":"none","vertical":false,"caption":"<caption>Comparación entre subtipos agresivos: Triple Negativo vs HER2+<\/caption>","data":[["1","2"],["HER2+","Triple Negativo"],[247,320],[73.27935222672065,83.125],[11.74089068825911,8.4375],[41.70040485829959,51.5625],[60.72874493927125,69.6875],[62.75303643724697,52.5],[100,100]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Subtipo<\/th>\n      <th>N<\/th>\n      <th>Grado 3 (%)<\/th>\n      <th>Estadio 3-4 (%)<\/th>\n      <th>Quimio (%)<\/th>\n      <th>Radio (%)<\/th>\n      <th>Fallecidos (%)<\/th>\n      <th>Recurrencia (%)<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":7,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":8,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6,7,8]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Subtipo_Agresivo","targets":1},{"name":"N","targets":2},{"name":"Grade3","targets":3},{"name":"Stage3_4","targets":4},{"name":"Chemo","targets":5},{"name":"Radio","targets":6},{"name":"Deceased","targets":7},{"name":"Recurred","targets":8}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render","options.columnDefs.3.render","options.columnDefs.4.render","options.columnDefs.5.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="análisis-de-k-core-1" class="level2" data-number="2.7">
<h2 data-number="2.7" class="anchored" data-anchor-id="análisis-de-k-core-1"><span class="header-section-number">2.7</span> Análisis de K-Core</h2>
<section id="k-core-en-red-pam50" class="level3" data-number="2.7.1">
<h3 data-number="2.7.1" class="anchored" data-anchor-id="k-core-en-red-pam50"><span class="header-section-number">2.7.1</span> K-Core en Red PAM50</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb34"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb34-1"><a href="#cb34-1" aria-hidden="true" tabindex="-1"></a>kcore_pam50 <span class="ot">&lt;-</span> <span class="fu">coreness</span>(red_pam50)</span>
<span id="cb34-2"><a href="#cb34-2" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>kcore <span class="ot">&lt;-</span> kcore_pam50</span>
<span id="cb34-3"><a href="#cb34-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-4"><a href="#cb34-4" aria-hidden="true" tabindex="-1"></a>kcore_summary_pam50 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb34-5"><a href="#cb34-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">K_Core =</span> <span class="fu">sort</span>(<span class="fu">unique</span>(kcore_pam50), <span class="at">decreasing =</span> <span class="cn">TRUE</span>)</span>
<span id="cb34-6"><a href="#cb34-6" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb34-7"><a href="#cb34-7" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb34-8"><a href="#cb34-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">N_Nodos =</span> <span class="fu">sapply</span>(K_Core, <span class="cf">function</span>(k) <span class="fu">sum</span>(kcore_pam50 <span class="sc">==</span> k)),</span>
<span id="cb34-9"><a href="#cb34-9" aria-hidden="true" tabindex="-1"></a>    <span class="at">Nodos =</span> <span class="fu">sapply</span>(K_Core, <span class="cf">function</span>(k) {</span>
<span id="cb34-10"><a href="#cb34-10" aria-hidden="true" tabindex="-1"></a>      nodos_k <span class="ot">&lt;-</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>name[kcore_pam50 <span class="sc">==</span> k]</span>
<span id="cb34-11"><a href="#cb34-11" aria-hidden="true" tabindex="-1"></a>      <span class="cf">if</span>(<span class="fu">length</span>(nodos_k) <span class="sc">&gt;</span> <span class="dv">5</span>) {</span>
<span id="cb34-12"><a href="#cb34-12" aria-hidden="true" tabindex="-1"></a>        <span class="fu">paste</span>(<span class="fu">paste</span>(<span class="fu">head</span>(nodos_k, <span class="dv">5</span>), <span class="at">collapse =</span> <span class="st">", "</span>), <span class="st">"..."</span>)</span>
<span id="cb34-13"><a href="#cb34-13" aria-hidden="true" tabindex="-1"></a>      } <span class="cf">else</span> {</span>
<span id="cb34-14"><a href="#cb34-14" aria-hidden="true" tabindex="-1"></a>        <span class="fu">paste</span>(nodos_k, <span class="at">collapse =</span> <span class="st">", "</span>)</span>
<span id="cb34-15"><a href="#cb34-15" aria-hidden="true" tabindex="-1"></a>      }</span>
<span id="cb34-16"><a href="#cb34-16" aria-hidden="true" tabindex="-1"></a>    })</span>
<span id="cb34-17"><a href="#cb34-17" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb34-18"><a href="#cb34-18" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb34-19"><a href="#cb34-19" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb34-20"><a href="#cb34-20" aria-hidden="true" tabindex="-1"></a>  kcore_summary_pam50,</span>
<span id="cb34-21"><a href="#cb34-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"K-Core"</span>, <span class="st">"N° Nodos"</span>, <span class="st">"Nodos en K-Core"</span>),</span>
<span id="cb34-22"><a href="#cb34-22" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb34-23"><a href="#cb34-23" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb34-24"><a href="#cb34-24" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb34-25"><a href="#cb34-25" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb34-26"><a href="#cb34-26" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb34-27"><a href="#cb34-27" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb34-28"><a href="#cb34-28" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Análisis de K-Core en red PAM50"</span></span>
<span id="cb34-29"><a href="#cb34-29" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-fca0e4649f353a88771b" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-fca0e4649f353a88771b">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Análisis de K-Core en red PAM50<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17"],[24,23,20,19,18,16,15,12,11,10,7,6,5,4,3,2,1],[28,1,2,1,2,2,1,1,1,1,2,1,2,1,2,1,8],["ER: Positive, PR: Negative, HER2: Negative, Grade: 3, Stage: 2 ...","Hormone: NO","AgeGroup: &gt;70, Stage: 1","PAM50: LumB","AgeGroup: 50-60, Stage: Unknown","Menopause: Pre, ER: Negative","Chemo: YES","LymphGroup: N&gt;3","SizeGroup: Large(30-50)","LymphGroup: N1","AgeGroup: 40-50, LymphGroup: N2-3","PAM50: Basal","PAM50: Her2, HER2: Positive","HistoSubtype: Mixed","PAM50: claudin-low, AgeGroup: &lt;40","Grade: 1","Stage: 3, SizeGroup: VeryLarge(&gt;50), LymphGroup: Unknown, Laterality: Unknown, HistoSubtype: Unknown ..."]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>K-Core<\/th>\n      <th>N° Nodos<\/th>\n      <th>Nodos en K-Core<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"className":"dt-right","targets":[1,2]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"K_Core","targets":1},{"name":"N_Nodos","targets":2},{"name":"Nodos","targets":3}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="visualización-k-core-pam50" class="level3" data-number="2.7.2">
<h3 data-number="2.7.2" class="anchored" data-anchor-id="visualización-k-core-pam50"><span class="header-section-number">2.7.2</span> Visualización K-Core PAM50</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb35"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb35-1"><a href="#cb35-1" aria-hidden="true" tabindex="-1"></a>p_kcore_pam50 <span class="ot">&lt;-</span> <span class="fu">ggraph</span>(red_pam50, <span class="at">layout =</span> layout_fr) <span class="sc">+</span></span>
<span id="cb35-2"><a href="#cb35-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_edge_link</span>(<span class="at">alpha =</span> <span class="fl">0.3</span>, <span class="at">color =</span> <span class="st">"gray70"</span>) <span class="sc">+</span></span>
<span id="cb35-3"><a href="#cb35-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_point</span>(<span class="fu">aes</span>(<span class="at">size =</span> degree, <span class="at">color =</span> kcore), <span class="at">alpha =</span> <span class="fl">0.8</span>) <span class="sc">+</span></span>
<span id="cb35-4"><a href="#cb35-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">geom_node_text</span>(</span>
<span id="cb35-5"><a href="#cb35-5" aria-hidden="true" tabindex="-1"></a>    <span class="fu">aes</span>(<span class="at">label =</span> <span class="fu">ifelse</span>(kcore <span class="sc">&gt;=</span> <span class="fu">quantile</span>(kcore, <span class="fl">0.8</span>), name, <span class="st">""</span>)),</span>
<span id="cb35-6"><a href="#cb35-6" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="dv">3</span>, <span class="at">repel =</span> <span class="cn">TRUE</span>, <span class="at">max.overlaps =</span> <span class="dv">15</span></span>
<span id="cb35-7"><a href="#cb35-7" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb35-8"><a href="#cb35-8" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_size_continuous</span>(<span class="at">range =</span> <span class="fu">c</span>(<span class="dv">2</span>, <span class="dv">10</span>)) <span class="sc">+</span></span>
<span id="cb35-9"><a href="#cb35-9" aria-hidden="true" tabindex="-1"></a>  <span class="fu">scale_color_viridis_c</span>(<span class="at">option =</span> <span class="st">"inferno"</span>) <span class="sc">+</span></span>
<span id="cb35-10"><a href="#cb35-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">labs</span>(</span>
<span id="cb35-11"><a href="#cb35-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">title =</span> <span class="st">"K-Core Analysis: Red PAM50"</span>,</span>
<span id="cb35-12"><a href="#cb35-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">subtitle =</span> <span class="st">"Núcleo denso de la red (nodos más interconectados)"</span>,</span>
<span id="cb35-13"><a href="#cb35-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">size =</span> <span class="st">"Grado"</span>,</span>
<span id="cb35-14"><a href="#cb35-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">color =</span> <span class="st">"K-Core"</span></span>
<span id="cb35-15"><a href="#cb35-15" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">+</span></span>
<span id="cb35-16"><a href="#cb35-16" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme_graph</span>() <span class="sc">+</span></span>
<span id="cb35-17"><a href="#cb35-17" aria-hidden="true" tabindex="-1"></a>  <span class="fu">theme</span>(</span>
<span id="cb35-18"><a href="#cb35-18" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.title =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">16</span>, <span class="at">face =</span> <span class="st">"bold"</span>),</span>
<span id="cb35-19"><a href="#cb35-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">plot.subtitle =</span> <span class="fu">element_text</span>(<span class="at">size =</span> <span class="dv">12</span>)</span>
<span id="cb35-20"><a href="#cb35-20" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb35-21"><a href="#cb35-21" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb35-22"><a href="#cb35-22" aria-hidden="true" tabindex="-1"></a><span class="fu">print</span>(p_kcore_pam50)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div>
<figure class="figure">
<p><img src="index_files/figure-html/viz_kcore_pam50-1.png" class="img-fluid figure-img" width="1344"></p>
</figure>
</div>
</div>
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb36"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb36-1"><a href="#cb36-1" aria-hidden="true" tabindex="-1"></a><span class="fu">ggsave</span>(<span class="st">"figuras_redes/kcore_pam50.png"</span>, p_kcore_pam50,</span>
<span id="cb36-2"><a href="#cb36-2" aria-hidden="true" tabindex="-1"></a>       <span class="at">width =</span> <span class="dv">14</span>, <span class="at">height =</span> <span class="dv">10</span>, <span class="at">dpi =</span> <span class="dv">300</span>, <span class="at">bg =</span> <span class="st">"white"</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
</div>
</section>
</section>
<section id="identificación-de-nodos-hub-y-puente-1" class="level2" data-number="2.8">
<h2 data-number="2.8" class="anchored" data-anchor-id="identificación-de-nodos-hub-y-puente-1"><span class="header-section-number">2.8</span> Identificación de Nodos Hub y Puente</h2>
<section id="gateway-score-nodos-puente" class="level3" data-number="2.8.1">
<h3 data-number="2.8.1" class="anchored" data-anchor-id="gateway-score-nodos-puente"><span class="header-section-number">2.8.1</span> Gateway Score (Nodos Puente)</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb37"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb37-1"><a href="#cb37-1" aria-hidden="true" tabindex="-1"></a>calcular_gateway <span class="ot">&lt;-</span> <span class="cf">function</span>(grafo) {</span>
<span id="cb37-2"><a href="#cb37-2" aria-hidden="true" tabindex="-1"></a>  gateway_scores <span class="ot">&lt;-</span> <span class="fu">sapply</span>(<span class="fu">V</span>(grafo), <span class="cf">function</span>(v) {</span>
<span id="cb37-3"><a href="#cb37-3" aria-hidden="true" tabindex="-1"></a>    vecinos <span class="ot">&lt;-</span> <span class="fu">neighbors</span>(grafo, v)</span>
<span id="cb37-4"><a href="#cb37-4" aria-hidden="true" tabindex="-1"></a>    <span class="cf">if</span>(<span class="fu">length</span>(vecinos) <span class="sc">==</span> <span class="dv">0</span>) <span class="fu">return</span>(<span class="dv">0</span>)</span>
<span id="cb37-5"><a href="#cb37-5" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb37-6"><a href="#cb37-6" aria-hidden="true" tabindex="-1"></a>    comunidad_v <span class="ot">&lt;-</span> <span class="fu">V</span>(grafo)<span class="sc">$</span>community[v]</span>
<span id="cb37-7"><a href="#cb37-7" aria-hidden="true" tabindex="-1"></a>    n_comunidades_vecinas <span class="ot">&lt;-</span> <span class="fu">length</span>(<span class="fu">unique</span>(<span class="fu">V</span>(grafo)<span class="sc">$</span>community[vecinos]))</span>
<span id="cb37-8"><a href="#cb37-8" aria-hidden="true" tabindex="-1"></a>    prop_otras_comunidades <span class="ot">&lt;-</span> <span class="fu">sum</span>(<span class="fu">V</span>(grafo)<span class="sc">$</span>community[vecinos] <span class="sc">!=</span> comunidad_v) <span class="sc">/</span> <span class="fu">length</span>(vecinos)</span>
<span id="cb37-9"><a href="#cb37-9" aria-hidden="true" tabindex="-1"></a>    </span>
<span id="cb37-10"><a href="#cb37-10" aria-hidden="true" tabindex="-1"></a>    gateway <span class="ot">&lt;-</span> n_comunidades_vecinas <span class="sc">*</span> prop_otras_comunidades <span class="sc">*</span> <span class="fu">length</span>(vecinos)</span>
<span id="cb37-11"><a href="#cb37-11" aria-hidden="true" tabindex="-1"></a>    <span class="fu">return</span>(gateway)</span>
<span id="cb37-12"><a href="#cb37-12" aria-hidden="true" tabindex="-1"></a>  })</span>
<span id="cb37-13"><a href="#cb37-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">return</span>(gateway_scores)</span>
<span id="cb37-14"><a href="#cb37-14" aria-hidden="true" tabindex="-1"></a>}</span>
<span id="cb37-15"><a href="#cb37-15" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-16"><a href="#cb37-16" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_pam50)<span class="sc">$</span>gateway <span class="ot">&lt;-</span> <span class="fu">calcular_gateway</span>(red_pam50)</span>
<span id="cb37-17"><a href="#cb37-17" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_3gene)<span class="sc">$</span>gateway <span class="ot">&lt;-</span> <span class="fu">calcular_gateway</span>(red_3gene)</span>
<span id="cb37-18"><a href="#cb37-18" aria-hidden="true" tabindex="-1"></a><span class="fu">V</span>(red_intclust)<span class="sc">$</span>gateway <span class="ot">&lt;-</span> <span class="fu">calcular_gateway</span>(red_intclust)</span>
<span id="cb37-19"><a href="#cb37-19" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-20"><a href="#cb37-20" aria-hidden="true" tabindex="-1"></a>gateway_pam50 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb37-21"><a href="#cb37-21" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>name,</span>
<span id="cb37-22"><a href="#cb37-22" aria-hidden="true" tabindex="-1"></a>  <span class="at">Gateway_Score =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>gateway, <span class="dv">3</span>),</span>
<span id="cb37-23"><a href="#cb37-23" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>degree,</span>
<span id="cb37-24"><a href="#cb37-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>community</span>
<span id="cb37-25"><a href="#cb37-25" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb37-26"><a href="#cb37-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(Gateway_Score)) <span class="sc">%&gt;%</span></span>
<span id="cb37-27"><a href="#cb37-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">head</span>(<span class="dv">15</span>)</span>
<span id="cb37-28"><a href="#cb37-28" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb37-29"><a href="#cb37-29" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb37-30"><a href="#cb37-30" aria-hidden="true" tabindex="-1"></a>  gateway_pam50,</span>
<span id="cb37-31"><a href="#cb37-31" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Característica"</span>, <span class="st">"Gateway Score"</span>, <span class="st">"Grado"</span>, <span class="st">"Comunidad"</span>),</span>
<span id="cb37-32"><a href="#cb37-32" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb37-33"><a href="#cb37-33" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">15</span>,</span>
<span id="cb37-34"><a href="#cb37-34" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb37-35"><a href="#cb37-35" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb37-36"><a href="#cb37-36" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb37-37"><a href="#cb37-37" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb37-38"><a href="#cb37-38" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Top 15 nodos puente (Gateway) en red PAM50 - Conectan diferentes comunidades"</span></span>
<span id="cb37-39"><a href="#cb37-39" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb37-40"><a href="#cb37-40" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="st">'Gateway_Score'</span>, <span class="dv">3</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-2b73587f3120a5d35ff6" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-2b73587f3120a5d35ff6">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Top 15 nodos puente (Gateway) en red PAM50 - Conectan diferentes comunidades<\/caption>","data":[["1","2","3","4","5","6","7","8","9","10","11","12","13","14","15"],["HistoSubtype: Ductal/NST","Radio: YES","Grade: 3","Relapse: 0:Not Recurred","PR: Negative","HER2: Negative","Laterality: Right","Survival: 0:LIVING","LymphGroup: N0","Surgery: BREAST CONSERVING","Hormone: NO","Surgery: MASTECTOMY","SizeGroup: Small(&lt;20mm)","Cell: Moderate","ER: Positive"],[93,84,78,75,69,69,69,66,66,60,60,57,57,57,54],[44,38,37,35,35,43,32,32,32,30,29,38,29,28,38],[1,1,1,3,1,2,3,3,3,3,1,2,3,3,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Característica<\/th>\n      <th>Gateway Score<\/th>\n      <th>Grado<\/th>\n      <th>Comunidad<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":15,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":2,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 3, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Caracteristica","targets":1},{"name":"Gateway_Score","targets":2},{"name":"Grado","targets":3},{"name":"Comunidad","targets":4}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,15,25,50,100]}},"evals":["options.columnDefs.0.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="nodos-hub-alto-grado-y-centralidad" class="level3" data-number="2.8.2">
<h3 data-number="2.8.2" class="anchored" data-anchor-id="nodos-hub-alto-grado-y-centralidad"><span class="header-section-number">2.8.2</span> Nodos Hub (Alto Grado y Centralidad)</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb38"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb38-1"><a href="#cb38-1" aria-hidden="true" tabindex="-1"></a>umbral_grado <span class="ot">&lt;-</span> <span class="fu">quantile</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>degree, <span class="fl">0.90</span>)</span>
<span id="cb38-2"><a href="#cb38-2" aria-hidden="true" tabindex="-1"></a>umbral_eigen <span class="ot">&lt;-</span> <span class="fu">quantile</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>eigenvector, <span class="fl">0.90</span>)</span>
<span id="cb38-3"><a href="#cb38-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-4"><a href="#cb38-4" aria-hidden="true" tabindex="-1"></a>hubs_pam50 <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb38-5"><a href="#cb38-5" aria-hidden="true" tabindex="-1"></a>  <span class="at">Caracteristica =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>name,</span>
<span id="cb38-6"><a href="#cb38-6" aria-hidden="true" tabindex="-1"></a>  <span class="at">Grado =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>degree,</span>
<span id="cb38-7"><a href="#cb38-7" aria-hidden="true" tabindex="-1"></a>  <span class="at">Eigenvector =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>eigenvector, <span class="dv">4</span>),</span>
<span id="cb38-8"><a href="#cb38-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">Betweenness =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>betweenness, <span class="dv">4</span>),</span>
<span id="cb38-9"><a href="#cb38-9" aria-hidden="true" tabindex="-1"></a>  <span class="at">Gateway =</span> <span class="fu">round</span>(<span class="fu">V</span>(red_pam50)<span class="sc">$</span>gateway, <span class="dv">3</span>),</span>
<span id="cb38-10"><a href="#cb38-10" aria-hidden="true" tabindex="-1"></a>  <span class="at">Comunidad =</span> <span class="fu">V</span>(red_pam50)<span class="sc">$</span>community</span>
<span id="cb38-11"><a href="#cb38-11" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb38-12"><a href="#cb38-12" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(Grado <span class="sc">&gt;=</span> umbral_grado <span class="sc">&amp;</span> Eigenvector <span class="sc">&gt;=</span> umbral_eigen) <span class="sc">%&gt;%</span></span>
<span id="cb38-13"><a href="#cb38-13" aria-hidden="true" tabindex="-1"></a>  <span class="fu">arrange</span>(<span class="fu">desc</span>(Grado))</span>
<span id="cb38-14"><a href="#cb38-14" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb38-15"><a href="#cb38-15" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb38-16"><a href="#cb38-16" aria-hidden="true" tabindex="-1"></a>  hubs_pam50,</span>
<span id="cb38-17"><a href="#cb38-17" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Característica"</span>, <span class="st">"Grado"</span>, <span class="st">"Eigenvector"</span>, <span class="st">"Betweenness"</span>, <span class="st">"Gateway"</span>, <span class="st">"Comunidad"</span>),</span>
<span id="cb38-18"><a href="#cb38-18" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb38-19"><a href="#cb38-19" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">15</span>,</span>
<span id="cb38-20"><a href="#cb38-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb38-21"><a href="#cb38-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb38-22"><a href="#cb38-22" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb38-23"><a href="#cb38-23" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb38-24"><a href="#cb38-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Nodos Hub en red PAM50 - Alta centralidad y conectividad"</span></span>
<span id="cb38-25"><a href="#cb38-25" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb38-26"><a href="#cb38-26" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">c</span>(<span class="st">'Eigenvector'</span>, <span class="st">'Betweenness'</span>, <span class="st">'Gateway'</span>), <span class="dv">4</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-dafe00a791682e7c494e" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-dafe00a791682e7c494e">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Nodos Hub en red PAM50 - Alta centralidad y conectividad<\/caption>","data":[["1","2","3","4"],["HistoSubtype: Ductal/NST","HER2: Negative","ER: Positive","Menopause: Post"],[44,43,38,37],[0.9298999999999999,1,0.9318,0.9442],[0.026,0.0195,0.0234,0.0019],[93,69,54,54],[1,2,2,2]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Característica<\/th>\n      <th>Grado<\/th>\n      <th>Eigenvector<\/th>\n      <th>Betweenness<\/th>\n      <th>Gateway<\/th>\n      <th>Comunidad<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":15,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":3,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":4,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 4, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Caracteristica","targets":1},{"name":"Grado","targets":2},{"name":"Eigenvector","targets":3},{"name":"Betweenness","targets":4},{"name":"Gateway","targets":5},{"name":"Comunidad","targets":6}],"order":[],"autoWidth":false,"orderClasses":false,"lengthMenu":[10,15,25,50,100]}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render","options.columnDefs.2.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="análisis-de-supervivencia-por-subtipo" class="level2" data-number="2.9">
<h2 data-number="2.9" class="anchored" data-anchor-id="análisis-de-supervivencia-por-subtipo"><span class="header-section-number">2.9</span> Análisis de Supervivencia por Subtipo</h2>
<section id="supervivencia-por-pam50" class="level3" data-number="2.9.1">
<h3 data-number="2.9.1" class="anchored" data-anchor-id="supervivencia-por-pam50"><span class="header-section-number">2.9.1</span> Supervivencia por PAM50</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb39"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb39-1"><a href="#cb39-1" aria-hidden="true" tabindex="-1"></a>survival_pam50 <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb39-2"><a href="#cb39-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(PAM50) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(Survival)) <span class="sc">%&gt;%</span></span>
<span id="cb39-3"><a href="#cb39-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(PAM50, Survival) <span class="sc">%&gt;%</span></span>
<span id="cb39-4"><a href="#cb39-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">N =</span> <span class="fu">n</span>(), <span class="at">.groups =</span> <span class="st">"drop"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb39-5"><a href="#cb39-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(PAM50) <span class="sc">%&gt;%</span></span>
<span id="cb39-6"><a href="#cb39-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb39-7"><a href="#cb39-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total =</span> <span class="fu">sum</span>(N),</span>
<span id="cb39-8"><a href="#cb39-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Porcentaje =</span> <span class="fu">round</span>(N <span class="sc">/</span> Total <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>)</span>
<span id="cb39-9"><a href="#cb39-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb39-10"><a href="#cb39-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(</span>
<span id="cb39-11"><a href="#cb39-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">id_cols =</span> <span class="fu">c</span>(PAM50, Total),</span>
<span id="cb39-12"><a href="#cb39-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_from =</span> Survival,</span>
<span id="cb39-13"><a href="#cb39-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_from =</span> <span class="fu">c</span>(N, Porcentaje),</span>
<span id="cb39-14"><a href="#cb39-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_fill =</span> <span class="dv">0</span></span>
<span id="cb39-15"><a href="#cb39-15" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb39-16"><a href="#cb39-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb39-17"><a href="#cb39-17" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb39-18"><a href="#cb39-18" aria-hidden="true" tabindex="-1"></a>  survival_pam50,</span>
<span id="cb39-19"><a href="#cb39-19" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb39-20"><a href="#cb39-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb39-21"><a href="#cb39-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb39-22"><a href="#cb39-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb39-23"><a href="#cb39-23" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb39-24"><a href="#cb39-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb39-25"><a href="#cb39-25" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Distribución de supervivencia por subtipo PAM50"</span></span>
<span id="cb39-26"><a href="#cb39-26" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb39-27"><a href="#cb39-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">grep</span>(<span class="st">"Porcentaje"</span>, <span class="fu">names</span>(survival_pam50)), <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-618600f0528631fdb0c2" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-618600f0528631fdb0c2">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Distribución de supervivencia por subtipo PAM50<\/caption>","data":[["1","2","3","4","5","6","7"],["Basal","Her2","LumA","LumB","NC","Normal","claudin-low"],[209,224,700,475,6,148,218],[94,67,322,160,1,69,124],[115,157,378,315,5,79,94],[44.98,29.91,46,33.68,16.67,46.62,56.88],[55.02,70.09,54,66.31999999999999,83.33,53.38,43.12]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>PAM50<\/th>\n      <th>Total<\/th>\n      <th>N_0:LIVING<\/th>\n      <th>N_1:DECEASED<\/th>\n      <th>Porcentaje_0:LIVING<\/th>\n      <th>Porcentaje_1:DECEASED<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"PAM50","targets":1},{"name":"Total","targets":2},{"name":"N_0:LIVING","targets":3},{"name":"N_1:DECEASED","targets":4},{"name":"Porcentaje_0:LIVING","targets":5},{"name":"Porcentaje_1:DECEASED","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="recurrencia-por-subtipo" class="level3" data-number="2.9.2">
<h3 data-number="2.9.2" class="anchored" data-anchor-id="recurrencia-por-subtipo"><span class="header-section-number">2.9.2</span> Recurrencia por Subtipo</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb40"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb40-1"><a href="#cb40-1" aria-hidden="true" tabindex="-1"></a>recurrence_pam50 <span class="ot">&lt;-</span> datos_red <span class="sc">%&gt;%</span></span>
<span id="cb40-2"><a href="#cb40-2" aria-hidden="true" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(PAM50) <span class="sc">&amp;</span> <span class="sc">!</span><span class="fu">is.na</span>(Relapse)) <span class="sc">%&gt;%</span></span>
<span id="cb40-3"><a href="#cb40-3" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(PAM50, Relapse) <span class="sc">%&gt;%</span></span>
<span id="cb40-4"><a href="#cb40-4" aria-hidden="true" tabindex="-1"></a>  <span class="fu">summarise</span>(<span class="at">N =</span> <span class="fu">n</span>(), <span class="at">.groups =</span> <span class="st">"drop"</span>) <span class="sc">%&gt;%</span></span>
<span id="cb40-5"><a href="#cb40-5" aria-hidden="true" tabindex="-1"></a>  <span class="fu">group_by</span>(PAM50) <span class="sc">%&gt;%</span></span>
<span id="cb40-6"><a href="#cb40-6" aria-hidden="true" tabindex="-1"></a>  <span class="fu">mutate</span>(</span>
<span id="cb40-7"><a href="#cb40-7" aria-hidden="true" tabindex="-1"></a>    <span class="at">Total =</span> <span class="fu">sum</span>(N),</span>
<span id="cb40-8"><a href="#cb40-8" aria-hidden="true" tabindex="-1"></a>    <span class="at">Porcentaje =</span> <span class="fu">round</span>(N <span class="sc">/</span> Total <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">2</span>)</span>
<span id="cb40-9"><a href="#cb40-9" aria-hidden="true" tabindex="-1"></a>  ) <span class="sc">%&gt;%</span></span>
<span id="cb40-10"><a href="#cb40-10" aria-hidden="true" tabindex="-1"></a>  <span class="fu">pivot_wider</span>(</span>
<span id="cb40-11"><a href="#cb40-11" aria-hidden="true" tabindex="-1"></a>    <span class="at">id_cols =</span> <span class="fu">c</span>(PAM50, Total),</span>
<span id="cb40-12"><a href="#cb40-12" aria-hidden="true" tabindex="-1"></a>    <span class="at">names_from =</span> Relapse,</span>
<span id="cb40-13"><a href="#cb40-13" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_from =</span> <span class="fu">c</span>(N, Porcentaje),</span>
<span id="cb40-14"><a href="#cb40-14" aria-hidden="true" tabindex="-1"></a>    <span class="at">values_fill =</span> <span class="dv">0</span></span>
<span id="cb40-15"><a href="#cb40-15" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb40-16"><a href="#cb40-16" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb40-17"><a href="#cb40-17" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb40-18"><a href="#cb40-18" aria-hidden="true" tabindex="-1"></a>  recurrence_pam50,</span>
<span id="cb40-19"><a href="#cb40-19" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb40-20"><a href="#cb40-20" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span>,</span>
<span id="cb40-21"><a href="#cb40-21" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'Bfrtip'</span>,</span>
<span id="cb40-22"><a href="#cb40-22" aria-hidden="true" tabindex="-1"></a>    <span class="at">buttons =</span> <span class="fu">c</span>(<span class="st">'copy'</span>, <span class="st">'csv'</span>, <span class="st">'excel'</span>)</span>
<span id="cb40-23"><a href="#cb40-23" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb40-24"><a href="#cb40-24" aria-hidden="true" tabindex="-1"></a>  <span class="at">extensions =</span> <span class="st">'Buttons'</span>,</span>
<span id="cb40-25"><a href="#cb40-25" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Tasa de recurrencia por subtipo PAM50"</span></span>
<span id="cb40-26"><a href="#cb40-26" aria-hidden="true" tabindex="-1"></a>) <span class="sc">%&gt;%</span></span>
<span id="cb40-27"><a href="#cb40-27" aria-hidden="true" tabindex="-1"></a>  <span class="fu">formatRound</span>(<span class="fu">grep</span>(<span class="st">"Porcentaje"</span>, <span class="fu">names</span>(recurrence_pam50)), <span class="dv">2</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-7ad3399ecca0efeaa8c8" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-7ad3399ecca0efeaa8c8">{"x":{"filter":"none","vertical":false,"extensions":["Buttons"],"caption":"<caption>Tasa de recurrencia por subtipo PAM50<\/caption>","data":[["1","2","3","4","5","6","7"],["Basal","Her2","LumA","LumB","NC","Normal","claudin-low"],[209,224,699,475,6,148,218],[116,108,468,248,3,83,150],[93,116,231,227,3,65,68],[55.5,48.21,66.95,52.21,50,56.08,68.81],[44.5,51.79,33.05,47.79,50,43.92,31.19]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>PAM50<\/th>\n      <th>Total<\/th>\n      <th>N_0:Not Recurred<\/th>\n      <th>N_1:Recurred<\/th>\n      <th>Porcentaje_0:Not Recurred<\/th>\n      <th>Porcentaje_1:Recurred<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"pageLength":10,"dom":"Bfrtip","buttons":["copy","csv","excel"],"columnDefs":[{"targets":5,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"targets":6,"render":"function(data, type, row, meta) {\n    return type !== 'display' ? data : DTWidget.formatRound(data, 2, 3, \",\", \".\", null);\n  }"},{"className":"dt-right","targets":[2,3,4,5,6]},{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"PAM50","targets":1},{"name":"Total","targets":2},{"name":"N_0:Not Recurred","targets":3},{"name":"N_1:Recurred","targets":4},{"name":"Porcentaje_0:Not Recurred","targets":5},{"name":"Porcentaje_1:Recurred","targets":6}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":["options.columnDefs.0.render","options.columnDefs.1.render"],"jsHooks":[]}</script>
</div>
</div>
</section>
</section>
<section id="resumen-ejecutivo" class="level2" data-number="2.10">
<h2 data-number="2.10" class="anchored" data-anchor-id="resumen-ejecutivo"><span class="header-section-number">2.10</span> Resumen Ejecutivo</h2>
<section id="hallazgos-principales" class="level3" data-number="2.10.1">
<h3 data-number="2.10.1" class="anchored" data-anchor-id="hallazgos-principales"><span class="header-section-number">2.10.1</span> Hallazgos Principales</h3>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb41"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb41-1"><a href="#cb41-1" aria-hidden="true" tabindex="-1"></a>hallazgos <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb41-2"><a href="#cb41-2" aria-hidden="true" tabindex="-1"></a>  <span class="at">Aspecto =</span> <span class="fu">c</span>(</span>
<span id="cb41-3"><a href="#cb41-3" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Subtipo PAM50 más frecuente"</span>,</span>
<span id="cb41-4"><a href="#cb41-4" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Característica más central (PAM50)"</span>,</span>
<span id="cb41-5"><a href="#cb41-5" aria-hidden="true" tabindex="-1"></a>    <span class="st">"N° de comunidades (PAM50)"</span>,</span>
<span id="cb41-6"><a href="#cb41-6" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Densidad de red más alta"</span>,</span>
<span id="cb41-7"><a href="#cb41-7" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Característica hub principal"</span>,</span>
<span id="cb41-8"><a href="#cb41-8" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Gateway score más alto"</span>,</span>
<span id="cb41-9"><a href="#cb41-9" aria-hidden="true" tabindex="-1"></a>    <span class="st">"K-core máximo (PAM50)"</span>,</span>
<span id="cb41-10"><a href="#cb41-10" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Modularidad promedio"</span>,</span>
<span id="cb41-11"><a href="#cb41-11" aria-hidden="true" tabindex="-1"></a>    <span class="st">"Total de pacientes analizados"</span></span>
<span id="cb41-12"><a href="#cb41-12" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb41-13"><a href="#cb41-13" aria-hidden="true" tabindex="-1"></a>  <span class="at">Hallazgo =</span> <span class="fu">c</span>(</span>
<span id="cb41-14"><a href="#cb41-14" aria-hidden="true" tabindex="-1"></a>    pam50_dist<span class="sc">$</span>Subtipo[<span class="dv">1</span>],</span>
<span id="cb41-15"><a href="#cb41-15" aria-hidden="true" tabindex="-1"></a>    centralidad_pam50<span class="sc">$</span>Caracteristica[<span class="dv">1</span>],</span>
<span id="cb41-16"><a href="#cb41-16" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.character</span>(metricas_pam50<span class="sc">$</span>Valor[<span class="dv">6</span>]),</span>
<span id="cb41-17"><a href="#cb41-17" aria-hidden="true" tabindex="-1"></a>    comparacion<span class="sc">$</span>Red[<span class="fu">which.max</span>(comparacion<span class="sc">$</span>Densidad)],</span>
<span id="cb41-18"><a href="#cb41-18" aria-hidden="true" tabindex="-1"></a>    hubs_pam50<span class="sc">$</span>Caracteristica[<span class="dv">1</span>],</span>
<span id="cb41-19"><a href="#cb41-19" aria-hidden="true" tabindex="-1"></a>    gateway_pam50<span class="sc">$</span>Caracteristica[<span class="dv">1</span>],</span>
<span id="cb41-20"><a href="#cb41-20" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.character</span>(<span class="fu">max</span>(kcore_pam50)),</span>
<span id="cb41-21"><a href="#cb41-21" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.character</span>(<span class="fu">round</span>(<span class="fu">mean</span>(comparacion<span class="sc">$</span>Modularidad), <span class="dv">3</span>)),</span>
<span id="cb41-22"><a href="#cb41-22" aria-hidden="true" tabindex="-1"></a>    <span class="fu">as.character</span>(<span class="fu">nrow</span>(datos))</span>
<span id="cb41-23"><a href="#cb41-23" aria-hidden="true" tabindex="-1"></a>  )</span>
<span id="cb41-24"><a href="#cb41-24" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb41-25"><a href="#cb41-25" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb41-26"><a href="#cb41-26" aria-hidden="true" tabindex="-1"></a><span class="fu">datatable</span>(</span>
<span id="cb41-27"><a href="#cb41-27" aria-hidden="true" tabindex="-1"></a>  hallazgos,</span>
<span id="cb41-28"><a href="#cb41-28" aria-hidden="true" tabindex="-1"></a>  <span class="at">colnames =</span> <span class="fu">c</span>(<span class="st">"Aspecto Analizado"</span>, <span class="st">"Hallazgo Principal"</span>),</span>
<span id="cb41-29"><a href="#cb41-29" aria-hidden="true" tabindex="-1"></a>  <span class="at">options =</span> <span class="fu">list</span>(</span>
<span id="cb41-30"><a href="#cb41-30" aria-hidden="true" tabindex="-1"></a>    <span class="at">dom =</span> <span class="st">'t'</span>,</span>
<span id="cb41-31"><a href="#cb41-31" aria-hidden="true" tabindex="-1"></a>    <span class="at">pageLength =</span> <span class="dv">10</span></span>
<span id="cb41-32"><a href="#cb41-32" aria-hidden="true" tabindex="-1"></a>  ),</span>
<span id="cb41-33"><a href="#cb41-33" aria-hidden="true" tabindex="-1"></a>  <span class="at">caption =</span> <span class="st">"Resumen de hallazgos principales del análisis de redes"</span></span>
<span id="cb41-34"><a href="#cb41-34" aria-hidden="true" tabindex="-1"></a>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output-display">
<div class="datatables html-widget html-fill-item" id="htmlwidget-5f9ca6dee039e947ac65" style="width:100%;height:auto;"></div>
<script type="application/json" data-for="htmlwidget-5f9ca6dee039e947ac65">{"x":{"filter":"none","vertical":false,"caption":"<caption>Resumen de hallazgos principales del análisis de redes<\/caption>","data":[["1","2","3","4","5","6","7","8","9"],["Subtipo PAM50 más frecuente","Característica más central (PAM50)","N° de comunidades (PAM50)","Densidad de red más alta","Característica hub principal","Gateway score más alto","K-core máximo (PAM50)","Modularidad promedio","Total de pacientes analizados"],["LumA","HistoSubtype: Ductal/NST","5","3-Gene","HistoSubtype: Ductal/NST","HistoSubtype: Ductal/NST","24","0.103","2509"]],"container":"<table class=\"display\">\n  <thead>\n    <tr>\n      <th> <\/th>\n      <th>Aspecto Analizado<\/th>\n      <th>Hallazgo Principal<\/th>\n    <\/tr>\n  <\/thead>\n<\/table>","options":{"dom":"t","pageLength":10,"columnDefs":[{"orderable":false,"targets":0},{"name":" ","targets":0},{"name":"Aspecto","targets":1},{"name":"Hallazgo","targets":2}],"order":[],"autoWidth":false,"orderClasses":false}},"evals":[],"jsHooks":[]}</script>
</div>
</div>
</section>
<section id="implicaciones-clínicas" class="level3" data-number="2.10.2">
<h3 data-number="2.10.2" class="anchored" data-anchor-id="implicaciones-clínicas"><span class="header-section-number">2.10.2</span> Implicaciones Clínicas</h3>
<section id="características-clave-identificadas" class="level4" data-number="2.10.2.1">
<h4 data-number="2.10.2.1" class="anchored" data-anchor-id="características-clave-identificadas"><span class="header-section-number">2.10.2.1</span> Características clave identificadas:</h4>
<ol type="1">
<li><strong>Nodos Hub</strong>: Características que conectan múltiples aspectos del cáncer de mama
<ul>
<li>Alto grado de conectividad</li>
<li>Centralidad elevada en la red</li>
<li>Potencialmente útiles para estratificación de pacientes</li>
</ul></li>
<li><strong>Nodos Puente (Gateway)</strong>: Características que conectan diferentes comunidades
<ul>
<li>Puente entre subtipos moleculares</li>
<li>Relevantes para entender transiciones biológicas</li>
<li>Posibles blancos terapéuticos</li>
</ul></li>
<li><strong>Comunidades detectadas</strong>: Clusters de características que co-ocurren
<ul>
<li>Patrones de tratamiento</li>
<li>Asociaciones pronósticas</li>
<li>Grupos de riesgo</li>
</ul></li>
</ol>
</section>
</section>
<section id="consideraciones-para-investigación-futura" class="level3" data-number="2.10.3">
<h3 data-number="2.10.3" class="anchored" data-anchor-id="consideraciones-para-investigación-futura"><span class="header-section-number">2.10.3</span> Consideraciones para investigación futura:</h3>
<ul>
<li>Validación de patrones en cohortes independientes</li>
<li>Análisis de datos genómicos integrados</li>
<li>Estudio longitudinal de evolución de características</li>
<li>Desarrollo de modelos predictivos basados en redes</li>
</ul>
</section>
</section>
<section id="información-de-sesión" class="level2" data-number="2.11">
<h2 data-number="2.11" class="anchored" data-anchor-id="información-de-sesión"><span class="header-section-number">2.11</span> Información de Sesión</h2>
<div class="cell">
<details class="code-fold">
<summary>Code</summary>
<div class="sourceCode cell-code" id="cb42"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb42-1"><a href="#cb42-1" aria-hidden="true" tabindex="-1"></a><span class="fu">sessionInfo</span>()</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</details>
<div class="cell-output cell-output-stdout">
<pre><code>R version 4.4.2 (2024-10-31 ucrt)
Platform: x86_64-w64-mingw32/x64
Running under: Windows 11 x64 (build 26200)

Matrix products: default


locale:
[1] LC_COLLATE=Spanish_Chile.utf8  LC_CTYPE=Spanish_Chile.utf8   
[3] LC_MONETARY=Spanish_Chile.utf8 LC_NUMERIC=C                  
[5] LC_TIME=Spanish_Chile.utf8    

time zone: America/Santiago
tzcode source: internal

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
 [1] gridExtra_2.3      RColorBrewer_1.1-3 viridis_0.6.5      viridisLite_0.4.2 
 [5] ggraph_2.2.1       igraph_2.1.4       ggrepel_0.9.6      scales_1.4.0      
 [9] writexl_1.5.4      readxl_1.4.5       summarytools_1.1.4 table1_1.4.3      
[13] DT_0.33            kableExtra_1.4.0   naniar_1.1.0       lubridate_1.9.4   
[17] forcats_1.0.0      stringr_1.5.1      dplyr_1.1.4        purrr_1.0.4       
[21] readr_2.1.5        tidyr_1.3.1        tibble_3.3.0       ggplot2_3.5.2     
[25] tidyverse_2.0.0    knitr_1.50        

loaded via a namespace (and not attached):
 [1] tidyselect_1.2.1   farver_2.1.2       fastmap_1.2.0      tweenr_2.0.3      
 [5] digest_0.6.37      timechange_0.3.0   lifecycle_1.0.4    magrittr_2.0.3    
 [9] compiler_4.4.2     sass_0.4.10        rlang_1.1.6        tools_4.4.2       
[13] yaml_2.3.10        labeling_0.4.3     graphlayouts_1.2.2 htmlwidgets_1.6.4 
[17] plyr_1.8.9         xml2_1.3.8         withr_3.0.2        grid_4.4.2        
[21] polyclip_1.10-7    MASS_7.3-61        dichromat_2.0-0.1  cli_3.6.5         
[25] rmarkdown_2.29     ragg_1.4.0         generics_0.1.4     rstudioapi_0.17.1 
[29] reshape2_1.4.4     tzdb_0.5.0         cachem_1.1.0       ggforce_0.4.2     
[33] pander_0.6.6       cellranger_1.1.0   matrixStats_1.5.0  base64enc_0.1-3   
[37] vctrs_0.6.5        jsonlite_2.0.0     hms_1.1.3          rapportools_1.2   
[41] visdat_0.6.0       Formula_1.2-5      systemfonts_1.2.3  magick_2.8.7      
[45] crosstalk_1.2.1    jquerylib_0.1.4    glue_1.8.0         codetools_0.2-20  
[49] stringi_1.8.7      gtable_0.3.6       pillar_1.10.2      htmltools_0.5.8.1 
[53] R6_2.6.1           tcltk_4.4.2        textshaping_1.0.1  tidygraph_1.3.1   
[57] evaluate_1.0.3     backports_1.5.0    memoise_2.0.1      pryr_0.1.6        
[61] bslib_0.9.0        Rcpp_1.0.14        svglite_2.2.1      checkmate_2.3.2   
[65] xfun_0.52          pkgconfig_2.0.3   </code></pre>
</div>
</div>
<hr>
<p><strong>Nota</strong>: Todas las figuras han sido guardadas en la carpeta <code>figuras_redes/</code> para su uso posterior.</p>
</section>
</section>
<section id="resultados-1" class="level1" data-number="3">
<h1 data-number="3"><span class="header-section-number">3</span> Resultados</h1>
<p>La investigación se sustentó en el conjunto de datos METABRIC, que reúne información de 2509 mujeres con cáncer de mama invasivo. Este cohorte fue analizado mediante diferentes clasificaciones moleculares (PAM50 + Claudin‑low, clasificador de 3 genes e Integrative Cluster) y variables clínicas como estado de receptores hormonales, grado y estadio tumoral, tratamientos recibidos y desenlaces. El primer paso consistió en caracterizar la distribución de subtipos moleculares; se observó que el subtipo <strong>Luminal A</strong> del sistema PAM50 predominó en el 35 % de los casos. Esta información, plasmada en la <strong>Tabla 1</strong>, evidencia la abundancia de tumores luminales A en la cohorte, los cuales suelen asociarse a mejor pronóstico y a la presencia de receptores hormonales positivos. De manera similar, la <strong>Tabla 2</strong> muestra que, según el clasificador de tres genes, la mayoría de los tumores correspondían a los subtipos <strong>ER+/HER2‑</strong> de baja y alta proliferación (36 % y 35 %, respectivamente), reforzando la idea de que los tumores luminales son los más frecuentes en la muestra. El sistema Integrative Cluster identificó once agrupaciones, siendo los clústeres 8 y 3 los más comunes, pero dada su complejidad se mencionan de forma complementaria sin profundizar en ellos.</p>
<p>Tras esta caracterización inicial, se procedió a construir redes de co‑ocurrencia para estudiar cómo se relacionan las variables entre sí en cada sistema de clasificación. Cada nodo de estas redes representa una característica (por ejemplo, “ER positivo” o “Surgery: MASTECTOMY”), y las conexiones reflejan que dichas características se presentan juntas en un mismo paciente. La red PAM50 resultó constituida por 57 nodos y 604 aristas, con una densidad de 0,3784 y cinco comunidades definidas. Por su parte, la red del clasificador de tres genes presentó 53 nodos y 553 conexiones, siendo la más densa (0,4013). La red Integrative Cluster tuvo 61 nodos, 676 aristas y una densidad ligeramente menor (0,3694). Estos indicadores se resumen en la <strong>Tabla 3</strong> y ponen de manifiesto que, a pesar de las diferencias en el número de nodos, las tres redes comparten un alto nivel de <em>clustering</em> (~0,75), lo que indica que las características tienden a agruparse en triángulos o tríadas. Además, la modularidad de las redes fue cercana a 0,10, sugiriendo que aunque existen comunidades bien definidas, el grado de separación entre ellas es moderado. Para visualizar estas métricas comparativas de manera intuitiva, la <strong>Figura 2</strong> presenta un gráfico de barras en el que se aprecia claramente cómo la red 3‑Gene alcanza la mayor densidad, mientras que la PAM50 exhibe más comunidades pero con modularidad similar a las otras redes.</p>
<p>El análisis de centralidad permitió identificar cuáles variables actúan como nodos clave dentro de las redes. De manera consistente en todas las redes, <strong>“HistoSubtype: Ductal/NST”</strong> emergió como el nodo con mayor grado (es decir, con más conexiones), mayor <em>gateway score</em> y alta centralidad de intermediación. Esto significa que el carcinoma ductal invasivo, el tipo histológico más común, se relaciona con numerosas variables clínico‑moleculares y funciona como un puente entre diferentes comunidades de la red. Otros nodos destacados incluyeron el estado <strong>HER2 negativo</strong>, la positividad de receptores de estrógeno, la recepción de <strong>radioterapia</strong> y el tipo de cirugía (<em>mastectomía</em>), lo que sugiere que estas características están implicadas en múltiples patrones de co‑ocurrencia y podrían desempeñar un papel importante en la toma de decisiones terapéuticas. Para ilustrar estos hallazgos se propone incluir una <strong>Tabla 4</strong> que liste las veinte variables más centrales de la red PAM50 y sus medidas de grado, intermediación y eigenvector, así como una <strong>Figura 3</strong> que represente gráficamente la red, resaltando con mayor tamaño los nodos más conectados y con colores distintos las comunidades a las que pertenecen.</p>
<p>El estudio de las comunidades dentro de la red PAM50 reveló cinco agrupaciones principales. La comunidad más numerosa (Comunidad 2) estaba integrada por nodos relacionados con la negatividad de HER2, la positividad de estrógeno y el tipo de cirugía (mastectomía); la segunda comunidad (Comunidad 1) se caracterizaba por la presencia de tumores ductales/NST, radioterapia y grado 3; y la tercera (Comunidad 3) agrupaba variables de recurrencia, lateralidad y supervivencia. Este análisis sugiere que las pacientes con HER2 negativo y receptores hormonales positivos suelen someterse a mastectomía y se encuentran en un mismo subconjunto, mientras que las que presentan histotipo ductal y grado alto tienden a requerir radioterapia. La <strong>Tabla 5</strong> resume el número de nodos, grado medio y principales variables de cada comunidad, lo que facilita la identificación de patrones clínicos recurrentes y posibles subgrupos de riesgo. Aunque las redes 3‑Gene e Integrative Cluster muestran cuatro y seis comunidades respectivamente, su estructura es similar y por ello la discusión se centra en la red PAM50.</p>
<p>Para profundizar en los patrones clínicos, se compararon subgrupos específicos. En el caso de los subtipos luminales A y B, la <strong>Tabla 6</strong> muestra que ambos presentan tasas muy altas de receptores de estrógeno positivos, pero <strong>LumB</strong> se asocia a tumores de mayor grado (53 % grado 3 versus 24 % en LumA) y a mayor frecuencia de positividad HER2 (9,5 % versus 3 %). Ello explica por qué los pacientes con subtipo LumB tienen tratamientos más agresivos, mayor uso de terapia hormonal y radioterapia. En cuanto a los subtipos agresivos, la <strong>Tabla 7</strong> compara los tumores <strong>Triple Negativo</strong> y <strong>HER2+</strong> y revela que el primero tiene la mayor proporción de tumores de alto grado (83 %) y mayor uso de quimioterapia (51,6 %), mientras que el subtipo HER2+ presenta la tasa de mortalidad más alta (62,8 %). Cabe señalar que ambos subtipos muestran una recurrencia del 100 %, destacando su mal pronóstico.</p>
<p>El análisis de supervivencia y recurrencia por subtipo se presenta en las <strong>Tablas 8 y 9</strong>. Se constató que los subtipos Her2 y LumB tenían las proporciones más altas de pacientes fallecidos (70 % y 66 %, respectivamente), mientras que los subtipos Basal, Her2 y Luminal B presentaron recurrencias en casi la mitad de los casos. Estas cifras deben interpretarse con cautela, pues reflejan un análisis retrospectivo de una cohorte específica. No obstante, apoyan la evidencia de que la clasificación molecular y las características clínicas se traducen en diferencias pronósticas tangibles.</p>
<p>En conjunto, estos resultados muestran que la interrelación de variables clínicas y moleculares mediante análisis de redes permite identificar nodos críticos (como el histotipo ductal y el estado HER2), comunidades de características que co‑ocurren y subgrupos de pacientes con diferentes perfiles de tratamiento y pronóstico. La inclusión de tablas y figuras específicas—como las que comparan la distribución de subtipos, describen las métricas de las redes, enumeran las variables centrales y detallan las comunidades y subgrupos clínicos—ayuda a sostener los argumentos de la tesis y a presentar la información de forma clara y comprensible para el lector.</p>
</section>
<section id="discusión" class="level1" data-number="4">
<h1 data-number="4"><span class="header-section-number">4</span> Discusión</h1>
<p>El análisis de la cohorte METABRIC puso de manifiesto que el subtipo <strong>Luminal A</strong> es el más frecuente, seguido por el subtipo <strong>Luminal B</strong> y otros subtipos menos comunes. Desde un punto de vista clínico, esta distribución coincide con la práctica diaria, donde los tumores luminales son predominantes y, en general, presentan una mejor respuesta a la hormonoterapia. Sin embargo, nuestros resultados muestran que, aunque ambos subtipos son ER positivos, las pacientes con subtipo Luminal B presentan una mayor proporción de tumores de grado 3 y una mayor frecuencia de HER2 positivo, lo que implica un comportamiento más agresivo y la necesidad de terapias combinadas (hormonoterapia + anti‑HER2 y radioterapia). Esta observación se apoya en la literatura, que describe a los tumores luminales B como de <strong>mayor agresividad</strong>, asociados a grado tumoral alto, mayor tamaño tumoral y afectación ganglionar y a tasas de supervivencia algo inferiores en comparación con los luminales A. Por lo tanto, la identificación de estas diferencias en la red PAM50 refuerza la necesidad de individualizar el manejo dentro de los subtipos luminales.</p>
<p>La comparación entre los subtipos <strong>Triple Negativo</strong> y <strong>HER2+</strong> evidenció que ambos presentan índices de recurrencia y mortalidad extremadamente elevados. Clínicamente, esto subraya la agresividad de estos tumores y la importancia de estrategias terapéuticas intensivas. Los tumores triples negativos carecen de blancos hormonales y de HER2, por lo que la quimioterapia sigue siendo el pilar del tratamiento; además, suelen asociarse a grados altos y a una peor supervivencia, hallazgos que fueron consistentes con nuestra cohorte. En contraste, los tumores HER2+ pueden beneficiarse de terapias dirigidas como trastuzumab o pertuzumab, lo que podría explicar por qué, a pesar de ser agresivos, mostraron una mortalidad algo menor que los triple negativos en nuestra serie. Estos resultados enfatizan la importancia de la clasificación molecular en la toma de decisiones terapéuticas y sugieren que la incorporación de terapias dirigidas puede modificar el pronóstico de subtipos específicos.</p>
<p>El análisis de redes de co‑ocurrencia aportó una visión complementaria al evidenciar que ciertas variables clínicas y moleculares—en particular el <strong>histotipo ductal/NST</strong>, la negatividad de HER2 y la positividad de receptores de estrógeno—actúan como nodos centrales que conectan múltiples características y comunidades. En términos de salud, esto significa que estas variables no solo son determinantes en la biología tumoral, sino que también están asociadas a la elección de tratamientos y a los desenlaces de las pacientes. La centralidad del histotipo ductal sugiere que la mayoría de las decisiones clínicas en la cohorte giran en torno a este tipo histológico, el cual es el más común en cáncer de mama invasivo. Asimismo, la existencia de un núcleo K‑core de 24 nodos que integra variables como <em>ER positivo</em>, <em>PR negativo</em> y <em>HER2 negativo</em> indica que estos factores forman el “corazón” de las interacciones clínicas, lo que podría guiar futuros estudios sobre estratificación y personalización del tratamiento.</p>
<p>Los hallazgos de supervivencia y recurrencia también ofrecen implicaciones prácticas: los subtipos Her2 y Luminal B mostraron las tasas de mortalidad más altas, mientras que los subtipos Basal, Her2 y Luminal B presentaron recurrencias en casi la mitad de los casos. Estas cifras deben interpretarse con cautela, pues reflejan un análisis retrospectivo de una cohorte específica. No obstante, apoyan la evidencia de que la clasificación molecular y las características clínicas se traducen en diferencias pronósticas tangibles.</p>
<p>A pesar de sus aportes, este estudio presenta limitaciones. En primer lugar, se basa en un solo conjunto de datos retrospectivo, lo que puede no reflejar la diversidad de poblaciones y prácticas terapéuticas actuales. Además, el análisis de redes es descriptivo y no establece causalidad; la co‑ocurrencia de variables no implica que una cause la otra. Por último, la falta de información genómica más profunda y de datos longitudinales impide evaluar cómo cambian las interrelaciones entre variables a lo largo del tiempo o tras intervenciones específicas.</p>
<p>En el futuro, sería valioso replicar este análisis en cohortes independientes y con datos más recientes, así como integrar información genómica y proteómica para enriquecer las redes. También se podrían explorar métodos de análisis dinámico que evalúen cómo evolucionan las redes con el tratamiento y la progresión de la enfermedad. Desde una perspectiva clínica, la identificación de nodos centrales y comunidades abre la puerta a estrategias de estratificación de pacientes y a la búsqueda de nuevos blancos terapéuticos. El desafío radica en traducir estos hallazgos a la práctica clínica, combinando la ciencia de datos con ensayos clínicos que validen la utilidad de estos enfoques en la mejora de los resultados de salud.</p>
</section>

</main>
<!-- /main column -->
<script id="quarto-html-after-body" type="application/javascript">
  window.document.addEventListener("DOMContentLoaded", function (event) {
    const icon = "";
    const anchorJS = new window.AnchorJS();
    anchorJS.options = {
      placement: 'right',
      icon: icon
    };
    anchorJS.add('.anchored');
    const isCodeAnnotation = (el) => {
      for (const clz of el.classList) {
        if (clz.startsWith('code-annotation-')) {                     
          return true;
        }
      }
      return false;
    }
    const onCopySuccess = function(e) {
      // button target
      const button = e.trigger;
      // don't keep focus
      button.blur();
      // flash "checked"
      button.classList.add('code-copy-button-checked');
      var currentTitle = button.getAttribute("title");
      button.setAttribute("title", "Copied!");
      let tooltip;
      if (window.bootstrap) {
        button.setAttribute("data-bs-toggle", "tooltip");
        button.setAttribute("data-bs-placement", "left");
        button.setAttribute("data-bs-title", "Copied!");
        tooltip = new bootstrap.Tooltip(button, 
          { trigger: "manual", 
            customClass: "code-copy-button-tooltip",
            offset: [0, -8]});
        tooltip.show();    
      }
      setTimeout(function() {
        if (tooltip) {
          tooltip.hide();
          button.removeAttribute("data-bs-title");
          button.removeAttribute("data-bs-toggle");
          button.removeAttribute("data-bs-placement");
        }
        button.setAttribute("title", currentTitle);
        button.classList.remove('code-copy-button-checked');
      }, 1000);
      // clear code selection
      e.clearSelection();
    }
    const getTextToCopy = function(trigger) {
        const codeEl = trigger.previousElementSibling.cloneNode(true);
        for (const childEl of codeEl.children) {
          if (isCodeAnnotation(childEl)) {
            childEl.remove();
          }
        }
        return codeEl.innerText;
    }
    const clipboard = new window.ClipboardJS('.code-copy-button:not([data-in-quarto-modal])', {
      text: getTextToCopy
    });
    clipboard.on('success', onCopySuccess);
    if (window.document.getElementById('quarto-embedded-source-code-modal')) {
      const clipboardModal = new window.ClipboardJS('.code-copy-button[data-in-quarto-modal]', {
        text: getTextToCopy,
        container: window.document.getElementById('quarto-embedded-source-code-modal')
      });
      clipboardModal.on('success', onCopySuccess);
    }
      var localhostRegex = new RegExp(/^(?:http|https):\/\/localhost\:?[0-9]*\//);
      var mailtoRegex = new RegExp(/^mailto:/);
        var filterRegex = new RegExp('/' + window.location.host + '/');
      var isInternal = (href) => {
          return filterRegex.test(href) || localhostRegex.test(href) || mailtoRegex.test(href);
      }
      // Inspect non-navigation links and adorn them if external
     var links = window.document.querySelectorAll('a[href]:not(.nav-link):not(.navbar-brand):not(.toc-action):not(.sidebar-link):not(.sidebar-item-toggle):not(.pagination-link):not(.no-external):not([aria-hidden]):not(.dropdown-item):not(.quarto-navigation-tool):not(.about-link)');
      for (var i=0; i<links.length; i++) {
        const link = links[i];
        if (!isInternal(link.href)) {
          // undo the damage that might have been done by quarto-nav.js in the case of
          // links that we want to consider external
          if (link.dataset.originalHref !== undefined) {
            link.href = link.dataset.originalHref;
          }
        }
      }
    function tippyHover(el, contentFn, onTriggerFn, onUntriggerFn) {
      const config = {
        allowHTML: true,
        maxWidth: 500,
        delay: 100,
        arrow: false,
        appendTo: function(el) {
            return el.parentElement;
        },
        interactive: true,
        interactiveBorder: 10,
        theme: 'quarto',
        placement: 'bottom-start',
      };
      if (contentFn) {
        config.content = contentFn;
      }
      if (onTriggerFn) {
        config.onTrigger = onTriggerFn;
      }
      if (onUntriggerFn) {
        config.onUntrigger = onUntriggerFn;
      }
      window.tippy(el, config); 
    }
    const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
    for (var i=0; i<noterefs.length; i++) {
      const ref = noterefs[i];
      tippyHover(ref, function() {
        // use id or data attribute instead here
        let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
        try { href = new URL(href).hash; } catch {}
        const id = href.replace(/^#\/?/, "");
        const note = window.document.getElementById(id);
        if (note) {
          return note.innerHTML;
        } else {
          return "";
        }
      });
    }
    const xrefs = window.document.querySelectorAll('a.quarto-xref');
    const processXRef = (id, note) => {
      // Strip column container classes
      const stripColumnClz = (el) => {
        el.classList.remove("page-full", "page-columns");
        if (el.children) {
          for (const child of el.children) {
            stripColumnClz(child);
          }
        }
      }
      stripColumnClz(note)
      if (id === null || id.startsWith('sec-')) {
        // Special case sections, only their first couple elements
        const container = document.createElement("div");
        if (note.children && note.children.length > 2) {
          container.appendChild(note.children[0].cloneNode(true));
          for (let i = 1; i < note.children.length; i++) {
            const child = note.children[i];
            if (child.tagName === "P" && child.innerText === "") {
              continue;
            } else {
              container.appendChild(child.cloneNode(true));
              break;
            }
          }
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(container);
          }
          return container.innerHTML
        } else {
          if (window.Quarto?.typesetMath) {
            window.Quarto.typesetMath(note);
          }
          return note.innerHTML;
        }
      } else {
        // Remove any anchor links if they are present
        const anchorLink = note.querySelector('a.anchorjs-link');
        if (anchorLink) {
          anchorLink.remove();
        }
        if (window.Quarto?.typesetMath) {
          window.Quarto.typesetMath(note);
        }
        if (note.classList.contains("callout")) {
          return note.outerHTML;
        } else {
          return note.innerHTML;
        }
      }
    }
    for (var i=0; i<xrefs.length; i++) {
      const xref = xrefs[i];
      tippyHover(xref, undefined, function(instance) {
        instance.disable();
        let url = xref.getAttribute('href');
        let hash = undefined; 
        if (url.startsWith('#')) {
          hash = url;
        } else {
          try { hash = new URL(url).hash; } catch {}
        }
        if (hash) {
          const id = hash.replace(/^#\/?/, "");
          const note = window.document.getElementById(id);
          if (note !== null) {
            try {
              const html = processXRef(id, note.cloneNode(true));
              instance.setContent(html);
            } finally {
              instance.enable();
              instance.show();
            }
          } else {
            // See if we can fetch this
            fetch(url.split('#')[0])
            .then(res => res.text())
            .then(html => {
              const parser = new DOMParser();
              const htmlDoc = parser.parseFromString(html, "text/html");
              const note = htmlDoc.getElementById(id);
              if (note !== null) {
                const html = processXRef(id, note);
                instance.setContent(html);
              } 
            }).finally(() => {
              instance.enable();
              instance.show();
            });
          }
        } else {
          // See if we can fetch a full url (with no hash to target)
          // This is a special case and we should probably do some content thinning / targeting
          fetch(url)
          .then(res => res.text())
          .then(html => {
            const parser = new DOMParser();
            const htmlDoc = parser.parseFromString(html, "text/html");
            const note = htmlDoc.querySelector('main.content');
            if (note !== null) {
              // This should only happen for chapter cross references
              // (since there is no id in the URL)
              // remove the first header
              if (note.children.length > 0 && note.children[0].tagName === "HEADER") {
                note.children[0].remove();
              }
              const html = processXRef(null, note);
              instance.setContent(html);
            } 
          }).finally(() => {
            instance.enable();
            instance.show();
          });
        }
      }, function(instance) {
      });
    }
        let selectedAnnoteEl;
        const selectorForAnnotation = ( cell, annotation) => {
          let cellAttr = 'data-code-cell="' + cell + '"';
          let lineAttr = 'data-code-annotation="' +  annotation + '"';
          const selector = 'span[' + cellAttr + '][' + lineAttr + ']';
          return selector;
        }
        const selectCodeLines = (annoteEl) => {
          const doc = window.document;
          const targetCell = annoteEl.getAttribute("data-target-cell");
          const targetAnnotation = annoteEl.getAttribute("data-target-annotation");
          const annoteSpan = window.document.querySelector(selectorForAnnotation(targetCell, targetAnnotation));
          const lines = annoteSpan.getAttribute("data-code-lines").split(",");
          const lineIds = lines.map((line) => {
            return targetCell + "-" + line;
          })
          let top = null;
          let height = null;
          let parent = null;
          if (lineIds.length > 0) {
              //compute the position of the single el (top and bottom and make a div)
              const el = window.document.getElementById(lineIds[0]);
              top = el.offsetTop;
              height = el.offsetHeight;
              parent = el.parentElement.parentElement;
            if (lineIds.length > 1) {
              const lastEl = window.document.getElementById(lineIds[lineIds.length - 1]);
              const bottom = lastEl.offsetTop + lastEl.offsetHeight;
              height = bottom - top;
            }
            if (top !== null && height !== null && parent !== null) {
              // cook up a div (if necessary) and position it 
              let div = window.document.getElementById("code-annotation-line-highlight");
              if (div === null) {
                div = window.document.createElement("div");
                div.setAttribute("id", "code-annotation-line-highlight");
                div.style.position = 'absolute';
                parent.appendChild(div);
              }
              div.style.top = top - 2 + "px";
              div.style.height = height + 4 + "px";
              div.style.left = 0;
              let gutterDiv = window.document.getElementById("code-annotation-line-highlight-gutter");
              if (gutterDiv === null) {
                gutterDiv = window.document.createElement("div");
                gutterDiv.setAttribute("id", "code-annotation-line-highlight-gutter");
                gutterDiv.style.position = 'absolute';
                const codeCell = window.document.getElementById(targetCell);
                const gutter = codeCell.querySelector('.code-annotation-gutter');
                gutter.appendChild(gutterDiv);
              }
              gutterDiv.style.top = top - 2 + "px";
              gutterDiv.style.height = height + 4 + "px";
            }
            selectedAnnoteEl = annoteEl;
          }
        };
        const unselectCodeLines = () => {
          const elementsIds = ["code-annotation-line-highlight", "code-annotation-line-highlight-gutter"];
          elementsIds.forEach((elId) => {
            const div = window.document.getElementById(elId);
            if (div) {
              div.remove();
            }
          });
          selectedAnnoteEl = undefined;
        };
          // Handle positioning of the toggle
      window.addEventListener(
        "resize",
        throttle(() => {
          elRect = undefined;
          if (selectedAnnoteEl) {
            selectCodeLines(selectedAnnoteEl);
          }
        }, 10)
      );
      function throttle(fn, ms) {
      let throttle = false;
      let timer;
        return (...args) => {
          if(!throttle) { // first call gets through
              fn.apply(this, args);
              throttle = true;
          } else { // all the others get throttled
              if(timer) clearTimeout(timer); // cancel #2
              timer = setTimeout(() => {
                fn.apply(this, args);
                timer = throttle = false;
              }, ms);
          }
        };
      }
        // Attach click handler to the DT
        const annoteDls = window.document.querySelectorAll('dt[data-target-cell]');
        for (const annoteDlNode of annoteDls) {
          annoteDlNode.addEventListener('click', (event) => {
            const clickedEl = event.target;
            if (clickedEl !== selectedAnnoteEl) {
              unselectCodeLines();
              const activeEl = window.document.querySelector('dt[data-target-cell].code-annotation-active');
              if (activeEl) {
                activeEl.classList.remove('code-annotation-active');
              }
              selectCodeLines(clickedEl);
              clickedEl.classList.add('code-annotation-active');
            } else {
              // Unselect the line
              unselectCodeLines();
              clickedEl.classList.remove('code-annotation-active');
            }
          });
        }
    const findCites = (el) => {
      const parentEl = el.parentElement;
      if (parentEl) {
        const cites = parentEl.dataset.cites;
        if (cites) {
          return {
            el,
            cites: cites.split(' ')
          };
        } else {
          return findCites(el.parentElement)
        }
      } else {
        return undefined;
      }
    };
    var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
    for (var i=0; i<bibliorefs.length; i++) {
      const ref = bibliorefs[i];
      const citeInfo = findCites(ref);
      if (citeInfo) {
        tippyHover(citeInfo.el, function() {
          var popup = window.document.createElement('div');
          citeInfo.cites.forEach(function(cite) {
            var citeDiv = window.document.createElement('div');
            citeDiv.classList.add('hanging-indent');
            citeDiv.classList.add('csl-entry');
            var biblioDiv = window.document.getElementById('ref-' + cite);
            if (biblioDiv) {
              citeDiv.innerHTML = biblioDiv.innerHTML;
            }
            popup.appendChild(citeDiv);
          });
          return popup.innerHTML;
        });
      }
    }
  });
  </script>
</div> <!-- /content -->




<script src="index_files/libs/quarto-html/zenscroll-min.js"></script>
</body></html>